{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "6025988b",
   "metadata": {},
   "source": [
    "# SETTING UP ENVIRONMENT"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "20ed7071",
   "metadata": {},
   "source": [
    "1) <b>fork github</b> (add on github and use git pull):"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b871c8d3",
   "metadata": {},
   "source": [
    "*git clone git@github.com:flurinh/mt.git*"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "77588c8e",
   "metadata": {},
   "source": [
    "2) <b>create conda environment</b>"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9b403eb8",
   "metadata": {},
   "source": [
    "*conda create --name gpcr --file requirements.txt*"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "417e690e",
   "metadata": {},
   "source": [
    "*conda activate gpcr*"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bf708782",
   "metadata": {},
   "source": [
    "3) <b>create (missing) folders</b>"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "03692fdf",
   "metadata": {},
   "source": [
    "unzip file"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b25340c3",
   "metadata": {},
   "source": [
    "4) <b>check if everything works</b> (import)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "36ec8af3",
   "metadata": {},
   "outputs": [],
   "source": [
    "from processing.utils import *\n",
    "from processing.utils2 import *\n",
    "from processing.utils3 import *\n",
    "from processing.gpcrdb_soup import *\n",
    "from processing.download import *\n",
    "from processing.processor import *\n",
    "from processing.df_to_cif import *\n",
    "from analysis.analysis import *"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "bd69d6f6",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "# DATA PROCESSING"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "898b23b3",
   "metadata": {
    "hidden": true
   },
   "source": [
    "The data (and file structure) is provided in a zipfile, so you do not have to download and process everything.\n",
    "This walkthrough shows how to (theoretically) build from scratch and/or update your outdated data."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a97681b8",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "## DOWNLOADING / UPDATING METADATA"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "60bfe700",
   "metadata": {
    "hidden": true
   },
   "source": [
    "With metadata we primarily refer to the [gpcrdb](https://gpcrdb.org) [GPCR structure table](https://gpcrdb.org/structure/) containing a curated list of gpcrs (their name, uniprot id etc) - new entries in this list may automatically be updated!"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "cf6e3e71",
   "metadata": {
    "hidden": true
   },
   "source": [
    "In our code we use a dedicated downloading class which handles both the <b>table download</b> and the <b>cif file download</b> of all missing structures. The table is saved to the subdirectory \"*data/gpcrdb/*\", the cif (or pdb) files are put into *data/mmcif/*."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "38ab5726",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  0%|                                                                                                                              | 0/1 [00:00<?, ?it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "writing gpcrdb table to file: data/data_table.pkl\n",
      "Reloading pdb files...\n",
      "Missing pdbs: ['1F88']\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 1/1 [00:00<00:00,  1.44it/s]\n"
     ]
    }
   ],
   "source": [
    "D = Download(fileformat='cif')\n",
    "D.download_table(reload=True)  # without reload-flag set to True (i.e. file exists) we do not re-download\n",
    "# since these are unprocessed files, existing files are NOT RE-DOWNLOADED\n",
    "# however we \"reference\" the gpcrdb gpcr structure table to see if we have missing structures, and download those\n",
    "D.download_pdbs()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "730d6e73",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "## DOWNLOADING MORE (META)DATA"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "37a63e5e",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Next we initialize our processing class, creatively named ***CifProcessor***, which includes all the functions used to gather more metainformation; Such as download data from **sifts mapping** (aka the mapping of the sequence of the pdb structure against the related uniprot gene sequence), the **(generic residue) numbering** from gpcrdb, as well as does the calculation of consecutive **psi-** and **phi-angles** and **assigns the generic residue numbers** for both the gpcr or (in complexes) the gprotein. (This of course includes alignment functions and many more). Last and not least the common data format in this processor class, as we will see, are python **DataFrames**."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b123920c",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Let's start slow by initializing the processor:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "cbc5b506",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "p = CifProcessor()"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "38d8d3d9",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Next we load 3 datatables (referred to as metainfo): sifts mapping ('p.mappings'), table with uniprot ids and corresponding generic residue numbers ('p.numbering') and our previously downloaded data structure table ('p.table'). "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "c6657b50",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "p.read_pkl_metainfo()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "4c7e98ec",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>identifier</th>\n",
       "      <th>name</th>\n",
       "      <th>mappings</th>\n",
       "      <th>PDB</th>\n",
       "      <th>uniprot</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>D9IEF7_BPT4</td>\n",
       "      <td>D9IEF7_BPT4</td>\n",
       "      <td>{'entity_id': 1, 'chain_id': 'A', 'start': {'a...</td>\n",
       "      <td>6LUQ</td>\n",
       "      <td>D9IEF7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>DRD2_HUMAN</td>\n",
       "      <td>DRD2_HUMAN</td>\n",
       "      <td>{'entity_id': 1, 'chain_id': 'A', 'start': {'a...</td>\n",
       "      <td>6LUQ</td>\n",
       "      <td>P14416</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>DRD2_HUMAN</td>\n",
       "      <td>DRD2_HUMAN</td>\n",
       "      <td>{'entity_id': 1, 'chain_id': 'A', 'start': {'a...</td>\n",
       "      <td>6LUQ</td>\n",
       "      <td>P14416</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    identifier         name  \\\n",
       "0  D9IEF7_BPT4  D9IEF7_BPT4   \n",
       "1   DRD2_HUMAN   DRD2_HUMAN   \n",
       "2   DRD2_HUMAN   DRD2_HUMAN   \n",
       "\n",
       "                                            mappings   PDB uniprot  \n",
       "0  {'entity_id': 1, 'chain_id': 'A', 'start': {'a...  6LUQ  D9IEF7  \n",
       "1  {'entity_id': 1, 'chain_id': 'A', 'start': {'a...  6LUQ  P14416  \n",
       "2  {'entity_id': 1, 'chain_id': 'A', 'start': {'a...  6LUQ  P14416  "
      ]
     },
     "execution_count": 19,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "p.mappings.head(3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "76fa6c49",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PDB</th>\n",
       "      <th>identifier</th>\n",
       "      <th>family</th>\n",
       "      <th>numbering</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>6LUQ</td>\n",
       "      <td>drd2_human</td>\n",
       "      <td>001_001_004_002</td>\n",
       "      <td>[{'sequence_number': 1, 'amino_acid': 'M', 'pr...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>5L7D</td>\n",
       "      <td>smo_human</td>\n",
       "      <td>006_001_001_011</td>\n",
       "      <td>[{'sequence_number': 1, 'amino_acid': 'M', 'pr...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>5TUD</td>\n",
       "      <td>5ht2b_human</td>\n",
       "      <td>001_001_001_007</td>\n",
       "      <td>[{'sequence_number': 1, 'amino_acid': 'M', 'pr...</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    PDB   identifier           family  \\\n",
       "0  6LUQ   drd2_human  001_001_004_002   \n",
       "1  5L7D    smo_human  006_001_001_011   \n",
       "2  5TUD  5ht2b_human  001_001_001_007   \n",
       "\n",
       "                                           numbering  \n",
       "0  [{'sequence_number': 1, 'amino_acid': 'M', 'pr...  \n",
       "1  [{'sequence_number': 1, 'amino_acid': 'M', 'pr...  \n",
       "2  [{'sequence_number': 1, 'amino_acid': 'M', 'pr...  "
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "p.numbering.head(3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "b0ca7649",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>uniprot(gene)</th>\n",
       "      <th>receptor family</th>\n",
       "      <th>Cl.</th>\n",
       "      <th>Species</th>\n",
       "      <th>Method</th>\n",
       "      <th>PDB</th>\n",
       "      <th>Resolution</th>\n",
       "      <th>Preferred Chain</th>\n",
       "      <th>State</th>\n",
       "      <th>Degree active %</th>\n",
       "      <th>Family</th>\n",
       "      <th>Subtype</th>\n",
       "      <th>Function</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>STE2</td>\n",
       "      <td>STE2</td>\n",
       "      <td>D1(Ste2-likefungalpheromone)</td>\n",
       "      <td>strainATCC204508/S288c</td>\n",
       "      <td>cryo-EM</td>\n",
       "      <td>7AD3</td>\n",
       "      <td>3.5</td>\n",
       "      <td>A</td>\n",
       "      <td>Active</td>\n",
       "      <td>-</td>\n",
       "      <td>GPa1family</td>\n",
       "      <td>αPa1</td>\n",
       "      <td>Agonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>OPRM</td>\n",
       "      <td>Opioid</td>\n",
       "      <td>A(Rhodopsin)</td>\n",
       "      <td>Mouse</td>\n",
       "      <td>X-ray</td>\n",
       "      <td>4DKL</td>\n",
       "      <td>2.8</td>\n",
       "      <td>A</td>\n",
       "      <td>Inactive</td>\n",
       "      <td>7</td>\n",
       "      <td>-</td>\n",
       "      <td>-</td>\n",
       "      <td>Antagonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>OXYR</td>\n",
       "      <td>Vasopressinandoxytocin</td>\n",
       "      <td>A(Rhodopsin)</td>\n",
       "      <td>Human</td>\n",
       "      <td>X-ray</td>\n",
       "      <td>6TPK</td>\n",
       "      <td>3.2</td>\n",
       "      <td>A</td>\n",
       "      <td>Inactive</td>\n",
       "      <td>24</td>\n",
       "      <td>-</td>\n",
       "      <td>-</td>\n",
       "      <td>Antagonist</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  uniprot(gene)         receptor family                           Cl.  \\\n",
       "0          STE2                    STE2  D1(Ste2-likefungalpheromone)   \n",
       "1          OPRM                  Opioid                  A(Rhodopsin)   \n",
       "2          OXYR  Vasopressinandoxytocin                  A(Rhodopsin)   \n",
       "\n",
       "                  Species   Method   PDB  Resolution Preferred Chain  \\\n",
       "0  strainATCC204508/S288c  cryo-EM  7AD3         3.5               A   \n",
       "1                   Mouse    X-ray  4DKL         2.8               A   \n",
       "2                   Human    X-ray  6TPK         3.2               A   \n",
       "\n",
       "      State Degree active %      Family Subtype    Function  \n",
       "0    Active               -  GPa1family    αPa1     Agonist  \n",
       "1  Inactive               7           -       -  Antagonist  \n",
       "2  Inactive              24           -       -  Antagonist  "
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "p.table.head(3)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "7c2ec516",
   "metadata": {
    "hidden": true
   },
   "source": [
    "This next step takes a very long time (**~30 minutes**), depending on if we make calls to download generic residue numbers (for uniprot ids) from [gpcrdb](https://docs.gpcrdb.org/web_services.html). (Of course if you have new samples, setting *reload_numbering* to *False* will still mean you are going to update your data to include it.) If you have an existing table, **NOT RELOADING IS ADVISED**!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 29,
   "id": "43b84d1d",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "557it [00:00, 5354.56it/s]\n"
     ]
    }
   ],
   "source": [
    "p.make_metainfo(reload_numbering=False, reload_mapping=False, overwrite=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d5939495",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "## CREATING RAW DATAFRAMES"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1e6e2c7b",
   "metadata": {
    "hidden": true
   },
   "source": [
    "To start processing files from scratch you may run this next line, it will **DELETE** all pickle files (*.pkl*, the format of the dataframes) in your specified folder:"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d84a5966",
   "metadata": {
    "hidden": true
   },
   "source": [
    "*p.del_pkl(folder='data/raw/')*"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a1f39500",
   "metadata": {
    "hidden": true
   },
   "source": [
    "So far we only have a folder with unprocessed cif files - data is in text format. Very slow to process etc, so we first create the base structure of our dataset: a list of dataframes, **one dataframe per structure**. (During the loading process we also calculate the psi, phi and omega angles for all residues in the structure.) These are then saved (without any further processing to the subdirectory *data/raw/*."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 30,
   "id": "2f5a5397",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "557it [00:00, 29309.45it/s]\n"
     ]
    }
   ],
   "source": [
    "p.make_raws(overwrite=False)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "87282e36",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Setting overwrite to False will skip all processed samples! If it is set to True, you process all your samples and overwrite the existing data."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1cd1908f",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "# GENERIC RESIDUE NUMBERS"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9493da95",
   "metadata": {
    "hidden": true
   },
   "source": [
    "As mentioned, the dataformat we use (after loading the cif files) are DataFrames (implements a variety of SQL methods and provides easy pythonic access to big datatables). Now each of our structures corresponds to a single DataFrame in the main data container in our processor class: p.dfl (dataframe list), for easier access and search it is paired with a list containing only the respective pdb name, p.dfl_list."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 33,
   "id": "fb521420",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[]"
      ]
     },
     "execution_count": 33,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "p.dfl"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "016b0139",
   "metadata": {
    "hidden": true
   },
   "source": [
    "What happened? Well currently we have no new structures (since a few days ago when I last updated the data), thus no new structures were processed and added to the data list. To load our data:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "08f48fa1",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading files without generic numbers...\n"
     ]
    }
   ],
   "source": [
    "p.read_pkl(mode='', folder='data/raw/')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "954020be",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Let's have a look at the first entry:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 38,
   "id": "5e2320b2",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PDB</th>\n",
       "      <th>group_PDB</th>\n",
       "      <th>auth_asym_id</th>\n",
       "      <th>label_asym_id</th>\n",
       "      <th>label_seq_id</th>\n",
       "      <th>auth_seq_id</th>\n",
       "      <th>label_comp_id</th>\n",
       "      <th>id</th>\n",
       "      <th>label_atom_id</th>\n",
       "      <th>type_symbol</th>\n",
       "      <th>Cartn_x</th>\n",
       "      <th>Cartn_y</th>\n",
       "      <th>Cartn_z</th>\n",
       "      <th>phi</th>\n",
       "      <th>omega</th>\n",
       "      <th>psi</th>\n",
       "      <th>identifier</th>\n",
       "      <th>label_comp_sid</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>2J4Y</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>MET</td>\n",
       "      <td>4</td>\n",
       "      <td>N</td>\n",
       "      <td>N</td>\n",
       "      <td>-32.043</td>\n",
       "      <td>16.427</td>\n",
       "      <td>-13.796</td>\n",
       "      <td>-144.390068</td>\n",
       "      <td>179.597085</td>\n",
       "      <td>-176.94101</td>\n",
       "      <td>OPSD_BOVIN</td>\n",
       "      <td>M</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2J4Y</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>MET</td>\n",
       "      <td>5</td>\n",
       "      <td>CA</td>\n",
       "      <td>C</td>\n",
       "      <td>-31.142</td>\n",
       "      <td>16.987</td>\n",
       "      <td>-12.785</td>\n",
       "      <td>-144.390068</td>\n",
       "      <td>179.597085</td>\n",
       "      <td>-176.94101</td>\n",
       "      <td>OPSD_BOVIN</td>\n",
       "      <td>M</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2J4Y</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>2</td>\n",
       "      <td>1</td>\n",
       "      <td>MET</td>\n",
       "      <td>6</td>\n",
       "      <td>C</td>\n",
       "      <td>C</td>\n",
       "      <td>-29.663</td>\n",
       "      <td>16.868</td>\n",
       "      <td>-13.200</td>\n",
       "      <td>-144.390068</td>\n",
       "      <td>179.597085</td>\n",
       "      <td>-176.94101</td>\n",
       "      <td>OPSD_BOVIN</td>\n",
       "      <td>M</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    PDB group_PDB auth_asym_id label_asym_id  label_seq_id auth_seq_id  \\\n",
       "0  2J4Y      ATOM            A             A             2           1   \n",
       "1  2J4Y      ATOM            A             A             2           1   \n",
       "2  2J4Y      ATOM            A             A             2           1   \n",
       "\n",
       "  label_comp_id id label_atom_id type_symbol  Cartn_x Cartn_y  Cartn_z  \\\n",
       "0           MET  4             N           N  -32.043  16.427  -13.796   \n",
       "1           MET  5            CA           C  -31.142  16.987  -12.785   \n",
       "2           MET  6             C           C  -29.663  16.868  -13.200   \n",
       "\n",
       "          phi       omega        psi  identifier label_comp_sid  \n",
       "0 -144.390068  179.597085 -176.94101  OPSD_BOVIN              M  \n",
       "1 -144.390068  179.597085 -176.94101  OPSD_BOVIN              M  \n",
       "2 -144.390068  179.597085 -176.94101  OPSD_BOVIN              M  "
      ]
     },
     "execution_count": 38,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "p.dfl[10].head(3)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "874a7a8d",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "'2J4Y'"
      ]
     },
     "execution_count": 39,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "p.dfl_list[10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "4a43bf8d",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>uniprot(gene)</th>\n",
       "      <th>receptor family</th>\n",
       "      <th>Cl.</th>\n",
       "      <th>Species</th>\n",
       "      <th>Method</th>\n",
       "      <th>PDB</th>\n",
       "      <th>Resolution</th>\n",
       "      <th>Preferred Chain</th>\n",
       "      <th>State</th>\n",
       "      <th>Degree active %</th>\n",
       "      <th>Family</th>\n",
       "      <th>Subtype</th>\n",
       "      <th>Function</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>STE2</td>\n",
       "      <td>STE2</td>\n",
       "      <td>D1(Ste2-likefungalpheromone)</td>\n",
       "      <td>strainATCC204508/S288c</td>\n",
       "      <td>cryo-EM</td>\n",
       "      <td>7AD3</td>\n",
       "      <td>3.5</td>\n",
       "      <td>A</td>\n",
       "      <td>Active</td>\n",
       "      <td>-</td>\n",
       "      <td>GPa1family</td>\n",
       "      <td>αPa1</td>\n",
       "      <td>Agonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>OPRM</td>\n",
       "      <td>Opioid</td>\n",
       "      <td>A(Rhodopsin)</td>\n",
       "      <td>Mouse</td>\n",
       "      <td>X-ray</td>\n",
       "      <td>4DKL</td>\n",
       "      <td>2.8</td>\n",
       "      <td>A</td>\n",
       "      <td>Inactive</td>\n",
       "      <td>7</td>\n",
       "      <td>-</td>\n",
       "      <td>-</td>\n",
       "      <td>Antagonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>OXYR</td>\n",
       "      <td>Vasopressinandoxytocin</td>\n",
       "      <td>A(Rhodopsin)</td>\n",
       "      <td>Human</td>\n",
       "      <td>X-ray</td>\n",
       "      <td>6TPK</td>\n",
       "      <td>3.2</td>\n",
       "      <td>A</td>\n",
       "      <td>Inactive</td>\n",
       "      <td>24</td>\n",
       "      <td>-</td>\n",
       "      <td>-</td>\n",
       "      <td>Antagonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>CCR2</td>\n",
       "      <td>Chemokine</td>\n",
       "      <td>A(Rhodopsin)</td>\n",
       "      <td>Human</td>\n",
       "      <td>X-ray</td>\n",
       "      <td>5T1A</td>\n",
       "      <td>2.8</td>\n",
       "      <td>A</td>\n",
       "      <td>Inactive</td>\n",
       "      <td>30</td>\n",
       "      <td>-</td>\n",
       "      <td>-</td>\n",
       "      <td>AllostericantagonistAntagonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>CNR2</td>\n",
       "      <td>Cannabinoid</td>\n",
       "      <td>A(Rhodopsin)</td>\n",
       "      <td>Human</td>\n",
       "      <td>X-ray</td>\n",
       "      <td>5ZTY</td>\n",
       "      <td>2.8</td>\n",
       "      <td>A</td>\n",
       "      <td>Inactive</td>\n",
       "      <td>6</td>\n",
       "      <td>-</td>\n",
       "      <td>-</td>\n",
       "      <td>Antagonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>552</th>\n",
       "      <td>NPY2R</td>\n",
       "      <td>NeuropeptideY</td>\n",
       "      <td>A(Rhodopsin)</td>\n",
       "      <td>Human</td>\n",
       "      <td>X-ray</td>\n",
       "      <td>7DDZ</td>\n",
       "      <td>2.8</td>\n",
       "      <td>A</td>\n",
       "      <td>Inactive</td>\n",
       "      <td>1</td>\n",
       "      <td>-</td>\n",
       "      <td>-</td>\n",
       "      <td>Antagonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>553</th>\n",
       "      <td>NTR1</td>\n",
       "      <td>Neurotensin</td>\n",
       "      <td>A(Rhodopsin)</td>\n",
       "      <td>Rat</td>\n",
       "      <td>cryo-EM</td>\n",
       "      <td>7L0P</td>\n",
       "      <td>4.1</td>\n",
       "      <td>C</td>\n",
       "      <td>Active</td>\n",
       "      <td>100</td>\n",
       "      <td>Gi/o</td>\n",
       "      <td>αi1</td>\n",
       "      <td>Agonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>554</th>\n",
       "      <td>ADA2C</td>\n",
       "      <td>Adrenoceptors</td>\n",
       "      <td>A(Rhodopsin)</td>\n",
       "      <td>Human</td>\n",
       "      <td>X-ray</td>\n",
       "      <td>6KUW</td>\n",
       "      <td>2.8</td>\n",
       "      <td>A</td>\n",
       "      <td>Inactive</td>\n",
       "      <td>1</td>\n",
       "      <td>-</td>\n",
       "      <td>-</td>\n",
       "      <td>Antagonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>555</th>\n",
       "      <td>NTR1</td>\n",
       "      <td>Neurotensin</td>\n",
       "      <td>A(Rhodopsin)</td>\n",
       "      <td>Rat</td>\n",
       "      <td>cryo-EM</td>\n",
       "      <td>7L0Q</td>\n",
       "      <td>4.3</td>\n",
       "      <td>C</td>\n",
       "      <td>Active</td>\n",
       "      <td>100</td>\n",
       "      <td>Gi/o</td>\n",
       "      <td>αi1</td>\n",
       "      <td>Agonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>556</th>\n",
       "      <td>NTR1</td>\n",
       "      <td>Neurotensin</td>\n",
       "      <td>A(Rhodopsin)</td>\n",
       "      <td>Rat</td>\n",
       "      <td>cryo-EM</td>\n",
       "      <td>7L0R</td>\n",
       "      <td>4.2</td>\n",
       "      <td>C</td>\n",
       "      <td>Active</td>\n",
       "      <td>100</td>\n",
       "      <td>Gi/o</td>\n",
       "      <td>αi1</td>\n",
       "      <td>Agonist</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>557 rows × 13 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "    uniprot(gene)         receptor family                           Cl.  \\\n",
       "0            STE2                    STE2  D1(Ste2-likefungalpheromone)   \n",
       "1            OPRM                  Opioid                  A(Rhodopsin)   \n",
       "2            OXYR  Vasopressinandoxytocin                  A(Rhodopsin)   \n",
       "3            CCR2               Chemokine                  A(Rhodopsin)   \n",
       "4            CNR2             Cannabinoid                  A(Rhodopsin)   \n",
       "..            ...                     ...                           ...   \n",
       "552         NPY2R           NeuropeptideY                  A(Rhodopsin)   \n",
       "553          NTR1             Neurotensin                  A(Rhodopsin)   \n",
       "554         ADA2C           Adrenoceptors                  A(Rhodopsin)   \n",
       "555          NTR1             Neurotensin                  A(Rhodopsin)   \n",
       "556          NTR1             Neurotensin                  A(Rhodopsin)   \n",
       "\n",
       "                    Species   Method   PDB  Resolution Preferred Chain  \\\n",
       "0    strainATCC204508/S288c  cryo-EM  7AD3         3.5               A   \n",
       "1                     Mouse    X-ray  4DKL         2.8               A   \n",
       "2                     Human    X-ray  6TPK         3.2               A   \n",
       "3                     Human    X-ray  5T1A         2.8               A   \n",
       "4                     Human    X-ray  5ZTY         2.8               A   \n",
       "..                      ...      ...   ...         ...             ...   \n",
       "552                   Human    X-ray  7DDZ         2.8               A   \n",
       "553                     Rat  cryo-EM  7L0P         4.1               C   \n",
       "554                   Human    X-ray  6KUW         2.8               A   \n",
       "555                     Rat  cryo-EM  7L0Q         4.3               C   \n",
       "556                     Rat  cryo-EM  7L0R         4.2               C   \n",
       "\n",
       "        State Degree active %      Family Subtype  \\\n",
       "0      Active               -  GPa1family    αPa1   \n",
       "1    Inactive               7           -       -   \n",
       "2    Inactive              24           -       -   \n",
       "3    Inactive              30           -       -   \n",
       "4    Inactive               6           -       -   \n",
       "..        ...             ...         ...     ...   \n",
       "552  Inactive               1           -       -   \n",
       "553    Active             100        Gi/o     αi1   \n",
       "554  Inactive               1           -       -   \n",
       "555    Active             100        Gi/o     αi1   \n",
       "556    Active             100        Gi/o     αi1   \n",
       "\n",
       "                           Function  \n",
       "0                           Agonist  \n",
       "1                        Antagonist  \n",
       "2                        Antagonist  \n",
       "3    AllostericantagonistAntagonist  \n",
       "4                        Antagonist  \n",
       "..                              ...  \n",
       "552                      Antagonist  \n",
       "553                         Agonist  \n",
       "554                      Antagonist  \n",
       "555                         Agonist  \n",
       "556                         Agonist  \n",
       "\n",
       "[557 rows x 13 columns]"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "p.table"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "121fb812",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "## FILTERING, REDUCED TABLES AND PDB LISTS"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "d4252eb2",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Creating filters is easy with the processor and can primarily be done in two ways. The first is by applying the class function *p.make_filter()*:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "473c8aae",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "f_act = p.make_filter(State='Active')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "188fc883",
   "metadata": {
    "hidden": true
   },
   "source": [
    "The implemented filter options (such as filtering by states etc) with examples in brackets are:\n",
    "- Species  (Human, Bovine)\n",
    "- State  (Active, Inactive, Intermediate)\n",
    "- Cl  (Rhodopsin, A, ...)\n",
    "- Family  (Gi/o, Gs, ...)\n",
    "- Subtype  (αi1, ...)\n",
    "- Resolution  (<=3.5)\n",
    "- Function  (Agonist, Antagonist)\n",
    "- gprotein  (filters out all complexes where not both Family and Subtype are within a predifined set of regular gproteins)\n",
    "- pdb_ids  (eg. \\['7AD3', '4DKL'\\] -> creates a filter only containing the listed pdbs if present in our metainfo table) "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0faf9a99",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Filters may be \"applied\", where we effectively remove all data except the ones present in the filter (simultaneously being removed from both *p.dfl* and *p.dfl_list*."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6b236a00",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "p.apply_filter(f_act)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "47c3fcd5",
   "metadata": {
    "hidden": true
   },
   "source": [
    "The second option and this is what happens behind the scene of the *p.make_filter()* function: **QUERIES** (SQL)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "45ca647d",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "f_act = f_act[f_act['Method']=='cryo-EM']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "d5594b71",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>uniprot(gene)</th>\n",
       "      <th>receptor family</th>\n",
       "      <th>Cl.</th>\n",
       "      <th>Species</th>\n",
       "      <th>Method</th>\n",
       "      <th>PDB</th>\n",
       "      <th>Resolution</th>\n",
       "      <th>Preferred Chain</th>\n",
       "      <th>State</th>\n",
       "      <th>Degree active %</th>\n",
       "      <th>Family</th>\n",
       "      <th>Subtype</th>\n",
       "      <th>Function</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>STE2</td>\n",
       "      <td>STE2</td>\n",
       "      <td>D1(Ste2-likefungalpheromone)</td>\n",
       "      <td>strainATCC204508/S288c</td>\n",
       "      <td>cryo-EM</td>\n",
       "      <td>7AD3</td>\n",
       "      <td>3.5</td>\n",
       "      <td>A</td>\n",
       "      <td>Active</td>\n",
       "      <td>-</td>\n",
       "      <td>GPa1family</td>\n",
       "      <td>αPa1</td>\n",
       "      <td>Agonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>5HT1B</td>\n",
       "      <td>5-Hydroxytryptamine</td>\n",
       "      <td>A(Rhodopsin)</td>\n",
       "      <td>Human</td>\n",
       "      <td>cryo-EM</td>\n",
       "      <td>6G79</td>\n",
       "      <td>3.8</td>\n",
       "      <td>S</td>\n",
       "      <td>Active</td>\n",
       "      <td>100</td>\n",
       "      <td>Gi/o</td>\n",
       "      <td>αo</td>\n",
       "      <td>Agonist</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>PTH1R</td>\n",
       "      <td>Parathyroidhormone</td>\n",
       "      <td>B1(Secretin)</td>\n",
       "      <td>Human</td>\n",
       "      <td>cryo-EM</td>\n",
       "      <td>6NBI</td>\n",
       "      <td>4.0</td>\n",
       "      <td>R</td>\n",
       "      <td>Active</td>\n",
       "      <td>100</td>\n",
       "      <td>Gs</td>\n",
       "      <td>αs</td>\n",
       "      <td>Agonist</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "  uniprot(gene)      receptor family                           Cl.  \\\n",
       "0          STE2                 STE2  D1(Ste2-likefungalpheromone)   \n",
       "6         5HT1B  5-Hydroxytryptamine                  A(Rhodopsin)   \n",
       "8         PTH1R   Parathyroidhormone                  B1(Secretin)   \n",
       "\n",
       "                  Species   Method   PDB  Resolution Preferred Chain   State  \\\n",
       "0  strainATCC204508/S288c  cryo-EM  7AD3         3.5               A  Active   \n",
       "6                   Human  cryo-EM  6G79         3.8               S  Active   \n",
       "8                   Human  cryo-EM  6NBI         4.0               R  Active   \n",
       "\n",
       "  Degree active %      Family Subtype Function  \n",
       "0               -  GPa1family    αPa1  Agonist  \n",
       "6             100        Gi/o      αo  Agonist  \n",
       "8             100          Gs      αs  Agonist  "
      ]
     },
     "execution_count": 10,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "f_act.head(3)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9f63525e",
   "metadata": {
    "hidden": true
   },
   "source": [
    "So filters really are just filtered metainfotables and since they are easily interchangable / by extension also pdb lists. "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "65cb4903",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "## GPCR GENERIC RESIDUE NUMBERS"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "92f35bd5",
   "metadata": {
    "hidden": true
   },
   "source": [
    "The tricky thing about assigning generic residue numbers to structures is that many times the residue sequence of the structure is not exactly the same as that of the *correct, corresponding gene*, further a structure may include many different chains, include fused parts, be in a complex with gproteins etc"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "c100b98f",
   "metadata": {
    "hidden": true
   },
   "source": [
    "We got the **generic residue number** from [gpcrdb](https://docs.gpcrdb.org/web_services.html), now we want to allign the uniprot sequence to the structure's sequence - this is why we got the **sifts mapping** (it's a crude alignment where we have pairs of starting and end points of intervals that align of both sequences). Unfortunately the sifts is often not correct - there are often shifts or other miss-alignments withing a given sifts interval, sometimes the sifts interval do not even have equal length... In cases where this happens we do sequence alignment ourselves."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "23c819b5",
   "metadata": {
    "hidden": true
   },
   "source": [
    "The **DATAFLOW** is as follows:\n",
    "- p.table contains the \"preferred chain\" information that tells us which chain in the pdb is the gpcr.\n",
    "- p.mapping contains the sifts mapping (we check if the interval length is correct).\n",
    "- We get the uniprot sequence.\n",
    "- We use SQL to quickly produce new columns in our dataframe - very much like *label_seq_id* (which corresponds to the autogenerated sequence id of each atom (of its residue), but instead gives the position (if said atom/residue is present in the gene) within the uniprot sequence...\n",
    "- .. as well as the corresponding uniprot residue name.\n",
    "- We check, if there are (too) many non-matching residues of the alignment between structure and uniprot sequence and..\n",
    "- .. should there be too many too many non-matching residues, we, instead of using the sifts data to align structure to uniprot, do a sequence alignment.\n",
    "- Lastly, given the uniprot sequence number we can look up the generic residue numbers in p.numbering"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "da9d943a",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "p.allow_exception=False  # default"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a2383c73",
   "metadata": {
    "hidden": true
   },
   "source": [
    "If set to True, this flag makes the processor skip any structures that throw errors, may be useful if you have \"dirty data\", but don't want to go through the pain of sorting out anything that won't get processed."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2240d9e3",
   "metadata": {
    "hidden": true
   },
   "source": [
    "To start from scratch let's delete the processor (and all data loaded into memory) ... "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "618d2483",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "del p"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "8df63a86",
   "metadata": {
    "hidden": true
   },
   "source": [
    "... and reinitialize it, quickly, in 3 lines:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "e7a756c2",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "p = CifProcessor()\n",
    "p.read_pkl_metainfo()\n",
    "p.read_pkl(mode='', folder='data/raw/')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "89d93198",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Before (finally) running the generic residue assignment a quick explanation of the function inputs:\n",
    "- f: filter, this is a dataframe gained from filtering the p.table\n",
    "- pdb_ids: a list of pdbs to process (optional; an empty list means we use the filter)\n",
    "- overwrite: if set to True, after processing all structures get saved (overwriting existing files)\n",
    "- folder: location where the processed files get stored"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "dff3491d",
   "metadata": {
    "hidden": true
   },
   "source": [
    "You need to provide a filter-like input, so the processor knows which data it should assign generic residue numbers to. Imagine loading all the data from the raw folder and being interested in *Class A (Rhodopsin)* of *humans* structures only... Notably, providing a list of ***pdb_ids*** **overwrites whatever** ***filter*** you give it. If you want **NO FILTER** (i.e. process all your raw structures) just use ***f=p.table*** (not applying any filter at all)."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 43,
   "id": "15283d91",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  0%|                                                                                                                            | 0/557 [00:00<?, ?it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "5K2A P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5K2A P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  0%|▏                                                                                                                   | 1/557 [00:00<08:33,  1.08it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[167] generic residue numbers for the receptor...\n",
      "\n",
      "6KUY P08913\n",
      "Found error!\n",
      "6KUY: Using uniprot (looking up P08913) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADA2A_HUMAN\n",
      "start pdb: 1\n",
      "start uni: 35\n",
      "\n",
      "6KUY P08913\n",
      "Found error!\n",
      "6KUY: Using uniprot (looking up P08913) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADA2A_HUMAN\n",
      "start pdb: 274\n",
      "start uni: 335\n",
      "\n",
      "6KUY P08913\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  0%|▍                                                                                                                   | 2/557 [00:02<10:54,  1.18s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[336] generic residue numbers for the receptor...\n",
      "\n",
      "6M1H P41586\n",
      "Found error!\n",
      "6M1H: Using uniprot (looking up P41586) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for PACR_HUMAN\n",
      "start pdb: 6\n",
      "start uni: 23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  1%|▌                                                                                                                   | 3/557 [00:03<11:45,  1.27s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[354] generic residue numbers for the receptor...\n",
      "\n",
      "6WW2 Q13467\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6WW2 Q13467\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  1%|▊                                                                                                                   | 4/557 [00:05<15:30,  1.68s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[469] generic residue numbers for the receptor...\n",
      "\n",
      "3OE8 P61073\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3OE8 P61073\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  1%|█                                                                                                                   | 5/557 [00:08<19:45,  2.15s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[46] generic residue numbers for the receptor...\n",
      "\n",
      "5WF6 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5WF6 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  1%|█▏                                                                                                                  | 6/557 [00:09<16:06,  1.75s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[224] generic residue numbers for the receptor...\n",
      "\n",
      "4RWS P61073\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4RWS P61073\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  1%|█▍                                                                                                                  | 7/557 [00:11<14:14,  1.55s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[117] generic residue numbers for the receptor...\n",
      "\n",
      "6LPB P41586\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  1%|█▋                                                                                                                  | 8/557 [00:12<15:06,  1.65s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[347] generic residue numbers for the receptor...\n",
      "\n",
      "4N4W Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  2%|█▊                                                                                                                  | 9/557 [00:13<12:51,  1.41s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[101] generic residue numbers for the receptor...\n",
      "\n",
      "5VAI G1SGD4\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  2%|██                                                                                                                 | 10/557 [00:15<14:15,  1.56s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[215] generic residue numbers for the receptor...\n",
      "\n",
      "3AYM P31356\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  2%|██▎                                                                                                                | 11/557 [00:17<13:38,  1.50s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[30] generic residue numbers for the receptor...\n",
      "\n",
      "4PXF P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  2%|██▍                                                                                                                | 12/557 [00:17<11:23,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[109] generic residue numbers for the receptor...\n",
      "\n",
      "5T1A P41597\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5T1A P41597\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  2%|██▋                                                                                                                | 13/557 [00:18<10:45,  1.19s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[195] generic residue numbers for the receptor...\n",
      "\n",
      "5ZK8 P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5ZK8 P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  3%|██▉                                                                                                                | 14/557 [00:19<09:54,  1.10s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[248] generic residue numbers for the receptor...\n",
      "\n",
      "6KPG P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  3%|███                                                                                                                | 15/557 [00:21<12:43,  1.41s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[332] generic residue numbers for the receptor...\n",
      "\n",
      "6FJ3 Q03431\n",
      "Found error!\n",
      "6FJ3: Using uniprot (looking up Q03431) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for PTH1R_HUMAN\n",
      "start pdb: 3\n",
      "start uni: 24\n",
      "\n",
      "6FJ3 Q03431\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  3%|███▎                                                                                                               | 16/557 [00:23<12:17,  1.36s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[286] generic residue numbers for the receptor...\n",
      "\n",
      "3OE9 P61073\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3OE9 P61073\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  3%|███▌                                                                                                               | 17/557 [00:24<13:25,  1.49s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[47] generic residue numbers for the receptor...\n",
      "\n",
      "7DTV P41180\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  3%|███▋                                                                                                               | 18/557 [00:28<19:16,  2.15s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[526] generic residue numbers for the receptor...\n",
      "\n",
      "6CM4 P14416\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6CM4 P14416\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  3%|███▉                                                                                                               | 19/557 [00:29<15:54,  1.77s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[267] generic residue numbers for the receptor...\n",
      "\n",
      "5UNH P50052\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  4%|████▏                                                                                                              | 20/557 [00:30<14:17,  1.60s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[209] generic residue numbers for the receptor...\n",
      "\n",
      "7DDZ P49146\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "7DDZ P49146\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  4%|████▎                                                                                                              | 21/557 [00:31<12:46,  1.43s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[519] generic residue numbers for the receptor...\n",
      "\n",
      "4DAJ P08483\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4DAJ P08483\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  4%|████▌                                                                                                              | 22/557 [00:35<18:43,  2.10s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[79] generic residue numbers for the receptor...\n",
      "\n",
      "7DHI P07550\n",
      "Found error!\n",
      "7DHI: Using uniprot (looking up P07550) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB2_HUMAN\n",
      "start pdb: 25\n",
      "start uni: 1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  4%|████▋                                                                                                              | 23/557 [00:36<16:19,  1.83s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[522] generic residue numbers for the receptor...\n",
      "\n",
      "6PS7 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6PS7 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  4%|████▉                                                                                                              | 24/557 [00:37<13:55,  1.57s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[410] generic residue numbers for the receptor...\n",
      "\n",
      "6O3C P56726\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  4%|█████▏                                                                                                             | 25/557 [00:38<12:59,  1.46s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[379] generic residue numbers for the receptor...\n",
      "\n",
      "6WHA P28223\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  5%|█████▎                                                                                                             | 26/557 [00:40<13:18,  1.50s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[462] generic residue numbers for the receptor...\n",
      "\n",
      "3AYN P31356\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  5%|█████▌                                                                                                             | 27/557 [00:41<12:51,  1.45s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[31] generic residue numbers for the receptor...\n",
      "\n",
      "5TZY O14842\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5TZY O14842\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  5%|█████▊                                                                                                             | 28/557 [00:42<11:17,  1.28s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[202] generic residue numbers for the receptor...\n",
      "\n",
      "4AMI P07700\n",
      "Found error!\n",
      "4AMI: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  5%|█████▉                                                                                                             | 29/557 [00:43<09:59,  1.14s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[71] generic residue numbers for the receptor...\n",
      "\n",
      "7BW0 Q8TDU6\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  5%|██████▏                                                                                                            | 30/557 [00:45<11:57,  1.36s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[491] generic residue numbers for the receptor...\n",
      "\n",
      "5GLI P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5GLI P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5GLI P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  6%|██████▍                                                                                                            | 31/557 [00:46<11:39,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[159] generic residue numbers for the receptor...\n",
      "\n",
      "6WJC P11229\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6WJC P11229\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  6%|██████▌                                                                                                            | 32/557 [00:47<11:12,  1.28s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[466] generic residue numbers for the receptor...\n",
      "\n",
      "6GDG P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  6%|██████▊                                                                                                            | 33/557 [00:49<11:34,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[297] generic residue numbers for the receptor...\n",
      "\n",
      "6WHC P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  6%|███████                                                                                                            | 34/557 [00:51<13:16,  1.52s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[463] generic residue numbers for the receptor...\n",
      "\n",
      "7KH0 P30518\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  6%|███████▏                                                                                                           | 35/557 [00:53<14:29,  1.67s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[541] generic residue numbers for the receptor...\n",
      "\n",
      "6VCB P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  6%|███████▍                                                                                                           | 36/557 [00:54<15:05,  1.74s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[452] generic residue numbers for the receptor...\n",
      "\n",
      "5OLH P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5OLH P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  7%|███████▋                                                                                                           | 37/557 [00:55<12:55,  1.49s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[188] generic residue numbers for the receptor...\n",
      "\n",
      "7LCK P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  7%|███████▊                                                                                                           | 38/557 [00:56<11:19,  1.31s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[552] generic residue numbers for the receptor...\n",
      "\n",
      "5X33 Q9WTK1\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5X33 Q9WTK1\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  7%|████████                                                                                                           | 39/557 [00:58<11:26,  1.32s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[230] generic residue numbers for the receptor...\n",
      "\n",
      "6ME9 P49286\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6ME9 P49286\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  7%|████████▎                                                                                                          | 40/557 [00:59<12:05,  1.40s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[364] generic residue numbers for the receptor...\n",
      "\n",
      "4PY0 Q9H244\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4PY0 Q9H244\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  7%|████████▍                                                                                                          | 41/557 [01:00<10:54,  1.27s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[111] generic residue numbers for the receptor...\n",
      "\n",
      "6OYA P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  8%|████████▋                                                                                                          | 42/557 [01:02<11:54,  1.39s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[394] generic residue numbers for the receptor...\n",
      "\n",
      "4Z34 Q92633\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4Z34 Q92633\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  8%|████████▉                                                                                                          | 43/557 [01:03<10:39,  1.24s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[133] generic residue numbers for the receptor...\n",
      "\n",
      "6KK7 P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6KK7 P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6KK7 P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  8%|█████████                                                                                                          | 44/557 [01:05<12:55,  1.51s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[326] generic residue numbers for the receptor...\n",
      "\n",
      "6VI4 P41145\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  8%|█████████▎                                                                                                         | 45/557 [01:06<12:08,  1.42s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[453] generic residue numbers for the receptor...\n",
      "\n",
      "6X18 P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  8%|█████████▍                                                                                                         | 46/557 [01:08<14:12,  1.67s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[472] generic residue numbers for the receptor...\n",
      "\n",
      "6PS3 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6PS3 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  8%|█████████▋                                                                                                         | 47/557 [01:09<12:36,  1.48s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[406] generic residue numbers for the receptor...\n",
      "\n",
      "6KPC P34972\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6KPC P34972\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  9%|█████████▉                                                                                                         | 48/557 [01:11<12:16,  1.45s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[330] generic residue numbers for the receptor...\n",
      "\n",
      "4ZUD P30556\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  9%|██████████                                                                                                         | 49/557 [01:11<10:20,  1.22s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[139] generic residue numbers for the receptor...\n",
      "\n",
      "6TOT O43613\n",
      "Found error!\n",
      "6TOT: Using uniprot (looking up O43613) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for OX1R_HUMAN\n",
      "start pdb: 4\n",
      "start uni: 28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  9%|██████████▎                                                                                                        | 50/557 [01:12<09:20,  1.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[431] generic residue numbers for the receptor...\n",
      "\n",
      "6UUN Q16602\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  9%|██████████▌                                                                                                        | 51/557 [01:14<11:31,  1.37s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[448] generic residue numbers for the receptor...\n",
      "\n",
      "4Z9G P34998\n",
      "Found error!\n",
      "4Z9G: Using uniprot (looking up P34998) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for CRFR1_HUMAN\n",
      "start pdb: 2\n",
      "start uni: 104\n",
      "\n",
      "4Z9G P34998\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  9%|██████████▋                                                                                                        | 52/557 [01:17<14:05,  1.67s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[136] generic residue numbers for the receptor...\n",
      "\n",
      "7JOZ P21728\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 10%|██████████▉                                                                                                        | 53/557 [01:18<14:20,  1.71s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[535] generic residue numbers for the receptor...\n",
      "\n",
      "6XBL Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 10%|███████████▏                                                                                                       | 54/557 [01:21<17:29,  2.09s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[477] generic residue numbers for the receptor...\n",
      "\n",
      "2HPY P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 10%|███████████▎                                                                                                       | 55/557 [01:23<15:11,  1.82s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[6] generic residue numbers for the receptor...\n",
      "\n",
      "2YCX P07700\n",
      "Found error!\n",
      "2YCX: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 10%|███████████▌                                                                                                       | 56/557 [01:23<12:37,  1.51s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[23] generic residue numbers for the receptor...\n",
      "\n",
      "3VG9 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 10%|███████████▊                                                                                                       | 57/557 [01:25<11:45,  1.41s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[64] generic residue numbers for the receptor...\n",
      "\n",
      "5ZKQ P25105\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5ZKQ P25105\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 10%|███████████▉                                                                                                       | 58/557 [01:26<11:49,  1.42s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[252] generic residue numbers for the receptor...\n",
      "\n",
      "5XSZ Q08BG4\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5XSZ Q08BG4\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 11%|████████████▏                                                                                                      | 59/557 [01:27<10:47,  1.30s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[239] generic residue numbers for the receptor...\n",
      "\n",
      "6PS2 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6PS2 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 11%|████████████▍                                                                                                      | 60/557 [01:28<10:10,  1.23s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[405] generic residue numbers for the receptor...\n",
      "\n",
      "7L0P P20789\n",
      "Found error!\n",
      "7L0P: Using uniprot (looking up P20789) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for NTR1_RAT\n",
      "start pdb: 5\n",
      "start uni: 50\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 11%|████████████▌                                                                                                      | 61/557 [01:29<09:52,  1.19s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[544] generic residue numbers for the receptor...\n",
      "\n",
      "6Z10 Q6IYF9\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 11%|████████████▊                                                                                                      | 62/557 [01:30<08:47,  1.07s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[480] generic residue numbers for the receptor...\n",
      "\n",
      "6IGK P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6IGK P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6IGK P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 11%|█████████████                                                                                                      | 63/557 [01:31<09:38,  1.17s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[312] generic residue numbers for the receptor...\n",
      "\n",
      "5ZK3 P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5ZK3 P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 11%|█████████████▏                                                                                                     | 64/557 [01:32<08:57,  1.09s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[247] generic residue numbers for the receptor...\n",
      "\n",
      "7CZ5 Q02643\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 12%|█████████████▍                                                                                                     | 65/557 [01:34<10:34,  1.29s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[513] generic residue numbers for the receptor...\n",
      "\n",
      "2YDO P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 12%|█████████████▋                                                                                                     | 66/557 [01:35<09:47,  1.20s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[26] generic residue numbers for the receptor...\n",
      "\n",
      "5OLG P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5OLG P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 12%|█████████████▊                                                                                                     | 67/557 [01:36<09:06,  1.12s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[187] generic residue numbers for the receptor...\n",
      "\n",
      "4RWA P41143\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 12%|██████████████                                                                                                     | 68/557 [01:37<09:17,  1.14s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[115] generic residue numbers for the receptor...\n",
      "\n",
      "4Z35 Q92633\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4Z35 Q92633\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 12%|██████████████▏                                                                                                    | 69/557 [01:38<08:42,  1.07s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[134] generic residue numbers for the receptor...\n",
      "\n",
      "3RZE P35367\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3RZE P35367\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3RZE P35367\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 13%|██████████████▍                                                                                                    | 70/557 [01:39<09:04,  1.12s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[57] generic residue numbers for the receptor...\n",
      "\n",
      "6QZH P32248\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 13%|██████████████▋                                                                                                    | 71/557 [01:40<08:53,  1.10s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[417] generic residue numbers for the receptor...\n",
      "\n",
      "3EML P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3EML P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 13%|██████████████▊                                                                                                    | 72/557 [01:41<08:41,  1.08s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[37] generic residue numbers for the receptor...\n",
      "\n",
      "5VEW P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5VEW P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5VEW P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5VEW P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 13%|███████████████                                                                                                    | 73/557 [01:44<12:16,  1.52s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[217] generic residue numbers for the receptor...\n",
      "\n",
      "5L7D Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5L7D Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 13%|███████████████▎                                                                                                   | 74/557 [01:47<15:11,  1.89s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[172] generic residue numbers for the receptor...\n",
      "\n",
      "1f88 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 13%|███████████████▍                                                                                                   | 75/557 [01:48<14:10,  1.76s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[0] generic residue numbers for the receptor...\n",
      "\n",
      "6KQI P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6KQI P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 14%|███████████████▋                                                                                                   | 76/557 [01:49<12:24,  1.55s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[333] generic residue numbers for the receptor...\n",
      "\n",
      "4IAR P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4IAR P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 14%|███████████████▉                                                                                                   | 77/557 [01:50<10:45,  1.35s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[89] generic residue numbers for the receptor...\n",
      "\n",
      "6UOA O75899\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 14%|████████████████                                                                                                   | 78/557 [01:53<13:44,  1.72s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[446] generic residue numbers for the receptor...\n",
      "\n",
      "2G87 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 14%|████████████████▎                                                                                                  | 79/557 [01:54<12:27,  1.56s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[5] generic residue numbers for the receptor...\n",
      "\n",
      "6ZDR P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6ZDR P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 14%|████████████████▌                                                                                                  | 80/557 [01:55<10:53,  1.37s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[481] generic residue numbers for the receptor...\n",
      "\n",
      "5UNF P50052\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 15%|████████████████▋                                                                                                  | 81/557 [01:56<10:33,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[207] generic residue numbers for the receptor...\n",
      "\n",
      "6RZ7 Q9NS75\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6RZ7 Q9NS75\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 15%|████████████████▉                                                                                                  | 82/557 [01:57<09:22,  1.18s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[422] generic residue numbers for the receptor...\n",
      "\n",
      "6E59 P25103\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6E59 P25103\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 15%|█████████████████▏                                                                                                 | 83/557 [01:58<09:14,  1.17s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[282] generic residue numbers for the receptor...\n",
      "\n",
      "6TO7 O43613\n",
      "Found error!\n",
      "6TO7: Using uniprot (looking up O43613) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for OX1R_HUMAN\n",
      "start pdb: 4\n",
      "start uni: 28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 15%|█████████████████▎                                                                                                 | 84/557 [01:59<08:25,  1.07s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[428] generic residue numbers for the receptor...\n",
      "\n",
      "5JTB P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5JTB P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 15%|█████████████████▌                                                                                                 | 85/557 [02:00<08:02,  1.02s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[166] generic residue numbers for the receptor...\n",
      "\n",
      "5V56 Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5V56 Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 15%|█████████████████▊                                                                                                 | 86/557 [02:03<12:15,  1.56s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[213] generic residue numbers for the receptor...\n",
      "\n",
      "6CMO P08100\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 16%|█████████████████▉                                                                                                 | 87/557 [02:05<14:41,  1.88s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[268] generic residue numbers for the receptor...\n",
      "\n",
      "6RZ6 Q9NS75\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6RZ6 Q9NS75\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 16%|██████████████████▏                                                                                                | 88/557 [02:06<12:15,  1.57s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[421] generic residue numbers for the receptor...\n",
      "\n",
      "2I35 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 16%|██████████████████▍                                                                                                | 89/557 [02:07<10:09,  1.30s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[7] generic residue numbers for the receptor...\n",
      "\n",
      "6WI9 P47872\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 16%|██████████████████▌                                                                                                | 90/557 [02:09<11:25,  1.47s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[464] generic residue numbers for the receptor...\n",
      "\n",
      "6PT3 P41143\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 16%|██████████████████▊                                                                                                | 91/557 [02:10<10:39,  1.37s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[414] generic residue numbers for the receptor...\n",
      "\n",
      "5MZP P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5MZP P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 17%|██████████████████▉                                                                                                | 92/557 [02:11<09:35,  1.24s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[176] generic residue numbers for the receptor...\n",
      "\n",
      "6BQG P28335\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6BQG P28335\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 17%|███████████████████▏                                                                                               | 93/557 [02:12<08:46,  1.13s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[263] generic residue numbers for the receptor...\n",
      "\n",
      "2Z73 P31356\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 17%|███████████████████▍                                                                                               | 94/557 [02:13<09:16,  1.20s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[28] generic residue numbers for the receptor...\n",
      "\n",
      "7CMU P35462\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 17%|███████████████████▌                                                                                               | 95/557 [02:15<10:47,  1.40s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[506] generic residue numbers for the receptor...\n",
      "\n",
      "4XEE P20789\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 17%|███████████████████▊                                                                                               | 96/557 [02:16<09:37,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[126] generic residue numbers for the receptor...\n",
      "\n",
      "7MBX P32238\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 17%|████████████████████                                                                                               | 97/557 [02:17<10:57,  1.43s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[555] generic residue numbers for the receptor...\n",
      "\n",
      "3ZEV P20789\n",
      "Found error!\n",
      "3ZEV: Using uniprot (looking up P20789) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for NTR1_RAT\n",
      "start pdb: 5\n",
      "start uni: 50\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 18%|████████████████████▏                                                                                              | 98/557 [02:19<10:27,  1.37s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[67] generic residue numbers for the receptor...\n",
      "\n",
      "6HLP P25103\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6HLP P25103\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 18%|████████████████████▍                                                                                              | 99/557 [02:20<09:51,  1.29s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[308] generic residue numbers for the receptor...\n",
      "\n",
      "4XNV P47900\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4XNV P47900\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 18%|████████████████████▍                                                                                             | 100/557 [02:21<08:55,  1.17s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[128] generic residue numbers for the receptor...\n",
      "\n",
      "6H7J P07700\n",
      "Found error!\n",
      "6H7J: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 6\n",
      "start uni: 44\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 18%|████████████████████▋                                                                                             | 101/557 [02:22<09:00,  1.19s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[301] generic residue numbers for the receptor...\n",
      "\n",
      "4EA3 P41146\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 18%|████████████████████▉                                                                                             | 102/557 [02:23<08:40,  1.14s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[82] generic residue numbers for the receptor...\n",
      "\n",
      "6M9T P43115\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6M9T P43115\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 18%|█████████████████████                                                                                             | 103/557 [02:24<08:25,  1.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[356] generic residue numbers for the receptor...\n",
      "\n",
      "6P9X P34998\n",
      "Found error!\n",
      "6P9X: Using uniprot (looking up P34998) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for CRFR1_HUMAN\n",
      "start pdb: 16\n",
      "start uni: 23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 19%|█████████████████████▎                                                                                            | 104/557 [02:25<08:42,  1.15s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[395] generic residue numbers for the receptor...\n",
      "\n",
      "7JVQ P21728\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 19%|█████████████████████▍                                                                                            | 105/557 [02:27<10:15,  1.36s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[538] generic residue numbers for the receptor...\n",
      "\n",
      "5IU4 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5IU4 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 19%|█████████████████████▋                                                                                            | 106/557 [02:28<09:16,  1.23s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[160] generic residue numbers for the receptor...\n",
      "\n",
      "4NTJ Q9H244\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4NTJ Q9H244\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 19%|█████████████████████▉                                                                                            | 107/557 [02:29<08:32,  1.14s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Error Status: False\n",
      "assigned dfl[104] generic residue numbers for the receptor...\n",
      "\n",
      "6KPF P34972\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 19%|██████████████████████                                                                                            | 108/557 [02:31<09:55,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[331] generic residue numbers for the receptor...\n",
      "\n",
      "5OLO P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5OLO P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 20%|██████████████████████▎                                                                                           | 109/557 [02:32<09:46,  1.31s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[189] generic residue numbers for the receptor...\n",
      "\n",
      "3V2Y P21453\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3V2Y P21453\n",
      "Found error!\n",
      "3V2Y: Using uniprot (looking up P21453) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for S1PR1_HUMAN\n",
      "start pdb: 410\n",
      "start uni: 244\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 20%|██████████████████████▌                                                                                           | 110/557 [02:33<09:20,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[63] generic residue numbers for the receptor...\n",
      "\n",
      "6W2X O75899\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 20%|██████████████████████▋                                                                                           | 111/557 [02:36<13:13,  1.78s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[458] generic residue numbers for the receptor...\n",
      "\n",
      "6TKO P07700\n",
      "Found error!\n",
      "6TKO: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 1\n",
      "start uni: 32\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 20%|██████████████████████▉                                                                                           | 112/557 [02:37<12:00,  1.62s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[427] generic residue numbers for the receptor...\n",
      "\n",
      "5T04 P20789\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5T04 P20789\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 20%|███████████████████████▏                                                                                          | 113/557 [02:38<10:47,  1.46s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[194] generic residue numbers for the receptor...\n",
      "\n",
      "6KO5 Q92847\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 20%|███████████████████████▎                                                                                          | 114/557 [02:40<10:27,  1.42s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[328] generic residue numbers for the receptor...\n",
      "\n",
      "6OS9 P30989\n",
      "Found error!\n",
      "6OS9: Using uniprot (looking up P30989) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for NTR1_HUMAN\n",
      "start pdb: 40\n",
      "start uni: 20\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 21%|███████████████████████▌                                                                                          | 115/557 [02:41<10:18,  1.40s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[390] generic residue numbers for the receptor...\n",
      "\n",
      "6K41 P18089\n",
      "Found error!\n",
      "6K41: Using uniprot (looking up P18089) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADA2B_HUMAN\n",
      "start pdb: 208\n",
      "start uni: 9\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 21%|███████████████████████▋                                                                                          | 116/557 [02:42<10:05,  1.37s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[322] generic residue numbers for the receptor...\n",
      "\n",
      "6OFJ P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 21%|███████████████████████▉                                                                                          | 117/557 [02:45<12:21,  1.69s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[381] generic residue numbers for the receptor...\n",
      "\n",
      "4X1H P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 21%|████████████████████████▏                                                                                         | 118/557 [02:46<10:13,  1.40s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[125] generic residue numbers for the receptor...\n",
      "\n",
      "4J4Q P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 21%|████████████████████████▎                                                                                         | 119/557 [02:46<08:40,  1.19s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[91] generic residue numbers for the receptor...\n",
      "\n",
      "7CKX P21728\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 22%|████████████████████████▌                                                                                         | 120/557 [02:48<10:01,  1.38s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[503] generic residue numbers for the receptor...\n",
      "\n",
      "4BUO P20789\n",
      "Found error!\n",
      "4BUO: Using uniprot (looking up P20789) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for NTR1_RAT\n",
      "start pdb: 4\n",
      "start uni: 50\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 22%|████████████████████████▊                                                                                         | 121/557 [02:49<08:53,  1.22s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[75] generic residue numbers for the receptor...\n",
      "\n",
      "2YCZ P07700\n",
      "Found error!\n",
      "2YCZ: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 22%|████████████████████████▉                                                                                         | 122/557 [02:50<08:01,  1.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[25] generic residue numbers for the receptor...\n",
      "\n",
      "6N52 P41594\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 22%|█████████████████████████▏                                                                                        | 123/557 [02:53<12:30,  1.73s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[372] generic residue numbers for the receptor...\n",
      "\n",
      "2ZIY P31356\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 22%|█████████████████████████▍                                                                                        | 124/557 [02:54<10:23,  1.44s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[29] generic residue numbers for the receptor...\n",
      "\n",
      "7KNU Q16602\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 22%|█████████████████████████▌                                                                                        | 125/557 [02:55<09:26,  1.31s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[543] generic residue numbers for the receptor...\n",
      "\n",
      "5CGC P41594\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5CGC P41594\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 23%|█████████████████████████▊                                                                                        | 126/557 [02:56<08:33,  1.19s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[143] generic residue numbers for the receptor...\n",
      "\n",
      "6XBK Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 23%|█████████████████████████▉                                                                                        | 127/557 [02:59<12:20,  1.72s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[476] generic residue numbers for the receptor...\n",
      "\n",
      "6N48 P07550\n",
      "Found error!\n",
      "6N48: Using uniprot (looking up P07550) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB2_HUMAN\n",
      "start pdb: 178\n",
      "start uni: 29\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 23%|██████████████████████████▏                                                                                       | 128/557 [02:59<10:19,  1.44s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[369] generic residue numbers for the receptor...\n",
      "\n",
      "6I9K B1B1U5\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 23%|██████████████████████████▍                                                                                       | 129/557 [03:00<08:39,  1.21s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[309] generic residue numbers for the receptor...\n",
      "\n",
      "6IGL P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6IGL P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6IGL P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 23%|██████████████████████████▌                                                                                       | 130/557 [03:01<09:00,  1.27s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[313] generic residue numbers for the receptor...\n",
      "\n",
      "5DYS P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 24%|██████████████████████████▊                                                                                       | 131/557 [03:02<07:45,  1.09s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[153] generic residue numbers for the receptor...\n",
      "\n",
      "5ZBH P25929\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5ZBH P25929\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 24%|███████████████████████████                                                                                       | 132/557 [03:03<07:47,  1.10s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[244] generic residue numbers for the receptor...\n",
      "\n",
      "6D26 Q9Y5Y4\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6D26 Q9Y5Y4\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6D26 Q9Y5Y4\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 24%|███████████████████████████▏                                                                                      | 133/557 [03:05<08:08,  1.15s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[269] generic residue numbers for the receptor...\n",
      "\n",
      "5A8E P07700\n",
      "Found error!\n",
      "5A8E: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 24%|███████████████████████████▍                                                                                      | 134/557 [03:05<06:48,  1.04it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[141] generic residue numbers for the receptor...\n",
      "\n",
      "6J21 P25103\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6J21 P25103\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 24%|███████████████████████████▋                                                                                      | 135/557 [03:06<06:45,  1.04it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[318] generic residue numbers for the receptor...\n",
      "\n",
      "6D9H P30542\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 24%|███████████████████████████▊                                                                                      | 136/557 [03:07<07:43,  1.10s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[273] generic residue numbers for the receptor...\n",
      "\n",
      "6DDE P42866\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6DDE P42866\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 25%|████████████████████████████                                                                                      | 137/557 [03:10<10:06,  1.44s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[274] generic residue numbers for the receptor...\n",
      "\n",
      "4MQT P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4MQT P08172\n",
      "Found error!\n",
      "4MQT: Using uniprot (looking up P08172) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ACM2_HUMAN\n",
      "start pdb: 36\n",
      "start uni: 19\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 25%|████████████████████████████▏                                                                                     | 138/557 [03:11<08:46,  1.26s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[100] generic residue numbers for the receptor...\n",
      "\n",
      "4EIY P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4EIY P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 25%|████████████████████████████▍                                                                                     | 139/557 [03:11<08:03,  1.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[83] generic residue numbers for the receptor...\n",
      "\n",
      "6FKA P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 25%|████████████████████████████▋                                                                                     | 140/557 [03:12<07:03,  1.02s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[291] generic residue numbers for the receptor...\n",
      "\n",
      "5TGZ P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5TGZ P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 25%|████████████████████████████▊                                                                                     | 141/557 [03:13<06:59,  1.01s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[198] generic residue numbers for the receptor...\n",
      "\n",
      "5UZ7 P30988\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 25%|█████████████████████████████                                                                                     | 142/557 [03:15<09:15,  1.34s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[211] generic residue numbers for the receptor...\n",
      "\n",
      "6VMS P14416\n",
      "Found error!\n",
      "6VMS: Using uniprot (looking up P14416) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for DRD2_HUMAN\n",
      "start pdb: 170\n",
      "start uni: 29\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 26%|█████████████████████████████▎                                                                                    | 143/557 [03:17<09:17,  1.35s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[455] generic residue numbers for the receptor...\n",
      "\n",
      "6PS5 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6PS5 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 26%|█████████████████████████████▍                                                                                    | 144/557 [03:18<08:35,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[408] generic residue numbers for the receptor...\n",
      "\n",
      "6GPS P41597\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6GPS P41597\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 26%|█████████████████████████████▋                                                                                    | 145/557 [03:19<09:06,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[298] generic residue numbers for the receptor...\n",
      "\n",
      "5D5A P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5D5A P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 26%|█████████████████████████████▉                                                                                    | 146/557 [03:20<08:34,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[146] generic residue numbers for the receptor...\n",
      "\n",
      "6XBM Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 26%|██████████████████████████████                                                                                    | 147/557 [03:23<11:00,  1.61s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[478] generic residue numbers for the receptor...\n",
      "\n",
      "7CRH P21728\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 27%|██████████████████████████████▎                                                                                   | 148/557 [03:24<11:25,  1.68s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[508] generic residue numbers for the receptor...\n",
      "\n",
      "4ZWJ P08100\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 27%|██████████████████████████████▍                                                                                   | 149/557 [03:29<17:54,  2.63s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[140] generic residue numbers for the receptor...\n",
      "\n",
      "4QKX P07550\n",
      "Found error!\n",
      "4QKX: Using uniprot (looking up P07550) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB2_HUMAN\n",
      "start pdb: 178\n",
      "start uni: 29\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 27%|██████████████████████████████▋                                                                                   | 150/557 [03:30<14:06,  2.08s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[114] generic residue numbers for the receptor...\n",
      "\n",
      "5X93 P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5X93 P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 27%|██████████████████████████████▉                                                                                   | 151/557 [03:31<11:56,  1.76s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[232] generic residue numbers for the receptor...\n",
      "\n",
      "5EE7 P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5EE7 P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 27%|███████████████████████████████                                                                                   | 152/557 [03:32<10:17,  1.52s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[154] generic residue numbers for the receptor...\n",
      "\n",
      "5TVN P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5TVN P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5TVN P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 27%|███████████████████████████████▎                                                                                  | 153/557 [03:33<09:22,  1.39s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[200] generic residue numbers for the receptor...\n",
      "\n",
      "2X72 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 28%|███████████████████████████████▌                                                                                  | 154/557 [03:34<07:57,  1.18s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[16] generic residue numbers for the receptor...\n",
      "\n",
      "2YCY P07700\n",
      "Found error!\n",
      "2YCY: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 28%|███████████████████████████████▋                                                                                  | 155/557 [03:35<07:12,  1.08s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[24] generic residue numbers for the receptor...\n",
      "\n",
      "5GLH P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5GLH P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 28%|███████████████████████████████▉                                                                                  | 156/557 [03:36<07:14,  1.08s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[158] generic residue numbers for the receptor...\n",
      "\n",
      "7C61 P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "7C61 P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 28%|████████████████████████████████▏                                                                                 | 157/557 [03:38<08:58,  1.35s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[494] generic residue numbers for the receptor...\n",
      "\n",
      "3ZPQ P07700\n",
      "Found error!\n",
      "3ZPQ: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 28%|████████████████████████████████▎                                                                                 | 158/557 [03:39<07:52,  1.18s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[68] generic residue numbers for the receptor...\n",
      "\n",
      "6UUS Q16602\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 29%|████████████████████████████████▌                                                                                 | 159/557 [03:41<09:31,  1.44s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[449] generic residue numbers for the receptor...\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 29%|████████████████████████████████▋                                                                                 | 160/557 [03:41<07:14,  1.09s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[271] generic residue numbers for the receptor...\n",
      "\n",
      "2YCW P07700\n",
      "Found error!\n",
      "2YCW: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 29%|████████████████████████████████▉                                                                                 | 161/557 [03:42<06:42,  1.02s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[22] generic residue numbers for the receptor...\n",
      "\n",
      "6ME4 P48039\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6ME4 P48039\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 29%|█████████████████████████████████▏                                                                                | 162/557 [03:43<06:48,  1.03s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[359] generic residue numbers for the receptor...\n",
      "\n",
      "6B73 P41145\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 29%|█████████████████████████████████▎                                                                                | 163/557 [03:44<07:08,  1.09s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[261] generic residue numbers for the receptor...\n",
      "\n",
      "5UIW P51681\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5UIW P51681\n",
      "Found error!\n",
      "5UIW: Using uniprot (looking up P51681) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for CCR5_HUMAN\n",
      "start pdb: 280\n",
      "start uni: 227\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 29%|█████████████████████████████████▌                                                                                | 164/557 [03:45<07:04,  1.08s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[206] generic residue numbers for the receptor...\n",
      "\n",
      "4BWB P20789\n",
      "Found error!\n",
      "4BWB: Using uniprot (looking up P20789) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for NTR1_RAT\n",
      "start pdb: 5\n",
      "start uni: 50\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 30%|█████████████████████████████████▊                                                                                | 165/557 [03:46<06:36,  1.01s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[78] generic residue numbers for the receptor...\n",
      "\n",
      "2Y03 P07700\n",
      "Found error!\n",
      "2Y03: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 30%|█████████████████████████████████▉                                                                                | 166/557 [03:47<06:12,  1.05it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[20] generic residue numbers for the receptor...\n",
      "\n",
      "6BQH P28335\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6BQH P28335\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 30%|██████████████████████████████████▏                                                                               | 167/557 [03:48<05:55,  1.10it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[264] generic residue numbers for the receptor...\n",
      "\n",
      "6PT2 P41143\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 30%|██████████████████████████████████▍                                                                               | 168/557 [03:49<06:24,  1.01it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[413] generic residue numbers for the receptor...\n",
      "\n",
      "7BU6 P08588\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 30%|██████████████████████████████████▌                                                                               | 169/557 [03:50<07:05,  1.10s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[488] generic residue numbers for the receptor...\n",
      "\n",
      "6LFL P25025\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6LFL P25025\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 31%|██████████████████████████████████▊                                                                               | 170/557 [03:51<07:00,  1.09s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[337] generic residue numbers for the receptor...\n",
      "\n",
      "6WQA P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6WQA P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 31%|██████████████████████████████████▉                                                                               | 171/557 [03:52<06:37,  1.03s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[468] generic residue numbers for the receptor...\n",
      "\n",
      "7MBY P32238\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 31%|███████████████████████████████████▏                                                                              | 172/557 [03:54<07:39,  1.19s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[556] generic residue numbers for the receptor...\n",
      "\n",
      "5IU8 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5IU8 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 31%|███████████████████████████████████▍                                                                              | 173/557 [03:55<07:07,  1.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[162] generic residue numbers for the receptor...\n",
      "\n",
      "3PBL P35462\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3PBL P35462\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 31%|███████████████████████████████████▌                                                                              | 174/557 [03:56<08:18,  1.30s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[49] generic residue numbers for the receptor...\n",
      "\n",
      "4S0V O43614\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4S0V O43614\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 31%|███████████████████████████████████▊                                                                              | 175/557 [03:57<07:57,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[118] generic residue numbers for the receptor...\n",
      "\n",
      "6KUX P08913\n",
      "Found error!\n",
      "6KUX: Using uniprot (looking up P08913) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADA2A_HUMAN\n",
      "start pdb: 1\n",
      "start uni: 35\n",
      "\n",
      "6KUX P08913\n",
      "Found error!\n",
      "6KUX: Using uniprot (looking up P08913) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADA2A_HUMAN\n",
      "start pdb: 274\n",
      "start uni: 335\n",
      "\n",
      "6KUX P08913\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 32%|████████████████████████████████████                                                                              | 176/557 [03:59<07:55,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[335] generic residue numbers for the receptor...\n",
      "\n",
      "7DTW P41180\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 32%|████████████████████████████████████▏                                                                             | 177/557 [04:02<12:00,  1.90s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[527] generic residue numbers for the receptor...\n",
      "\n",
      "6HLO P25103\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6HLO P25103\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 32%|████████████████████████████████████▍                                                                             | 178/557 [04:03<10:29,  1.66s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[307] generic residue numbers for the receptor...\n",
      "\n",
      "7BR3 P30968\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "7BR3 P30968\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 32%|████████████████████████████████████▋                                                                             | 179/557 [04:04<09:20,  1.48s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[486] generic residue numbers for the receptor...\n",
      "\n",
      "6RZ9 Q9NS75\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6RZ9 Q9NS75\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 32%|████████████████████████████████████▊                                                                             | 180/557 [04:05<08:05,  1.29s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[424] generic residue numbers for the receptor...\n",
      "\n",
      "6TOD O43613\n",
      "Found error!\n",
      "6TOD: Using uniprot (looking up O43613) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for OX1R_HUMAN\n",
      "start pdb: 4\n",
      "start uni: 28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 32%|█████████████████████████████████████                                                                             | 181/557 [04:06<07:16,  1.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[429] generic residue numbers for the receptor...\n",
      "\n",
      "5YQZ P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5YQZ P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 33%|█████████████████████████████████████▏                                                                            | 182/557 [04:08<08:20,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[242] generic residue numbers for the receptor...\n",
      "\n",
      "5VEX P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5VEX P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5VEX P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5VEX P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 33%|█████████████████████████████████████▍                                                                            | 183/557 [04:10<10:34,  1.70s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[218] generic residue numbers for the receptor...\n",
      "\n",
      "2RH1 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "2RH1 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 33%|█████████████████████████████████████▋                                                                            | 184/557 [04:11<09:21,  1.50s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[14] generic residue numbers for the receptor...\n",
      "\n",
      "7CKZ P21728\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 33%|█████████████████████████████████████▊                                                                            | 185/557 [04:13<09:54,  1.60s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[505] generic residue numbers for the receptor...\n",
      "\n",
      "4XNW P47900\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4XNW P47900\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 33%|██████████████████████████████████████                                                                            | 186/557 [04:15<09:52,  1.60s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[129] generic residue numbers for the receptor...\n",
      "\n",
      "6LML P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 34%|██████████████████████████████████████▎                                                                           | 187/557 [04:17<10:33,  1.71s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[345] generic residue numbers for the receptor...\n",
      "\n",
      "5WIU P21917\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5WIU P21917\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 34%|██████████████████████████████████████▍                                                                           | 188/557 [04:18<08:57,  1.46s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[225] generic residue numbers for the receptor...\n",
      "\n",
      "6AKX P51681\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6AKX P51681\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 34%|██████████████████████████████████████▋                                                                           | 189/557 [04:19<08:59,  1.47s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[257] generic residue numbers for the receptor...\n",
      "\n",
      "7D77 Q86Y34\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 34%|██████████████████████████████████████▉                                                                           | 190/557 [04:22<10:44,  1.76s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[517] generic residue numbers for the receptor...\n",
      "\n",
      "5TE5 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 34%|███████████████████████████████████████                                                                           | 191/557 [04:22<08:48,  1.44s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[197] generic residue numbers for the receptor...\n",
      "\n",
      "7CX2 P43116\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 34%|███████████████████████████████████████▎                                                                          | 192/557 [04:24<09:07,  1.50s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[510] generic residue numbers for the receptor...\n",
      "\n",
      "3OAX P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 35%|███████████████████████████████████████▌                                                                          | 193/557 [04:25<08:32,  1.41s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[42] generic residue numbers for the receptor...\n",
      "\n",
      "6WIV O75899\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 35%|███████████████████████████████████████▋                                                                          | 194/557 [04:28<11:39,  1.93s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[465] generic residue numbers for the receptor...\n",
      "\n",
      "6ME5 P48039\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6ME5 P48039\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 35%|███████████████████████████████████████▉                                                                          | 195/557 [04:29<10:03,  1.67s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[360] generic residue numbers for the receptor...\n",
      "\n",
      "7L1V O43614\n",
      "Found error!\n",
      "7L1V: Using uniprot (looking up O43614) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for OX2R_HUMAN\n",
      "start pdb: 9\n",
      "start uni: 3\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 35%|████████████████████████████████████████                                                                          | 196/557 [04:31<09:34,  1.59s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[549] generic residue numbers for the receptor...\n",
      "\n",
      "7LCJ P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 35%|████████████████████████████████████████▎                                                                         | 197/557 [04:32<08:17,  1.38s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[551] generic residue numbers for the receptor...\n",
      "\n",
      "6OIJ P11229\n",
      "Found error!\n",
      "6OIJ: Using uniprot (looking up P11229) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ACM1_HUMAN\n",
      "start pdb: 11\n",
      "start uni: 2\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 36%|████████████████████████████████████████▌                                                                         | 198/557 [04:33<08:51,  1.48s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[382] generic residue numbers for the receptor...\n",
      "\n",
      "4NC3 P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4NC3 P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 36%|████████████████████████████████████████▋                                                                         | 199/557 [04:34<07:43,  1.30s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[103] generic residue numbers for the receptor...\n",
      "\n",
      "6VJM O75899\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 36%|████████████████████████████████████████▉                                                                         | 200/557 [04:37<10:38,  1.79s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[454] generic residue numbers for the receptor...\n",
      "\n",
      "4PHU O14842\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4PHU O14842\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 36%|█████████████████████████████████████████▏                                                                        | 201/557 [04:38<09:08,  1.54s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[108] generic residue numbers for the receptor...\n",
      "\n",
      "4Z36 Q92633\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4Z36 Q92633\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 36%|█████████████████████████████████████████▎                                                                        | 202/557 [04:39<07:59,  1.35s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[135] generic residue numbers for the receptor...\n",
      "\n",
      "6NIY P30988\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 36%|█████████████████████████████████████████▌                                                                        | 203/557 [04:41<08:46,  1.49s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[377] generic residue numbers for the receptor...\n",
      "\n",
      "2VT4 P07700\n",
      "Found error!\n",
      "2VT4: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 1\n",
      "start uni: 31\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 37%|█████████████████████████████████████████▊                                                                        | 204/557 [04:42<08:25,  1.43s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[15] generic residue numbers for the receptor...\n",
      "\n",
      "6ME2 P48039\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6ME2 P48039\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 37%|█████████████████████████████████████████▉                                                                        | 205/557 [04:43<07:45,  1.32s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[357] generic residue numbers for the receptor...\n",
      "\n",
      "6IBB Q6IYF9\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 37%|██████████████████████████████████████████▏                                                                       | 206/557 [04:44<07:46,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[310] generic residue numbers for the receptor...\n",
      "\n",
      "6KNM P35414\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6KNM P35414\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 37%|██████████████████████████████████████████▎                                                                       | 207/557 [04:46<07:18,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[327] generic residue numbers for the receptor...\n",
      "\n",
      "4GRV P20789\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4GRV P20789\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 37%|██████████████████████████████████████████▌                                                                       | 208/557 [04:47<06:54,  1.19s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[87] generic residue numbers for the receptor...\n",
      "\n",
      "5DGY P08100\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 38%|██████████████████████████████████████████▊                                                                       | 209/557 [04:51<13:20,  2.30s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[149] generic residue numbers for the receptor...\n",
      "\n",
      "5UNG P50052\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 38%|██████████████████████████████████████████▉                                                                       | 210/557 [04:52<10:35,  1.83s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[208] generic residue numbers for the receptor...\n",
      "\n",
      "6WH4 P28223\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6WH4 P28223\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 38%|███████████████████████████████████████████▏                                                                      | 211/557 [04:54<10:52,  1.89s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[461] generic residue numbers for the receptor...\n",
      "\n",
      "5N2S P30542\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 38%|███████████████████████████████████████████▍                                                                      | 212/557 [04:55<08:49,  1.54s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[178] generic residue numbers for the receptor...\n",
      "\n",
      "5YWY P35408\n",
      "Found error!\n",
      "5YWY: Using uniprot (looking up P35408) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for PE2R4_HUMAN\n",
      "start pdb: 4\n",
      "start uni: 4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 38%|███████████████████████████████████████████▌                                                                      | 213/557 [04:56<07:45,  1.35s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[243] generic residue numbers for the receptor...\n",
      "\n",
      "5CXV P11229\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5CXV P11229\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5CXV P11229\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 38%|███████████████████████████████████████████▊                                                                      | 214/557 [04:57<07:35,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[145] generic residue numbers for the receptor...\n",
      "\n",
      "5EN0 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 39%|████████████████████████████████████████████                                                                      | 215/557 [04:58<07:05,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[155] generic residue numbers for the receptor...\n",
      "\n",
      "6BD4 Q9ULV1\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6BD4 Q9ULV1\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 39%|████████████████████████████████████████████▏                                                                     | 216/557 [04:59<06:31,  1.15s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[262] generic residue numbers for the receptor...\n",
      "\n",
      "6JOD P50052\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 39%|████████████████████████████████████████████▍                                                                     | 217/557 [05:00<06:44,  1.19s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[319] generic residue numbers for the receptor...\n",
      "\n",
      "6X19 P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 39%|████████████████████████████████████████████▌                                                                     | 218/557 [05:03<08:22,  1.48s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[473] generic residue numbers for the receptor...\n",
      "\n",
      "7LJD P21728\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 39%|████████████████████████████████████████████▊                                                                     | 219/557 [05:04<08:58,  1.59s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[554] generic residue numbers for the receptor...\n",
      "\n",
      "6DRY P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6DRY P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 39%|█████████████████████████████████████████████                                                                     | 220/557 [05:05<07:45,  1.38s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[278] generic residue numbers for the receptor...\n",
      "\n",
      "6TOS O43613\n",
      "Found error!\n",
      "6TOS: Using uniprot (looking up O43613) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for OX1R_HUMAN\n",
      "start pdb: 4\n",
      "start uni: 28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 40%|█████████████████████████████████████████████▏                                                                    | 221/557 [05:06<06:51,  1.22s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[430] generic residue numbers for the receptor...\n",
      "\n",
      "5XJM P50052\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5XJM P50052\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 40%|█████████████████████████████████████████████▍                                                                    | 222/557 [05:08<07:35,  1.36s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[235] generic residue numbers for the receptor...\n",
      "\n",
      "6LFO P25025\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 40%|█████████████████████████████████████████████▋                                                                    | 223/557 [05:10<09:06,  1.64s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[339] generic residue numbers for the receptor...\n",
      "\n",
      "5NM2 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5NM2 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 40%|█████████████████████████████████████████████▊                                                                    | 224/557 [05:12<09:10,  1.65s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[183] generic residue numbers for the receptor...\n",
      "\n",
      "6DRZ P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6DRZ P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 40%|██████████████████████████████████████████████                                                                    | 225/557 [05:13<07:50,  1.42s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[279] generic residue numbers for the receptor...\n",
      "\n",
      "6LN2 P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6LN2 P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 41%|██████████████████████████████████████████████▎                                                                   | 226/557 [05:15<08:33,  1.55s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[346] generic residue numbers for the receptor...\n",
      "\n",
      "6DS0 P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6DS0 P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 41%|██████████████████████████████████████████████▍                                                                   | 227/557 [05:15<07:22,  1.34s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[280] generic residue numbers for the receptor...\n",
      "\n",
      "6QNO P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 41%|██████████████████████████████████████████████▋                                                                   | 228/557 [05:17<08:29,  1.55s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[416] generic residue numbers for the receptor...\n",
      "\n",
      "7BVQ P08588\n",
      "Found error!\n",
      "7BVQ: Using uniprot (looking up P08588) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_HUMAN\n",
      "start pdb: 171\n",
      "start uni: 54\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 41%|██████████████████████████████████████████████▊                                                                   | 229/557 [05:19<07:44,  1.42s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[490] generic residue numbers for the receptor...\n",
      "\n",
      "6OL9 P08912\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6OL9 P08912\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 41%|███████████████████████████████████████████████                                                                   | 230/557 [05:19<06:56,  1.27s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[384] generic residue numbers for the receptor...\n",
      "\n",
      "6K1Q P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6K1Q P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 41%|███████████████████████████████████████████████▎                                                                  | 231/557 [05:20<06:28,  1.19s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[321] generic residue numbers for the receptor...\n",
      "\n",
      "2I36 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 42%|███████████████████████████████████████████████▍                                                                  | 232/557 [05:22<07:38,  1.41s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[8] generic residue numbers for the receptor...\n",
      "\n",
      "5YHL P35408\n",
      "Found error!\n",
      "5YHL: Using uniprot (looking up P35408) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for PE2R4_HUMAN\n",
      "start pdb: 4\n",
      "start uni: 4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 42%|███████████████████████████████████████████████▋                                                                  | 233/557 [05:23<06:49,  1.27s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[241] generic residue numbers for the receptor...\n",
      "\n",
      "6PS4 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6PS4 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 42%|███████████████████████████████████████████████▉                                                                  | 234/557 [05:24<06:26,  1.20s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[407] generic residue numbers for the receptor...\n",
      "\n",
      "6TP3 O43613\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 42%|████████████████████████████████████████████████                                                                  | 235/557 [05:25<06:16,  1.17s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[432] generic residue numbers for the receptor...\n",
      "\n",
      "6OS0 P30556\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6OS0 P30556\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 42%|████████████████████████████████████████████████▎                                                                 | 236/557 [05:27<06:15,  1.17s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[387] generic residue numbers for the receptor...\n",
      "\n",
      "6OT0 Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 43%|████████████████████████████████████████████████▌                                                                 | 237/557 [05:29<08:21,  1.57s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[392] generic residue numbers for the receptor...\n",
      "\n",
      "5WS3 O43614\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5WS3 O43614\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 43%|████████████████████████████████████████████████▋                                                                 | 238/557 [05:30<07:44,  1.46s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[229] generic residue numbers for the receptor...\n",
      "\n",
      "2Y00 P07700\n",
      "Found error!\n",
      "2Y00: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 43%|████████████████████████████████████████████████▉                                                                 | 239/557 [05:31<06:42,  1.26s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[17] generic residue numbers for the receptor...\n",
      "\n",
      "6ZDV P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6ZDV P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 43%|█████████████████████████████████████████████████                                                                 | 240/557 [05:32<06:07,  1.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[482] generic residue numbers for the receptor...\n",
      "\n",
      "6X1A P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 43%|█████████████████████████████████████████████████▎                                                                | 241/557 [05:34<07:43,  1.47s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[474] generic residue numbers for the receptor...\n",
      "\n",
      "6KK1 P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6KK1 P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6KK1 P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 43%|█████████████████████████████████████████████████▌                                                                | 242/557 [05:37<09:20,  1.78s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[325] generic residue numbers for the receptor...\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 44%|█████████████████████████████████████████████████▋                                                                | 243/557 [05:37<06:51,  1.31s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[221] generic residue numbers for the receptor...\n",
      "\n",
      "6N51 P41594\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 44%|█████████████████████████████████████████████████▉                                                                | 244/557 [05:41<10:46,  2.06s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[371] generic residue numbers for the receptor...\n",
      "\n",
      "6E67 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6E67 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 44%|██████████████████████████████████████████████████▏                                                               | 245/557 [05:43<10:28,  2.01s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[283] generic residue numbers for the receptor...\n",
      "\n",
      "4BVN P07700\n",
      "Found error!\n",
      "4BVN: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 44%|██████████████████████████████████████████████████▎                                                               | 246/557 [05:43<08:09,  1.57s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[77] generic residue numbers for the receptor...\n",
      "\n",
      "5DSG P08173\n",
      "Found error!\n",
      "5DSG: Using uniprot (looking up P08173) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ACM4_HUMAN\n",
      "start pdb: 3\n",
      "start uni: 22\n",
      "\n",
      "5DSG P08173\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 44%|██████████████████████████████████████████████████▌                                                               | 247/557 [05:45<08:03,  1.56s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[152] generic residue numbers for the receptor...\n",
      "\n",
      "6UVA Q16602\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 45%|██████████████████████████████████████████████████▊                                                               | 248/557 [05:47<08:52,  1.72s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[450] generic residue numbers for the receptor...\n",
      "\n",
      "6LFM P25025\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 45%|██████████████████████████████████████████████████▉                                                               | 249/557 [05:49<09:53,  1.93s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[338] generic residue numbers for the receptor...\n",
      "\n",
      "6PT0 P34972\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 45%|███████████████████████████████████████████████████▏                                                              | 250/557 [05:51<09:58,  1.95s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[412] generic residue numbers for the receptor...\n",
      "\n",
      "4GPO P07700\n",
      "Found error!\n",
      "4GPO: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 1\n",
      "start uni: 31\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 45%|███████████████████████████████████████████████████▎                                                              | 251/557 [05:52<08:11,  1.61s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[86] generic residue numbers for the receptor...\n",
      "\n",
      "3VW7 P25116\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3VW7 P25116\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 45%|███████████████████████████████████████████████████▌                                                              | 252/557 [05:53<07:16,  1.43s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[66] generic residue numbers for the receptor...\n",
      "\n",
      "6FUF P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 45%|███████████████████████████████████████████████████▊                                                              | 253/557 [05:54<06:28,  1.28s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[295] generic residue numbers for the receptor...\n",
      "\n",
      "6FK9 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 46%|███████████████████████████████████████████████████▉                                                              | 254/557 [05:55<05:34,  1.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[290] generic residue numbers for the receptor...\n",
      "\n",
      "6FK6 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 46%|████████████████████████████████████████████████████▏                                                             | 255/557 [05:55<04:53,  1.03it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[287] generic residue numbers for the receptor...\n",
      "\n",
      "7JVP P21728\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 46%|████████████████████████████████████████████████████▍                                                             | 256/557 [05:57<06:11,  1.23s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[537] generic residue numbers for the receptor...\n",
      "\n",
      "4XT3 P69332\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 46%|████████████████████████████████████████████████████▌                                                             | 257/557 [05:58<05:22,  1.08s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[131] generic residue numbers for the receptor...\n",
      "\n",
      "6XBJ Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 46%|████████████████████████████████████████████████████▊                                                             | 258/557 [06:01<07:39,  1.54s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[475] generic residue numbers for the receptor...\n",
      "\n",
      "6LI2 Q9Y2T5\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6LI2 Q9Y2T5\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 46%|█████████████████████████████████████████████████████                                                             | 259/557 [06:01<06:36,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[342] generic residue numbers for the receptor...\n",
      "\n",
      "6TP6 O43613\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 47%|█████████████████████████████████████████████████████▏                                                            | 260/557 [06:02<06:12,  1.26s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[434] generic residue numbers for the receptor...\n",
      "\n",
      "4YAY P30556\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 47%|█████████████████████████████████████████████████████▍                                                            | 261/557 [06:03<05:24,  1.10s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[132] generic residue numbers for the receptor...\n",
      "\n",
      "6RZ5 Q9Y271\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6RZ5 Q9Y271\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 47%|█████████████████████████████████████████████████████▌                                                            | 262/557 [06:05<06:31,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[420] generic residue numbers for the receptor...\n",
      "\n",
      "6PB0 P34998\n",
      "Found error!\n",
      "6PB0: Using uniprot (looking up P34998) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for CRFR1_HUMAN\n",
      "start pdb: 1\n",
      "start uni: 24\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 47%|█████████████████████████████████████████████████████▊                                                            | 263/557 [06:06<06:29,  1.32s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[397] generic residue numbers for the receptor...\n",
      "\n",
      "5VRA P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5VRA P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 47%|██████████████████████████████████████████████████████                                                            | 264/557 [06:08<06:57,  1.42s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[219] generic residue numbers for the receptor...\n",
      "\n",
      "3UZA P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 48%|██████████████████████████████████████████████████████▏                                                           | 265/557 [06:09<05:43,  1.18s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[60] generic residue numbers for the receptor...\n",
      "\n",
      "6NBH Q03431\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 48%|██████████████████████████████████████████████████████▍                                                           | 266/557 [06:11<07:01,  1.45s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[374] generic residue numbers for the receptor...\n",
      "\n",
      "7D68 O95838\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 48%|██████████████████████████████████████████████████████▋                                                           | 267/557 [06:13<07:46,  1.61s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[515] generic residue numbers for the receptor...\n",
      "\n",
      "6UP7 P30989\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 48%|██████████████████████████████████████████████████████▊                                                           | 268/557 [06:14<06:46,  1.41s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[447] generic residue numbers for the receptor...\n",
      "\n",
      "3OE6 P61073\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3OE6 P61073\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 48%|███████████████████████████████████████████████████████                                                           | 269/557 [06:15<06:10,  1.29s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[45] generic residue numbers for the receptor...\n",
      "\n",
      "5NDZ P55085\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5NDZ P55085\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 48%|███████████████████████████████████████████████████████▎                                                          | 270/557 [06:16<06:36,  1.38s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[180] generic residue numbers for the receptor...\n",
      "\n",
      "4BEZ P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 49%|███████████████████████████████████████████████████████▍                                                          | 271/557 [06:17<05:34,  1.17s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[74] generic residue numbers for the receptor...\n",
      "\n",
      "3C9M P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 49%|███████████████████████████████████████████████████████▋                                                          | 272/557 [06:18<04:51,  1.02s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[33] generic residue numbers for the receptor...\n",
      "\n",
      "5XEZ P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5XEZ P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 49%|███████████████████████████████████████████████████████▊                                                          | 273/557 [06:21<08:53,  1.88s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[233] generic residue numbers for the receptor...\n",
      "\n",
      "5D6L P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5D6L P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 49%|████████████████████████████████████████████████████████                                                          | 274/557 [06:23<07:41,  1.63s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[148] generic residue numbers for the receptor...\n",
      "\n",
      "6UO9 O75899\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 49%|████████████████████████████████████████████████████████▎                                                         | 275/557 [06:25<09:15,  1.97s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[445] generic residue numbers for the receptor...\n",
      "\n",
      "7C7S O75899\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 50%|████████████████████████████████████████████████████████▍                                                         | 276/557 [06:28<10:41,  2.28s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[497] generic residue numbers for the receptor...\n",
      "\n",
      "6TPK P30559\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 50%|████████████████████████████████████████████████████████▋                                                         | 277/557 [06:29<09:06,  1.95s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[437] generic residue numbers for the receptor...\n",
      "\n",
      "6KJV P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6KJV P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6KJV P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 50%|████████████████████████████████████████████████████████▉                                                         | 278/557 [06:32<09:20,  2.01s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[324] generic residue numbers for the receptor...\n",
      "\n",
      "5VBL P35414\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5VBL P35414\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 50%|█████████████████████████████████████████████████████████                                                         | 279/557 [06:32<07:44,  1.67s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[216] generic residue numbers for the receptor...\n",
      "\n",
      "6ME6 P49286\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6ME6 P49286\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 50%|█████████████████████████████████████████████████████████▎                                                        | 280/557 [06:34<07:39,  1.66s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[361] generic residue numbers for the receptor...\n",
      "\n",
      "5V57 Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5V57 Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 50%|█████████████████████████████████████████████████████████▌                                                        | 281/557 [06:37<09:18,  2.02s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[214] generic residue numbers for the receptor...\n",
      "\n",
      "4GBR P07550\n",
      "Found error!\n",
      "4GBR: Using uniprot (looking up P07550) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB2_HUMAN\n",
      "start pdb: 1\n",
      "start uni: 29\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 51%|█████████████████████████████████████████████████████████▋                                                        | 282/557 [06:38<07:26,  1.63s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[85] generic residue numbers for the receptor...\n",
      "\n",
      "3UON P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3UON P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 51%|█████████████████████████████████████████████████████████▉                                                        | 283/557 [06:39<06:35,  1.44s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[59] generic residue numbers for the receptor...\n",
      "\n",
      "6FFI P41594\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6FFI P41594\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 51%|██████████████████████████████████████████████████████████▏                                                       | 284/557 [06:40<05:51,  1.29s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[285] generic residue numbers for the receptor...\n",
      "\n",
      "6AQF P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6AQF P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 51%|██████████████████████████████████████████████████████████▎                                                       | 285/557 [06:41<05:19,  1.18s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[259] generic residue numbers for the receptor...\n",
      "\n",
      "3DQB P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 51%|██████████████████████████████████████████████████████████▌                                                       | 286/557 [06:42<05:10,  1.14s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[36] generic residue numbers for the receptor...\n",
      "\n",
      "4OR2 Q13255\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 52%|██████████████████████████████████████████████████████████▋                                                       | 287/557 [06:43<05:06,  1.14s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[107] generic residue numbers for the receptor...\n",
      "\n",
      "6TPJ O43614\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6TPJ O43614\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 52%|██████████████████████████████████████████████████████████▉                                                       | 288/557 [06:45<06:27,  1.44s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[436] generic residue numbers for the receptor...\n",
      "\n",
      "4LDE P07550\n",
      "Found error!\n",
      "4LDE: Using uniprot (looking up P07550) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB2_HUMAN\n",
      "start pdb: 178\n",
      "start uni: 29\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 52%|███████████████████████████████████████████████████████████▏                                                      | 289/557 [06:46<05:34,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[95] generic residue numbers for the receptor...\n",
      "\n",
      "5W0P P08100\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 52%|███████████████████████████████████████████████████████████▎                                                      | 290/557 [06:50<10:11,  2.29s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[220] generic residue numbers for the receptor...\n",
      "\n",
      "5N2R P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5N2R P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 52%|███████████████████████████████████████████████████████████▌                                                      | 291/557 [06:52<08:47,  1.98s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[177] generic residue numbers for the receptor...\n",
      "\n",
      "6IQL P51436\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6IQL P51436\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 52%|███████████████████████████████████████████████████████████▊                                                      | 292/557 [06:53<07:53,  1.79s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[316] generic residue numbers for the receptor...\n",
      "\n",
      "6LI0 Q9Y2T5\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6LI0 Q9Y2T5\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 53%|███████████████████████████████████████████████████████████▉                                                      | 293/557 [06:54<06:50,  1.55s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[340] generic residue numbers for the receptor...\n",
      "\n",
      "6J20 P25103\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6J20 P25103\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 53%|████████████████████████████████████████████████████████████▏                                                     | 294/557 [06:55<06:01,  1.37s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[317] generic residue numbers for the receptor...\n",
      "\n",
      "7DFP P14416\n",
      "Found error!\n",
      "7DFP: Using uniprot (looking up P14416) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for DRD2_HUMAN\n",
      "start pdb: 1\n",
      "start uni: 34\n",
      "\n",
      "7DFP P14416\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "7DFP P14416\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 53%|████████████████████████████████████████████████████████████▍                                                     | 295/557 [06:57<06:35,  1.51s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[521] generic residue numbers for the receptor...\n",
      "\n",
      "5X7D P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5X7D P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 53%|████████████████████████████████████████████████████████████▌                                                     | 296/557 [06:58<05:58,  1.37s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[231] generic residue numbers for the receptor...\n",
      "\n",
      "6LPJ P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6LPJ P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 53%|████████████████████████████████████████████████████████████▊                                                     | 297/557 [06:59<05:21,  1.24s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[348] generic residue numbers for the receptor...\n",
      "\n",
      "6D27 Q9Y5Y4\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6D27 Q9Y5Y4\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6D27 Q9Y5Y4\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 54%|████████████████████████████████████████████████████████████▉                                                     | 298/557 [07:00<05:22,  1.24s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[270] generic residue numbers for the receptor...\n",
      "\n",
      "6M1I P41586\n",
      "Found error!\n",
      "6M1I: Using uniprot (looking up P41586) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for PACR_HUMAN\n",
      "start pdb: 6\n",
      "start uni: 23\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 54%|█████████████████████████████████████████████████████████████▏                                                    | 299/557 [07:01<05:31,  1.29s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[355] generic residue numbers for the receptor...\n",
      "\n",
      "7BZ2 P07550\n",
      "Found error!\n",
      "7BZ2: Using uniprot (looking up P07550) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB2_HUMAN\n",
      "start pdb: 25\n",
      "start uni: 1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 54%|█████████████████████████████████████████████████████████████▍                                                    | 300/557 [07:03<05:20,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[492] generic residue numbers for the receptor...\n",
      "\n",
      "5UVI P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5UVI P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 54%|█████████████████████████████████████████████████████████████▌                                                    | 301/557 [07:03<04:52,  1.14s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[210] generic residue numbers for the receptor...\n",
      "\n",
      "5DHG P41146\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 54%|█████████████████████████████████████████████████████████████▊                                                    | 302/557 [07:05<05:11,  1.22s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[150] generic residue numbers for the receptor...\n",
      "\n",
      "5LWE P51686\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 54%|██████████████████████████████████████████████████████████████                                                    | 303/557 [07:06<04:51,  1.15s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[174] generic residue numbers for the receptor...\n",
      "\n",
      "5K2B P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5K2B P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 55%|██████████████████████████████████████████████████████████████▏                                                   | 304/557 [07:07<04:34,  1.08s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[168] generic residue numbers for the receptor...\n",
      "\n",
      "4QIM Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4QIM Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 55%|██████████████████████████████████████████████████████████████▍                                                   | 305/557 [07:08<04:33,  1.09s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[112] generic residue numbers for the receptor...\n",
      "\n",
      "6RNK Q6IYF9\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 55%|██████████████████████████████████████████████████████████████▋                                                   | 306/557 [07:09<04:08,  1.01it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[418] generic residue numbers for the receptor...\n",
      "\n",
      "1U19 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 55%|██████████████████████████████████████████████████████████████▊                                                   | 307/557 [07:10<04:22,  1.05s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[4] generic residue numbers for the receptor...\n",
      "\n",
      "6PRZ P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6PRZ P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 55%|███████████████████████████████████████████████████████████████                                                   | 308/557 [07:11<04:20,  1.04s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[402] generic residue numbers for the receptor...\n",
      "\n",
      "5WF5 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5WF5 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 55%|███████████████████████████████████████████████████████████████▏                                                  | 309/557 [07:12<04:17,  1.04s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[223] generic residue numbers for the receptor...\n",
      "\n",
      "5XPR P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5XPR P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 56%|███████████████████████████████████████████████████████████████▍                                                  | 310/557 [07:13<04:12,  1.02s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[236] generic residue numbers for the receptor...\n",
      "\n",
      "5ZBQ P25929\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 56%|███████████████████████████████████████████████████████████████▋                                                  | 311/557 [07:14<04:01,  1.02it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[245] generic residue numbers for the receptor...\n",
      "\n",
      "3PXO P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 56%|███████████████████████████████████████████████████████████████▊                                                  | 312/557 [07:14<03:38,  1.12it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[53] generic residue numbers for the receptor...\n",
      "\n",
      "6WGT P28223\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6WGT P28223\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 56%|████████████████████████████████████████████████████████████████                                                  | 313/557 [07:16<05:01,  1.23s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[460] generic residue numbers for the receptor...\n",
      "\n",
      "6RZ4 Q9Y271\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6RZ4 Q9Y271\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 56%|████████████████████████████████████████████████████████████████▎                                                 | 314/557 [07:17<04:36,  1.14s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Error Status: False\n",
      "assigned dfl[419] generic residue numbers for the receptor...\n",
      "\n",
      "6PS0 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6PS0 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 57%|████████████████████████████████████████████████████████████████▍                                                 | 315/557 [07:18<04:28,  1.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[403] generic residue numbers for the receptor...\n",
      "\n",
      "7CMV P35462\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 57%|████████████████████████████████████████████████████████████████▋                                                 | 316/557 [07:21<05:56,  1.48s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[507] generic residue numbers for the receptor...\n",
      "\n",
      "7DHR P07550\n",
      "Found error!\n",
      "7DHR: Using uniprot (looking up P07550) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB2_HUMAN\n",
      "start pdb: 25\n",
      "start uni: 1\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 57%|████████████████████████████████████████████████████████████████▉                                                 | 317/557 [07:22<05:29,  1.37s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[523] generic residue numbers for the receptor...\n",
      "\n",
      "6OBA P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6OBA P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 57%|█████████████████████████████████████████████████████████████████                                                 | 318/557 [07:23<05:06,  1.28s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[380] generic residue numbers for the receptor...\n",
      "\n",
      "6FFH P41594\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6FFH P41594\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 57%|█████████████████████████████████████████████████████████████████▎                                                | 319/557 [07:24<04:38,  1.17s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[284] generic residue numbers for the receptor...\n",
      "\n",
      "6LRY P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6LRY P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6LRY P24530\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 57%|█████████████████████████████████████████████████████████████████▍                                                | 320/557 [07:25<04:54,  1.24s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[351] generic residue numbers for the receptor...\n",
      "\n",
      "3REY P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 58%|█████████████████████████████████████████████████████████████████▋                                                | 321/557 [07:26<04:08,  1.05s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[55] generic residue numbers for the receptor...\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 58%|█████████████████████████████████████████████████████████████████▉                                                | 322/557 [07:26<03:15,  1.20it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[222] generic residue numbers for the receptor...\n",
      "\n",
      "6NWE P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 58%|██████████████████████████████████████████████████████████████████                                                | 323/557 [07:27<03:05,  1.26it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[378] generic residue numbers for the receptor...\n",
      "\n",
      "6LUQ P14416\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6LUQ P14416\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 58%|██████████████████████████████████████████████████████████████████▎                                               | 324/557 [07:28<03:12,  1.21it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[352] generic residue numbers for the receptor...\n",
      "\n",
      "7C2E P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 58%|██████████████████████████████████████████████████████████████████▌                                               | 325/557 [07:30<04:28,  1.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[493] generic residue numbers for the receptor...\n",
      "\n",
      "4K5Y P34998\n",
      "Found error!\n",
      "4K5Y: Using uniprot (looking up P34998) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for CRFR1_HUMAN\n",
      "start pdb: 1\n",
      "start uni: 103\n",
      "\n",
      "4K5Y P34998\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 59%|██████████████████████████████████████████████████████████████████▋                                               | 326/557 [07:32<05:55,  1.54s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[93] generic residue numbers for the receptor...\n",
      "\n",
      "6H7O P07700\n",
      "Found error!\n",
      "6H7O: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 6\n",
      "start uni: 44\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 59%|██████████████████████████████████████████████████████████████████▉                                               | 327/557 [07:33<05:31,  1.44s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[305] generic residue numbers for the receptor...\n",
      "\n",
      "6PS8 P48039\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6PS8 P48039\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 59%|███████████████████████████████████████████████████████████████████▏                                              | 328/557 [07:34<05:02,  1.32s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[411] generic residue numbers for the receptor...\n",
      "\n",
      "6OY9 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 59%|███████████████████████████████████████████████████████████████████▎                                              | 329/557 [07:36<05:17,  1.39s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[393] generic residue numbers for the receptor...\n",
      "\n",
      "5ZTY P34972\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5ZTY P34972\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 59%|███████████████████████████████████████████████████████████████████▌                                              | 330/557 [07:37<04:50,  1.28s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[253] generic residue numbers for the receptor...\n",
      "\n",
      "3NYA P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3NYA P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 59%|███████████████████████████████████████████████████████████████████▋                                              | 331/557 [07:38<04:34,  1.21s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[41] generic residue numbers for the receptor...\n",
      "\n",
      "5ZHP P08483\n",
      "Found error!\n",
      "5ZHP: Using uniprot (looking up P08483) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ACM3_RAT\n",
      "start pdb: 2\n",
      "start uni: 57\n",
      "\n",
      "5ZHP P08483\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 60%|███████████████████████████████████████████████████████████████████▉                                              | 332/557 [07:40<04:54,  1.31s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[246] generic residue numbers for the receptor...\n",
      "\n",
      "6FKC P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 60%|████████████████████████████████████████████████████████████████████▏                                             | 333/557 [07:40<04:10,  1.12s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[293] generic residue numbers for the receptor...\n",
      "\n",
      "6FK8 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 60%|████████████████████████████████████████████████████████████████████▎                                             | 334/557 [07:41<03:40,  1.01it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[289] generic residue numbers for the receptor...\n",
      "\n",
      "5TZR O14842\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5TZR O14842\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 60%|████████████████████████████████████████████████████████████████████▌                                             | 335/557 [07:42<03:37,  1.02it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[201] generic residue numbers for the receptor...\n",
      "\n",
      "6LI3 Q9Y2T5\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 60%|████████████████████████████████████████████████████████████████████▊                                             | 336/557 [07:44<04:23,  1.19s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[343] generic residue numbers for the receptor...\n",
      "\n",
      "5NM4 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5NM4 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 61%|████████████████████████████████████████████████████████████████████▉                                             | 337/557 [07:46<05:12,  1.42s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[184] generic residue numbers for the receptor...\n",
      "\n",
      "4ZJC O43613\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4ZJC O43613\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 61%|█████████████████████████████████████████████████████████████████████▏                                            | 338/557 [07:47<04:56,  1.35s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[138] generic residue numbers for the receptor...\n",
      "\n",
      "7EB2 O75899\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 61%|█████████████████████████████████████████████████████████████████████▍                                            | 339/557 [07:52<08:44,  2.41s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[533] generic residue numbers for the receptor...\n",
      "\n",
      "4L6R P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 61%|█████████████████████████████████████████████████████████████████████▌                                            | 340/557 [07:52<06:52,  1.90s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[94] generic residue numbers for the receptor...\n",
      "\n",
      "3C9L P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 61%|█████████████████████████████████████████████████████████████████████▊                                            | 341/557 [07:53<05:31,  1.54s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[32] generic residue numbers for the receptor...\n",
      "\n",
      "4XT1 P69332\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 61%|█████████████████████████████████████████████████████████████████████▉                                            | 342/557 [07:54<04:48,  1.34s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[130] generic residue numbers for the receptor...\n",
      "\n",
      "6TQ6 O43613\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 62%|██████████████████████████████████████████████████████████████████████▏                                           | 343/557 [07:55<04:29,  1.26s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[440] generic residue numbers for the receptor...\n",
      "\n",
      "1L9H P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 62%|██████████████████████████████████████████████████████████████████████▍                                           | 344/557 [07:56<04:18,  1.21s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[3] generic residue numbers for the receptor...\n",
      "\n",
      "3D4S P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3D4S P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 62%|██████████████████████████████████████████████████████████████████████▌                                           | 345/557 [07:57<04:05,  1.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[35] generic residue numbers for the receptor...\n",
      "\n",
      "2Y02 P07700\n",
      "Found error!\n",
      "2Y02: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 62%|██████████████████████████████████████████████████████████████████████▊                                           | 346/557 [07:58<03:42,  1.06s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[19] generic residue numbers for the receptor...\n",
      "\n",
      "3PWH P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 62%|███████████████████████████████████████████████████████████████████████                                           | 347/557 [07:59<03:12,  1.09it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[52] generic residue numbers for the receptor...\n",
      "\n",
      "5ZKP P25105\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5ZKP P25105\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 62%|███████████████████████████████████████████████████████████████████████▏                                          | 348/557 [08:00<03:17,  1.06it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[251] generic residue numbers for the receptor...\n",
      "\n",
      "2YDV P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 63%|███████████████████████████████████████████████████████████████████████▍                                          | 349/557 [08:00<02:56,  1.18it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[27] generic residue numbers for the receptor...\n",
      "\n",
      "6NBI Q03431\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 63%|███████████████████████████████████████████████████████████████████████▋                                          | 350/557 [08:03<04:32,  1.32s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[375] generic residue numbers for the receptor...\n",
      "\n",
      "5WKT P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 63%|███████████████████████████████████████████████████████████████████████▊                                          | 351/557 [08:03<03:53,  1.13s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[227] generic residue numbers for the receptor...\n",
      "\n",
      "6PB1 Q13324\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 63%|████████████████████████████████████████████████████████████████████████                                          | 352/557 [08:05<04:34,  1.34s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[398] generic residue numbers for the receptor...\n",
      "\n",
      "6MXT P07550\n",
      "Found error!\n",
      "6MXT: Using uniprot (looking up P07550) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB2_HUMAN\n",
      "start pdb: 164\n",
      "start uni: 29\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 63%|████████████████████████████████████████████████████████████████████████▏                                         | 353/557 [08:06<04:00,  1.18s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[368] generic residue numbers for the receptor...\n",
      "\n",
      "3PQR P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 64%|████████████████████████████████████████████████████████████████████████▍                                         | 354/557 [08:07<03:30,  1.04s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[51] generic residue numbers for the receptor...\n",
      "\n",
      "4LDO P07550\n",
      "Found error!\n",
      "4LDO: Using uniprot (looking up P07550) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB2_HUMAN\n",
      "start pdb: 178\n",
      "start uni: 29\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 64%|████████████████████████████████████████████████████████████████████████▋                                         | 355/557 [08:07<03:14,  1.04it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[97] generic residue numbers for the receptor...\n",
      "\n",
      "5O9H P21730\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 64%|████████████████████████████████████████████████████████████████████████▊                                         | 356/557 [08:08<03:14,  1.03it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[186] generic residue numbers for the receptor...\n",
      "Found no mapping -> not assigning any residue numbers\n",
      "assigned dfl[386] generic residue numbers for the receptor...\n",
      "\n",
      "5NDD P55085\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5NDD P55085\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 64%|█████████████████████████████████████████████████████████████████████████▎                                        | 358/557 [08:10<02:42,  1.23it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[179] generic residue numbers for the receptor...\n",
      "\n",
      "5OLV P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5OLV P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 64%|█████████████████████████████████████████████████████████████████████████▍                                        | 359/557 [08:11<02:46,  1.19it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[190] generic residue numbers for the receptor...\n",
      "\n",
      "3KJ6 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 65%|█████████████████████████████████████████████████████████████████████████▋                                        | 360/557 [08:12<03:00,  1.09it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[38] generic residue numbers for the receptor...\n",
      "\n",
      "6PS6 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6PS6 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 65%|█████████████████████████████████████████████████████████████████████████▉                                        | 361/557 [08:13<03:05,  1.06it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[409] generic residue numbers for the receptor...\n",
      "\n",
      "6AK3 P43115\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 65%|██████████████████████████████████████████████████████████████████████████                                        | 362/557 [08:14<03:02,  1.07it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[256] generic residue numbers for the receptor...\n",
      "\n",
      "6ME7 P49286\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6ME7 P49286\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 65%|██████████████████████████████████████████████████████████████████████████▎                                       | 363/557 [08:16<03:59,  1.24s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[362] generic residue numbers for the receptor...\n",
      "\n",
      "6IIU P21731\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6IIU P21731\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 65%|██████████████████████████████████████████████████████████████████████████▍                                       | 364/557 [08:17<03:46,  1.17s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[314] generic residue numbers for the receptor...\n",
      "\n",
      "3NY8 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3NY8 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 66%|██████████████████████████████████████████████████████████████████████████▋                                       | 365/557 [08:18<03:37,  1.13s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[39] generic residue numbers for the receptor...\n",
      "\n",
      "3V2W P21453\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3V2W P21453\n",
      "Found error!\n",
      "3V2W: Using uniprot (looking up P21453) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for S1PR1_HUMAN\n",
      "start pdb: 410\n",
      "start uni: 244\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 66%|██████████████████████████████████████████████████████████████████████████▉                                       | 366/557 [08:19<03:33,  1.12s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[62] generic residue numbers for the receptor...\n",
      "\n",
      "4XES P20789\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 66%|███████████████████████████████████████████████████████████████████████████                                       | 367/557 [08:20<03:20,  1.05s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[127] generic residue numbers for the receptor...\n",
      "\n",
      "4WW3 P31356\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 66%|███████████████████████████████████████████████████████████████████████████▎                                      | 368/557 [08:21<03:28,  1.10s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[124] generic residue numbers for the receptor...\n",
      "\n",
      "6N4B P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6N4B P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 66%|███████████████████████████████████████████████████████████████████████████▌                                      | 369/557 [08:23<04:41,  1.50s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[370] generic residue numbers for the receptor...\n",
      "\n",
      "4MQS P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4MQS P08172\n",
      "Found error!\n",
      "4MQS: Using uniprot (looking up P08172) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ACM2_HUMAN\n",
      "start pdb: 36\n",
      "start uni: 19\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 66%|███████████████████████████████████████████████████████████████████████████▋                                      | 370/557 [08:24<04:02,  1.29s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[99] generic residue numbers for the receptor...\n",
      "\n",
      "6FKD P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 67%|███████████████████████████████████████████████████████████████████████████▉                                      | 371/557 [08:25<03:27,  1.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[294] generic residue numbers for the receptor...\n",
      "\n",
      "5XF1 P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5XF1 P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 67%|████████████████████████████████████████████████████████████████████████████▏                                     | 372/557 [08:29<06:18,  2.05s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[234] generic residue numbers for the receptor...\n",
      "\n",
      "6TPG O43614\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6TPG O43614\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 67%|████████████████████████████████████████████████████████████████████████████▎                                     | 373/557 [08:30<05:28,  1.79s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[435] generic residue numbers for the receptor...\n",
      "\n",
      "6OIK P08172\n",
      "Found error!\n",
      "6OIK: Using uniprot (looking up P08172) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ACM2_HUMAN\n",
      "start pdb: 8\n",
      "start uni: 3\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 67%|████████████████████████████████████████████████████████████████████████████▌                                     | 374/557 [08:32<04:58,  1.63s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[383] generic residue numbers for the receptor...\n",
      "\n",
      "2R4R P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 67%|████████████████████████████████████████████████████████████████████████████▊                                     | 375/557 [08:33<04:28,  1.47s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[12] generic residue numbers for the receptor...\n",
      "\n",
      "6TPN O43614\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6TPN O43614\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 68%|████████████████████████████████████████████████████████████████████████████▉                                     | 376/557 [08:34<04:11,  1.39s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[438] generic residue numbers for the receptor...\n",
      "\n",
      "6H7L P07700\n",
      "Found error!\n",
      "6H7L: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 6\n",
      "start uni: 44\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 68%|█████████████████████████████████████████████████████████████████████████████▏                                    | 377/557 [08:35<04:00,  1.34s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[302] generic residue numbers for the receptor...\n",
      "\n",
      "2R4S P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 68%|█████████████████████████████████████████████████████████████████████████████▎                                    | 378/557 [08:36<03:44,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[13] generic residue numbers for the receptor...\n",
      "\n",
      "6P9Y P41586\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 68%|█████████████████████████████████████████████████████████████████████████████▌                                    | 379/557 [08:38<04:11,  1.41s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[396] generic residue numbers for the receptor...\n",
      "\n",
      "5WQC O43614\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5WQC O43614\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 68%|█████████████████████████████████████████████████████████████████████████████▊                                    | 380/557 [08:39<03:56,  1.34s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[228] generic residue numbers for the receptor...\n",
      "\n",
      "6E3Y Q16602\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 68%|█████████████████████████████████████████████████████████████████████████████▉                                    | 381/557 [08:41<04:50,  1.65s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[281] generic residue numbers for the receptor...\n",
      "\n",
      "4QIN Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4QIN Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 69%|██████████████████████████████████████████████████████████████████████████████▏                                   | 382/557 [08:42<04:16,  1.47s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[113] generic residue numbers for the receptor...\n",
      "\n",
      "6IBL P07700\n",
      "Found error!\n",
      "6IBL: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 115\n",
      "start uni: 44\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 69%|██████████████████████████████████████████████████████████████████████████████▍                                   | 383/557 [08:44<04:01,  1.39s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[311] generic residue numbers for the receptor...\n",
      "\n",
      "5TUD P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5TUD P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 69%|██████████████████████████████████████████████████████████████████████████████▌                                   | 384/557 [08:47<05:22,  1.87s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[199] generic residue numbers for the receptor...\n",
      "\n",
      "5JQH P07550\n",
      "Found error!\n",
      "5JQH: Using uniprot (looking up P07550) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB2_HUMAN\n",
      "start pdb: 179\n",
      "start uni: 30\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 69%|██████████████████████████████████████████████████████████████████████████████▊                                   | 385/557 [08:48<04:44,  1.65s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[165] generic residue numbers for the receptor...\n",
      "\n",
      "6AKY P51681\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6AKY P51681\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 69%|███████████████████████████████████████████████████████████████████████████████                                   | 386/557 [08:49<04:00,  1.40s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[258] generic residue numbers for the receptor...\n",
      "\n",
      "7K15 Q15722\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "7K15 Q15722\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 69%|███████████████████████████████████████████████████████████████████████████████▏                                  | 387/557 [08:50<03:37,  1.28s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[540] generic residue numbers for the receptor...\n",
      "\n",
      "4AMJ P07700\n",
      "Found error!\n",
      "4AMJ: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 70%|███████████████████████████████████████████████████████████████████████████████▍                                  | 388/557 [08:50<03:12,  1.14s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[72] generic residue numbers for the receptor...\n",
      "\n",
      "4UG2 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 70%|███████████████████████████████████████████████████████████████████████████████▌                                  | 389/557 [08:52<03:19,  1.19s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[122] generic residue numbers for the receptor...\n",
      "\n",
      "4MBS P51681\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4MBS P51681\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 70%|███████████████████████████████████████████████████████████████████████████████▊                                  | 390/557 [08:53<03:35,  1.29s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[98] generic residue numbers for the receptor...\n",
      "\n",
      "6NBF Q03431\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 70%|████████████████████████████████████████████████████████████████████████████████                                  | 391/557 [08:55<04:12,  1.52s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[373] generic residue numbers for the receptor...\n",
      "\n",
      "6PH7 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 70%|████████████████████████████████████████████████████████████████████████████████▏                                 | 392/557 [08:56<03:29,  1.27s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[401] generic residue numbers for the receptor...\n",
      "\n",
      "5KW2 O14842\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5KW2 O14842\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 71%|████████████████████████████████████████████████████████████████████████████████▍                                 | 393/557 [08:57<03:10,  1.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[171] generic residue numbers for the receptor...\n",
      "\n",
      "3NY9 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3NY9 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 71%|████████████████████████████████████████████████████████████████████████████████▋                                 | 394/557 [08:58<03:01,  1.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[40] generic residue numbers for the receptor...\n",
      "\n",
      "5ZKC P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5ZKC P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 71%|████████████████████████████████████████████████████████████████████████████████▊                                 | 395/557 [08:59<02:49,  1.05s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[250] generic residue numbers for the receptor...\n",
      "\n",
      "6MET P51681\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 71%|█████████████████████████████████████████████████████████████████████████████████                                 | 396/557 [09:00<03:16,  1.22s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[366] generic residue numbers for the receptor...\n",
      "\n",
      "7E2Y P08908\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 71%|█████████████████████████████████████████████████████████████████████████████████▎                                | 397/557 [09:02<03:27,  1.30s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[529] generic residue numbers for the receptor...\n",
      "\n",
      "7C6A P50052\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "7C6A P50052\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 71%|█████████████████████████████████████████████████████████████████████████████████▍                                | 398/557 [09:04<04:00,  1.51s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[495] generic residue numbers for the receptor...\n",
      "\n",
      "5U09 P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5U09 P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 72%|█████████████████████████████████████████████████████████████████████████████████▋                                | 399/557 [09:05<03:37,  1.38s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[203] generic residue numbers for the receptor...\n",
      "\n",
      "6C1Q P21730\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 72%|█████████████████████████████████████████████████████████████████████████████████▊                                | 400/557 [09:06<03:02,  1.17s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[265] generic residue numbers for the receptor...\n",
      "\n",
      "5G53 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 72%|██████████████████████████████████████████████████████████████████████████████████                                | 401/557 [09:07<03:13,  1.24s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[157] generic residue numbers for the receptor...\n",
      "\n",
      "6OSA P30989\n",
      "Found error!\n",
      "6OSA: Using uniprot (looking up P30989) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for NTR1_HUMAN\n",
      "start pdb: 40\n",
      "start uni: 20\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 72%|██████████████████████████████████████████████████████████████████████████████████▎                               | 402/557 [09:08<03:07,  1.21s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[391] generic residue numbers for the receptor...\n",
      "\n",
      "6ME3 P48039\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6ME3 P48039\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 72%|██████████████████████████████████████████████████████████████████████████████████▍                               | 403/557 [09:09<02:59,  1.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[358] generic residue numbers for the receptor...\n",
      "\n",
      "7DFL P35367\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 73%|██████████████████████████████████████████████████████████████████████████████████▋                               | 404/557 [09:11<03:38,  1.43s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[520] generic residue numbers for the receptor...\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 73%|██████████████████████████████████████████████████████████████████████████████████▉                               | 405/557 [09:12<02:58,  1.18s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[483] generic residue numbers for the receptor...\n",
      "\n",
      "6DO1 P30556\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6DO1 P30556\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 73%|███████████████████████████████████████████████████████████████████████████████████                               | 406/557 [09:14<03:34,  1.42s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[276] generic residue numbers for the receptor...\n",
      "\n",
      "5K2C P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5K2C P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 73%|███████████████████████████████████████████████████████████████████████████████████▎                              | 407/557 [09:15<03:10,  1.27s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[169] generic residue numbers for the receptor...\n",
      "\n",
      "2I37 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 73%|███████████████████████████████████████████████████████████████████████████████████▌                              | 408/557 [09:17<03:36,  1.45s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[9] generic residue numbers for the receptor...\n",
      "\n",
      "6GPX P41597\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6GPX P41597\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 73%|███████████████████████████████████████████████████████████████████████████████████▋                              | 409/557 [09:19<04:13,  1.71s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[299] generic residue numbers for the receptor...\n",
      "\n",
      "6DRX P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6DRX P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 74%|███████████████████████████████████████████████████████████████████████████████████▉                              | 410/557 [09:20<03:31,  1.44s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[277] generic residue numbers for the receptor...\n",
      "\n",
      "5IUB P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5IUB P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 74%|████████████████████████████████████████████████████████████████████████████████████                              | 411/557 [09:21<03:07,  1.29s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[164] generic residue numbers for the receptor...\n",
      "\n",
      "4UHR P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 74%|████████████████████████████████████████████████████████████████████████████████████▎                             | 412/557 [09:21<02:37,  1.08s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[123] generic residue numbers for the receptor...\n",
      "\n",
      "6GT3 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6GT3 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 74%|████████████████████████████████████████████████████████████████████████████████████▌                             | 413/557 [09:22<02:29,  1.04s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[300] generic residue numbers for the receptor...\n",
      "\n",
      "6OS1 P30556\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6OS1 P30556\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 74%|████████████████████████████████████████████████████████████████████████████████████▋                             | 414/557 [09:23<02:31,  1.06s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[388] generic residue numbers for the receptor...\n",
      "\n",
      "4PXZ Q9H244\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4PXZ Q9H244\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 75%|████████████████████████████████████████████████████████████████████████████████████▉                             | 415/557 [09:24<02:25,  1.03s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[110] generic residue numbers for the receptor...\n",
      "\n",
      "5CGD P41594\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5CGD P41594\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 75%|█████████████████████████████████████████████████████████████████████████████████████▏                            | 416/557 [09:25<02:18,  1.01it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[144] generic residue numbers for the receptor...\n",
      "\n",
      "7DTT P41180\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 75%|█████████████████████████████████████████████████████████████████████████████████████▎                            | 417/557 [09:29<04:07,  1.77s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[524] generic residue numbers for the receptor...\n",
      "\n",
      "7C7Q O75899\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 75%|█████████████████████████████████████████████████████████████████████████████████████▌                            | 418/557 [09:32<05:07,  2.21s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[496] generic residue numbers for the receptor...\n",
      "\n",
      "5F8U P07700\n",
      "Found error!\n",
      "5F8U: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 75%|█████████████████████████████████████████████████████████████████████████████████████▊                            | 419/557 [09:33<04:05,  1.78s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[156] generic residue numbers for the receptor...\n",
      "\n",
      "6B3J P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 75%|█████████████████████████████████████████████████████████████████████████████████████▉                            | 420/557 [09:35<04:10,  1.83s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[260] generic residue numbers for the receptor...\n",
      "\n",
      "4A4M P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 76%|██████████████████████████████████████████████████████████████████████████████████████▏                           | 421/557 [09:35<03:22,  1.49s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[70] generic residue numbers for the receptor...\n",
      "\n",
      "4ZJ8 O43613\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4ZJ8 O43613\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 76%|██████████████████████████████████████████████████████████████████████████████████████▎                           | 422/557 [09:37<03:07,  1.39s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[137] generic residue numbers for the receptor...\n",
      "\n",
      "7BTS P08588\n",
      "Found error!\n",
      "7BTS: Using uniprot (looking up P08588) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_HUMAN\n",
      "start pdb: 171\n",
      "start uni: 54\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 76%|██████████████████████████████████████████████████████████████████████████████████████▌                           | 423/557 [09:37<02:41,  1.20s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[487] generic residue numbers for the receptor...\n",
      "\n",
      "5NX2 P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 76%|██████████████████████████████████████████████████████████████████████████████████████▊                           | 424/557 [09:38<02:25,  1.10s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[185] generic residue numbers for the receptor...\n",
      "\n",
      "2J4Y P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 76%|██████████████████████████████████████████████████████████████████████████████████████▉                           | 425/557 [09:39<02:25,  1.10s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[10] generic residue numbers for the receptor...\n",
      "\n",
      "7JV5 P21728\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 76%|███████████████████████████████████████████████████████████████████████████████████████▏                          | 426/557 [09:41<02:53,  1.32s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[536] generic residue numbers for the receptor...\n",
      "\n",
      "6TQ7 O43613\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 77%|███████████████████████████████████████████████████████████████████████████████████████▍                          | 427/557 [09:42<02:42,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[441] generic residue numbers for the receptor...\n",
      "\n",
      "5YC8 P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5YC8 P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 77%|███████████████████████████████████████████████████████████████████████████████████████▌                          | 428/557 [09:43<02:27,  1.14s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[240] generic residue numbers for the receptor...\n",
      "\n",
      "7CKY P21728\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 77%|███████████████████████████████████████████████████████████████████████████████████████▊                          | 429/557 [09:45<03:04,  1.44s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[504] generic residue numbers for the receptor...\n",
      "\n",
      "6PS1 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6PS1 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 77%|████████████████████████████████████████████████████████████████████████████████████████                          | 430/557 [09:46<02:45,  1.30s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[404] generic residue numbers for the receptor...\n",
      "\n",
      "4DJH P41145\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4DJH P41145\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 77%|████████████████████████████████████████████████████████████████████████████████████████▏                         | 431/557 [09:48<03:02,  1.45s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[80] generic residue numbers for the receptor...\n",
      "\n",
      "6MH8 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6MH8 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 78%|████████████████████████████████████████████████████████████████████████████████████████▍                         | 432/557 [09:50<03:05,  1.48s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[367] generic residue numbers for the receptor...\n",
      "\n",
      "6TQ4 O43613\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 78%|████████████████████████████████████████████████████████████████████████████████████████▌                         | 433/557 [09:51<02:47,  1.35s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[439] generic residue numbers for the receptor...\n",
      "\n",
      "7L1U O43614\n",
      "Found error!\n",
      "7L1U: Using uniprot (looking up O43614) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for OX2R_HUMAN\n",
      "start pdb: 9\n",
      "start uni: 3\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 78%|████████████████████████████████████████████████████████████████████████████████████████▊                         | 434/557 [09:52<02:44,  1.34s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[548] generic residue numbers for the receptor...\n",
      "\n",
      "6FKB P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 78%|█████████████████████████████████████████████████████████████████████████████████████████                         | 435/557 [09:53<02:17,  1.13s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[292] generic residue numbers for the receptor...\n",
      "\n",
      "6NI3 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 78%|█████████████████████████████████████████████████████████████████████████████████████████▏                        | 436/557 [09:54<02:32,  1.26s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[376] generic residue numbers for the receptor...\n",
      "\n",
      "7KNT Q16602\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 78%|█████████████████████████████████████████████████████████████████████████████████████████▍                        | 437/557 [09:55<02:20,  1.17s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[542] generic residue numbers for the receptor...\n",
      "\n",
      "7E33 P28566\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 79%|█████████████████████████████████████████████████████████████████████████████████████████▋                        | 438/557 [09:57<02:52,  1.45s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[532] generic residue numbers for the receptor...\n",
      "\n",
      "7CUM O75899\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 79%|█████████████████████████████████████████████████████████████████████████████████████████▊                        | 439/557 [10:03<05:13,  2.65s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[509] generic residue numbers for the receptor...\n",
      "\n",
      "3VGA P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 79%|██████████████████████████████████████████████████████████████████████████████████████████                        | 440/557 [10:04<04:17,  2.20s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[65] generic residue numbers for the receptor...\n",
      "\n",
      "4DKL P42866\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4DKL P42866\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 79%|██████████████████████████████████████████████████████████████████████████████████████████▎                       | 441/557 [10:05<03:33,  1.84s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[81] generic residue numbers for the receptor...\n",
      "\n",
      "6A93 P28223\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6A93 P28223\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 79%|██████████████████████████████████████████████████████████████████████████████████████████▍                       | 442/557 [10:06<03:19,  1.73s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[254] generic residue numbers for the receptor...\n",
      "\n",
      "6U1N P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 80%|██████████████████████████████████████████████████████████████████████████████████████████▋                       | 443/557 [10:08<03:05,  1.63s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[443] generic residue numbers for the receptor...\n",
      "\n",
      "5WIV P21917\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5WIV P21917\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 80%|██████████████████████████████████████████████████████████████████████████████████████████▊                       | 444/557 [10:09<02:37,  1.40s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[226] generic residue numbers for the receptor...\n",
      "\n",
      "5OM4 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5OM4 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 80%|███████████████████████████████████████████████████████████████████████████████████████████                       | 445/557 [10:09<02:20,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[193] generic residue numbers for the receptor...\n",
      "\n",
      "7CKW P21728\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 80%|███████████████████████████████████████████████████████████████████████████████████████████▎                      | 446/557 [10:12<02:48,  1.52s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[502] generic residue numbers for the receptor...\n",
      "\n",
      "3P0G P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3P0G P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 80%|███████████████████████████████████████████████████████████████████████████████████████████▍                      | 447/557 [10:13<02:28,  1.35s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[48] generic residue numbers for the receptor...\n",
      "\n",
      "6DDF P42866\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6DDF P42866\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 80%|███████████████████████████████████████████████████████████████████████████████████████████▋                      | 448/557 [10:14<02:42,  1.49s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[275] generic residue numbers for the receptor...\n",
      "\n",
      "6PWC P30989\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 81%|███████████████████████████████████████████████████████████████████████████████████████████▉                      | 449/557 [10:16<02:34,  1.43s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[415] generic residue numbers for the receptor...\n",
      "\n",
      "4U15 P08483\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4U15 P08483\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 81%|████████████████████████████████████████████████████████████████████████████████████████████                      | 450/557 [10:17<02:37,  1.47s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[120] generic residue numbers for the receptor...\n",
      "\n",
      "3ODU P61073\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3ODU P61073\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 81%|████████████████████████████████████████████████████████████████████████████████████████████▎                     | 451/557 [10:19<02:49,  1.60s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[43] generic residue numbers for the receptor...\n",
      "\n",
      "5UEN P30542\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5UEN P30542\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 81%|████████████████████████████████████████████████████████████████████████████████████████████▌                     | 452/557 [10:21<02:47,  1.60s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[204] generic residue numbers for the receptor...\n",
      "\n",
      "6IIV P21731\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6IIV P21731\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 81%|████████████████████████████████████████████████████████████████████████████████████████████▋                     | 453/557 [10:22<02:38,  1.52s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[315] generic residue numbers for the receptor...\n",
      "\n",
      "3CAP P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 82%|████████████████████████████████████████████████████████████████████████████████████████████▉                     | 454/557 [10:23<02:24,  1.40s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[34] generic residue numbers for the receptor...\n",
      "\n",
      "5V54 P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5V54 P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5V54 P28222\n",
      "Found error!\n",
      "5V54: Using uniprot (looking up P28222) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for 5HT1B_HUMAN\n",
      "start pdb: 233\n",
      "start uni: 251\n",
      "\n",
      "5V54 P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5V54 P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5V54 P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5V54 P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 82%|█████████████████████████████████████████████████████████████████████████████████████████████                     | 455/557 [10:27<03:32,  2.09s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[212] generic residue numbers for the receptor...\n",
      "\n",
      "2Y01 P07700\n",
      "Found error!\n",
      "2Y01: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 82%|█████████████████████████████████████████████████████████████████████████████████████████████▎                    | 456/557 [10:28<02:51,  1.70s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[18] generic residue numbers for the receptor...\n",
      "\n",
      "5D5B P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5D5B P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 82%|█████████████████████████████████████████████████████████████████████████████████████████████▌                    | 457/557 [10:29<02:30,  1.50s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[147] generic residue numbers for the receptor...\n",
      "\n",
      "4O9R Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4O9R Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 82%|█████████████████████████████████████████████████████████████████████████████████████████████▋                    | 458/557 [10:30<02:14,  1.36s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[105] generic residue numbers for the receptor...\n",
      "\n",
      "7LJC P21728\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 82%|█████████████████████████████████████████████████████████████████████████████████████████████▉                    | 459/557 [10:32<02:26,  1.50s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[553] generic residue numbers for the receptor...\n",
      "\n",
      "5K2D P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5K2D P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 83%|██████████████████████████████████████████████████████████████████████████████████████████████▏                   | 460/557 [10:32<02:08,  1.32s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[170] generic residue numbers for the receptor...\n",
      "\n",
      "7E32 P28221\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 83%|██████████████████████████████████████████████████████████████████████████████████████████████▎                   | 461/557 [10:35<02:30,  1.57s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[531] generic residue numbers for the receptor...\n",
      "\n",
      "6S0L P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6S0L P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6S0L P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6S0L P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 83%|██████████████████████████████████████████████████████████████████████████████████████████████▌                   | 462/557 [10:36<02:20,  1.48s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[425] generic residue numbers for the receptor...\n",
      "\n",
      "4IAQ P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4IAQ P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 83%|██████████████████████████████████████████████████████████████████████████████████████████████▊                   | 463/557 [10:37<02:00,  1.28s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[88] generic residue numbers for the receptor...\n",
      "\n",
      "6PEL P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 83%|██████████████████████████████████████████████████████████████████████████████████████████████▉                   | 464/557 [10:37<01:42,  1.10s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[399] generic residue numbers for the receptor...\n",
      "\n",
      "6C1R P21730\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 83%|███████████████████████████████████████████████████████████████████████████████████████████████▏                  | 465/557 [10:38<01:29,  1.03it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[266] generic residue numbers for the receptor...\n",
      "\n",
      "4U14 P08483\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4U14 P08483\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 84%|███████████████████████████████████████████████████████████████████████████████████████████████▍                  | 466/557 [10:39<01:28,  1.03it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[119] generic residue numbers for the receptor...\n",
      "\n",
      "5DHH P41146\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 84%|███████████████████████████████████████████████████████████████████████████████████████████████▌                  | 467/557 [10:40<01:28,  1.01it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[151] generic residue numbers for the receptor...\n",
      "\n",
      "7CX4 P43116\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 84%|███████████████████████████████████████████████████████████████████████████████████████████████▊                  | 468/557 [10:42<01:44,  1.17s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[512] generic residue numbers for the receptor...\n",
      "\n",
      "4BV0 P20789\n",
      "Found error!\n",
      "4BV0: Using uniprot (looking up P20789) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for NTR1_RAT\n",
      "start pdb: 5\n",
      "start uni: 50\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 84%|███████████████████████████████████████████████████████████████████████████████████████████████▉                  | 469/557 [10:43<01:34,  1.07s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[76] generic residue numbers for the receptor...\n",
      "\n",
      "6TP4 O43613\n",
      "Found error!\n",
      "6TP4: Using uniprot (looking up O43613) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for OX1R_HUMAN\n",
      "start pdb: 4\n",
      "start uni: 28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 84%|████████████████████████████████████████████████████████████████████████████████████████████████▏                 | 470/557 [10:43<01:26,  1.00it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[433] generic residue numbers for the receptor...\n",
      "\n",
      "5XR8 P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5XR8 P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 85%|████████████████████████████████████████████████████████████████████████████████████████████████▍                 | 471/557 [10:44<01:24,  1.02it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[237] generic residue numbers for the receptor...\n",
      "\n",
      "1HZX P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 85%|████████████████████████████████████████████████████████████████████████████████████████████████▌                 | 472/557 [10:45<01:26,  1.01s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[2] generic residue numbers for the receptor...\n",
      "\n",
      "4N6H P41143\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 85%|████████████████████████████████████████████████████████████████████████████████████████████████▊                 | 473/557 [10:46<01:17,  1.08it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[102] generic residue numbers for the receptor...\n",
      "\n",
      "7AUE P32245\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 85%|█████████████████████████████████████████████████████████████████████████████████████████████████                 | 474/557 [10:48<01:36,  1.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[485] generic residue numbers for the receptor...\n",
      "\n",
      "6H7N P07700\n",
      "Found error!\n",
      "6H7N: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 6\n",
      "start uni: 44\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 85%|█████████████████████████████████████████████████████████████████████████████████████████████████▏                | 475/557 [10:49<01:44,  1.28s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[304] generic residue numbers for the receptor...\n",
      "\n",
      "7D3S P47872\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 85%|█████████████████████████████████████████████████████████████████████████████████████████████████▍                | 476/557 [10:51<01:55,  1.43s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[514] generic residue numbers for the receptor...\n",
      "\n",
      "3RFM P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 86%|█████████████████████████████████████████████████████████████████████████████████████████████████▋                | 477/557 [10:52<01:34,  1.18s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[56] generic residue numbers for the receptor...\n",
      "\n",
      "6V9S O43613\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6V9S O43613\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 86%|█████████████████████████████████████████████████████████████████████████████████████████████████▊                | 478/557 [10:53<01:32,  1.17s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[451] generic residue numbers for the receptor...\n",
      "\n",
      "5TE3 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 86%|██████████████████████████████████████████████████████████████████████████████████████████████████                | 479/557 [10:54<01:19,  1.02s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[196] generic residue numbers for the receptor...\n",
      "\n",
      "6ME8 P49286\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6ME8 P49286\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 86%|██████████████████████████████████████████████████████████████████████████████████████████████████▏               | 480/557 [10:55<01:31,  1.18s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[363] generic residue numbers for the receptor...\n",
      "\n",
      "7E2Z P08908\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 86%|██████████████████████████████████████████████████████████████████████████████████████████████████▍               | 481/557 [10:57<01:36,  1.27s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[530] generic residue numbers for the receptor...\n",
      "\n",
      "6UO8 O75899\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 87%|██████████████████████████████████████████████████████████████████████████████████████████████████▋               | 482/557 [11:00<02:13,  1.79s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[444] generic residue numbers for the receptor...\n",
      "\n",
      "6G79 P28222\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 87%|██████████████████████████████████████████████████████████████████████████████████████████████████▊               | 483/557 [11:01<02:01,  1.64s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[296] generic residue numbers for the receptor...\n",
      "\n",
      "2PED P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 87%|███████████████████████████████████████████████████████████████████████████████████████████████████               | 484/557 [11:02<01:49,  1.50s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[11] generic residue numbers for the receptor...\n",
      "\n",
      "3SN6 P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 87%|███████████████████████████████████████████████████████████████████████████████████████████████████▎              | 485/557 [11:04<02:05,  1.74s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[58] generic residue numbers for the receptor...\n",
      "\n",
      "3ZPR P07700\n",
      "Found error!\n",
      "3ZPR: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 87%|███████████████████████████████████████████████████████████████████████████████████████████████████▍              | 486/557 [11:05<01:43,  1.46s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[69] generic residue numbers for the receptor...\n",
      "\n",
      "6LW5 P25090\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 87%|███████████████████████████████████████████████████████████████████████████████████████████████████▋              | 487/557 [11:06<01:27,  1.26s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[353] generic residue numbers for the receptor...\n",
      "\n",
      "6OS2 P30556\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6OS2 P30556\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 88%|███████████████████████████████████████████████████████████████████████████████████████████████████▉              | 488/557 [11:07<01:23,  1.21s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[389] generic residue numbers for the receptor...\n",
      "\n",
      "5XRA P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5XRA P21554\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 88%|████████████████████████████████████████████████████████████████████████████████████████████████████              | 489/557 [11:08<01:17,  1.13s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[238] generic residue numbers for the receptor...\n",
      "\n",
      "6K42 P18089\n",
      "Found error!\n",
      "6K42: Using uniprot (looking up P18089) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADA2B_HUMAN\n",
      "start pdb: 208\n",
      "start uni: 9\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 88%|████████████████████████████████████████████████████████████████████████████████████████████████████▎             | 490/557 [11:09<01:19,  1.19s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[323] generic residue numbers for the receptor...\n",
      "\n",
      "5MZJ P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5MZJ P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 88%|████████████████████████████████████████████████████████████████████████████████████████████████████▍             | 491/557 [11:10<01:12,  1.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[175] generic residue numbers for the receptor...\n",
      "\n",
      "5NJ6 P55085\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5NJ6 P55085\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 88%|████████████████████████████████████████████████████████████████████████████████████████████████████▋             | 492/557 [11:12<01:23,  1.28s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[181] generic residue numbers for the receptor...\n",
      "\n",
      "4EJ4 P32300\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4EJ4 P32300\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 89%|████████████████████████████████████████████████████████████████████████████████████████████████████▉             | 493/557 [11:13<01:15,  1.18s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[84] generic residue numbers for the receptor...\n",
      "\n",
      "4OO9 P41594\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4OO9 P41594\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 89%|█████████████████████████████████████████████████████████████████████████████████████████████████████             | 494/557 [11:14<01:08,  1.09s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[106] generic residue numbers for the receptor...\n",
      "\n",
      "6KP6 P08173\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6KP6 P08173\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 89%|█████████████████████████████████████████████████████████████████████████████████████████████████████▎            | 495/557 [11:15<01:06,  1.08s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[329] generic residue numbers for the receptor...\n",
      "\n",
      "6S0Q P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6S0Q P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 89%|█████████████████████████████████████████████████████████████████████████████████████████████████████▌            | 496/557 [11:16<01:08,  1.13s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[426] generic residue numbers for the receptor...\n",
      "\n",
      "6WPW P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 89%|█████████████████████████████████████████████████████████████████████████████████████████████████████▋            | 497/557 [11:18<01:23,  1.40s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[467] generic residue numbers for the receptor...\n",
      "\n",
      "5NLX P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5NLX P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 89%|█████████████████████████████████████████████████████████████████████████████████████████████████████▉            | 498/557 [11:20<01:24,  1.43s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[182] generic residue numbers for the receptor...\n",
      "\n",
      "5L7I Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5L7I Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 90%|██████████████████████████████████████████████████████████████████████████████████████████████████████▏           | 499/557 [11:22<01:43,  1.78s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[173] generic residue numbers for the receptor...\n",
      "\n",
      "6TQ9 O43613\n",
      "Found error!\n",
      "6TQ9: Using uniprot (looking up O43613) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for OX1R_HUMAN\n",
      "start pdb: 4\n",
      "start uni: 28\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 90%|██████████████████████████████████████████████████████████████████████████████████████████████████████▎           | 500/557 [11:23<01:25,  1.50s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[442] generic residue numbers for the receptor...\n",
      "\n",
      "6LPL P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6LPL P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 90%|██████████████████████████████████████████████████████████████████████████████████████████████████████▌           | 501/557 [11:24<01:13,  1.32s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[350] generic residue numbers for the receptor...\n",
      "\n",
      "4RWD P41143\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 90%|██████████████████████████████████████████████████████████████████████████████████████████████████████▋           | 502/557 [11:25<01:10,  1.29s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[116] generic residue numbers for the receptor...\n",
      "\n",
      "7JJO P07700\n",
      "Found error!\n",
      "7JJO: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 194\n",
      "start uni: 31\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 90%|██████████████████████████████████████████████████████████████████████████████████████████████████████▉           | 503/557 [11:28<01:28,  1.65s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[534] generic residue numbers for the receptor...\n",
      "\n",
      "2Y04 P07700\n",
      "Found error!\n",
      "2Y04: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 3\n",
      "start uni: 33\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 90%|███████████████████████████████████████████████████████████████████████████████████████████████████████▏          | 504/557 [11:28<01:13,  1.39s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[21] generic residue numbers for the receptor...\n",
      "\n",
      "6WZG P47872\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 91%|███████████████████████████████████████████████████████████████████████████████████████████████████████▎          | 505/557 [11:30<01:20,  1.54s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[471] generic residue numbers for the receptor...\n",
      "\n",
      "4U16 P08483\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4U16 P08483\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 91%|███████████████████████████████████████████████████████████████████████████████████████████████████████▌          | 506/557 [11:32<01:17,  1.53s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[121] generic residue numbers for the receptor...\n",
      "\n",
      "7CFM Q8TDU6\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 91%|███████████████████████████████████████████████████████████████████████████████████████████████████████▊          | 507/557 [11:33<01:16,  1.54s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[500] generic residue numbers for the receptor...\n",
      "\n",
      "4IB4 P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "4IB4 P41595\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 91%|███████████████████████████████████████████████████████████████████████████████████████████████████████▉          | 508/557 [11:34<01:05,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[90] generic residue numbers for the receptor...\n",
      "\n",
      "7CFN Q8TDU6\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 91%|████████████████████████████████████████████████████████████████████████████████████████████████████████▏         | 509/557 [11:36<01:07,  1.40s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[501] generic residue numbers for the receptor...\n",
      "\n",
      "7CA5 O75899\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 92%|████████████████████████████████████████████████████████████████████████████████████████████████████████▍         | 510/557 [11:38<01:19,  1.69s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[499] generic residue numbers for the receptor...\n",
      "\n",
      "7L0S P20789\n",
      "Found error!\n",
      "7L0S: Using uniprot (looking up P20789) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for NTR1_RAT\n",
      "start pdb: 5\n",
      "start uni: 50\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 92%|████████████████████████████████████████████████████████████████████████████████████████████████████████▌         | 511/557 [11:39<01:11,  1.55s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[547] generic residue numbers for the receptor...\n",
      "\n",
      "6RZ8 Q9NS75\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6RZ8 Q9NS75\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 92%|████████████████████████████████████████████████████████████████████████████████████████████████████████▊         | 512/557 [11:40<00:59,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[423] generic residue numbers for the receptor...\n",
      "\n",
      "6LPK P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6LPK P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 92%|████████████████████████████████████████████████████████████████████████████████████████████████████████▉         | 513/557 [11:41<00:52,  1.20s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[349] generic residue numbers for the receptor...\n",
      "\n",
      "6W2Y Q9UBS5\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 92%|█████████████████████████████████████████████████████████████████████████████████████████████████████████▏        | 514/557 [11:44<01:11,  1.65s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[459] generic residue numbers for the receptor...\n",
      "\n",
      "3QAK P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3QAK P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 92%|█████████████████████████████████████████████████████████████████████████████████████████████████████████▍        | 515/557 [11:45<01:01,  1.46s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[54] generic residue numbers for the receptor...\n",
      "\n",
      "6H7M P07700\n",
      "Found error!\n",
      "6H7M: Using uniprot (looking up P07700) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB1_MELGA\n",
      "start pdb: 6\n",
      "start uni: 44\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 93%|█████████████████████████████████████████████████████████████████████████████████████████████████████████▌        | 516/557 [11:46<00:56,  1.38s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[303] generic residue numbers for the receptor...\n",
      "\n",
      "7LCI P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 93%|█████████████████████████████████████████████████████████████████████████████████████████████████████████▊        | 517/557 [11:48<01:00,  1.51s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[550] generic residue numbers for the receptor...\n",
      "\n",
      "7CX3 P43116\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 93%|██████████████████████████████████████████████████████████████████████████████████████████████████████████        | 518/557 [11:50<01:04,  1.64s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[511] generic residue numbers for the receptor...\n",
      "\n",
      "6W25 P32245\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6W25 P32245\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 93%|██████████████████████████████████████████████████████████████████████████████████████████████████████████▏       | 519/557 [11:51<00:55,  1.46s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[457] generic residue numbers for the receptor...\n",
      "\n",
      "5C1M P42866\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 93%|██████████████████████████████████████████████████████████████████████████████████████████████████████████▍       | 520/557 [11:51<00:45,  1.24s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[142] generic residue numbers for the receptor...\n",
      "\n",
      "6LMK P47871\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 94%|██████████████████████████████████████████████████████████████████████████████████████████████████████████▋       | 521/557 [11:53<00:51,  1.44s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[344] generic residue numbers for the receptor...\n",
      "\n",
      "7E2X P08908\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 94%|██████████████████████████████████████████████████████████████████████████████████████████████████████████▊       | 522/557 [11:55<00:50,  1.45s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[528] generic residue numbers for the receptor...\n",
      "\n",
      "6KUW P18825\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6KUW P18825\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 94%|███████████████████████████████████████████████████████████████████████████████████████████████████████████       | 523/557 [11:57<00:53,  1.58s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[334] generic residue numbers for the receptor...\n",
      "\n",
      "5UIG P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5UIG P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 94%|███████████████████████████████████████████████████████████████████████████████████████████████████████████▏      | 524/557 [11:58<00:45,  1.36s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[205] generic residue numbers for the receptor...\n",
      "\n",
      "3PDS P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3PDS P07550\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 94%|███████████████████████████████████████████████████████████████████████████████████████████████████████████▍      | 525/557 [11:59<00:40,  1.25s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[50] generic residue numbers for the receptor...\n",
      "\n",
      "7DTU P41180\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 94%|███████████████████████████████████████████████████████████████████████████████████████████████████████████▋      | 526/557 [12:02<00:59,  1.93s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[525] generic residue numbers for the receptor...\n",
      "\n",
      "1GZM P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 95%|███████████████████████████████████████████████████████████████████████████████████████████████████████████▊      | 527/557 [12:04<00:53,  1.79s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[1] generic residue numbers for the receptor...\n",
      "\n",
      "5IUA P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5IUA P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 95%|████████████████████████████████████████████████████████████████████████████████████████████████████████████      | 528/557 [12:04<00:44,  1.53s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[163] generic residue numbers for the receptor...\n",
      "\n",
      "4LDL P07550\n",
      "Found error!\n",
      "4LDL: Using uniprot (looking up P07550) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for ADRB2_HUMAN\n",
      "start pdb: 178\n",
      "start uni: 29\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 95%|████████████████████████████████████████████████████████████████████████████████████████████████████████████▎     | 529/557 [12:05<00:36,  1.30s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[96] generic residue numbers for the receptor...\n",
      "\n",
      "5OLZ P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5OLZ P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 95%|████████████████████████████████████████████████████████████████████████████████████████████████████████████▍     | 530/557 [12:06<00:31,  1.18s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[191] generic residue numbers for the receptor...\n",
      "\n",
      "7L0R P20789\n",
      "Found error!\n",
      "7L0R: Using uniprot (looking up P20789) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for NTR1_RAT\n",
      "start pdb: 5\n",
      "start uni: 50\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 95%|████████████████████████████████████████████████████████████████████████████████████████████████████████████▋     | 531/557 [12:07<00:30,  1.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[546] generic residue numbers for the receptor...\n",
      "\n",
      "6XOX P43220\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 96%|████████████████████████████████████████████████████████████████████████████████████████████████████████████▉     | 532/557 [12:09<00:36,  1.46s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[479] generic residue numbers for the receptor...\n",
      "\n",
      "4BEY P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 96%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████     | 533/557 [12:10<00:29,  1.23s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[73] generic residue numbers for the receptor...\n",
      "\n",
      "6WWZ P51684\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 96%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████▎    | 534/557 [12:12<00:32,  1.43s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[470] generic residue numbers for the receptor...\n",
      "\n",
      "6FK7 P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 96%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████▍    | 535/557 [12:13<00:26,  1.20s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[288] generic residue numbers for the receptor...\n",
      "\n",
      "5OM1 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5OM1 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 96%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████▋    | 536/557 [12:14<00:23,  1.11s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[192] generic residue numbers for the receptor...\n",
      "\n",
      "6A94 P28223\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6A94 P28223\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 96%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████▉    | 537/557 [12:15<00:26,  1.33s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[255] generic residue numbers for the receptor...\n",
      "\n",
      "6VN7 P32241\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 97%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████    | 538/557 [12:17<00:27,  1.46s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[456] generic residue numbers for the receptor...\n",
      "\n",
      "7BU7 P08588\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 97%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████▎   | 539/557 [12:18<00:23,  1.32s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[489] generic residue numbers for the receptor...\n",
      "\n",
      "4JKV Q99835\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 97%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████▌   | 540/557 [12:20<00:23,  1.36s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[92] generic residue numbers for the receptor...\n",
      "\n",
      "6PGS P02699\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 97%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████▋   | 541/557 [12:20<00:18,  1.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[400] generic residue numbers for the receptor...\n",
      "\n",
      "3OE0 P61073\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "3OE0 P61073\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 97%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████▉   | 542/557 [12:21<00:16,  1.12s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[44] generic residue numbers for the receptor...\n",
      "\n",
      "6LI1 Q9Y2T5\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6LI1 Q9Y2T5\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 97%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████▏  | 543/557 [12:22<00:15,  1.07s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[341] generic residue numbers for the receptor...\n",
      "\n",
      "7CA3 O75899\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 98%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████▎  | 544/557 [12:27<00:29,  2.26s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[498] generic residue numbers for the receptor...\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 98%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████▌  | 545/557 [12:28<00:20,  1.67s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[272] generic residue numbers for the receptor...\n",
      "\n",
      "7D76 Q86Y34\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 98%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████▋  | 546/557 [12:30<00:19,  1.78s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[516] generic residue numbers for the receptor...\n",
      "\n",
      "7L0Q P20789\n",
      "Found error!\n",
      "7L0Q: Using uniprot (looking up P20789) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for NTR1_RAT\n",
      "start pdb: 5\n",
      "start uni: 50\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 98%|███████████████████████████████████████████████████████████████████████████████████████████████████████████████▉  | 547/557 [12:31<00:16,  1.61s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[545] generic residue numbers for the receptor...\n",
      "\n",
      "7ARO P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "7ARO P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 98%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████▏ | 548/557 [12:32<00:12,  1.39s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[484] generic residue numbers for the receptor...\n",
      "\n",
      "7JVR P14416\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 99%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████▎ | 549/557 [12:34<00:12,  1.55s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[539] generic residue numbers for the receptor...\n",
      "\n",
      "6HLL P25103\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6HLL P25103\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 99%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████▌ | 550/557 [12:35<00:09,  1.41s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[306] generic residue numbers for the receptor...\n",
      "\n",
      "3UZC P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 99%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████▊ | 551/557 [12:35<00:06,  1.16s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[61] generic residue numbers for the receptor...\n",
      "\n",
      "6OMM P25090\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 99%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████▉ | 552/557 [12:37<00:06,  1.34s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[385] generic residue numbers for the receptor...\n",
      "\n",
      "6JZH P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "6JZH P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 99%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████▏| 553/557 [12:39<00:05,  1.40s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[320] generic residue numbers for the receptor...\n",
      "\n",
      "6MEO P51681\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 99%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████▍| 554/557 [12:40<00:04,  1.39s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[365] generic residue numbers for the receptor...\n",
      "\n",
      "7D7M P35408\n",
      "Found error!\n",
      "7D7M: Using uniprot (looking up P35408) as a reference, due to missmatch in SIFTS!\n",
      "Looking up uniprot fasta sequence for PE2R4_HUMAN\n",
      "start pdb: 5\n",
      "start uni: 4\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████▌| 555/557 [12:41<00:02,  1.41s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Final Error Check...\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[518] generic residue numbers for the receptor...\n",
      "\n",
      "5IU7 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5IU7 P29274\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████▊| 556/557 [12:42<00:01,  1.27s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[161] generic residue numbers for the receptor...\n",
      "\n",
      "5ZKB P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "\n",
      "5ZKB P08172\n",
      "Trying to assign error free uniprotlabels based on SIFTS!\n",
      "intervals : []\n",
      "Error Status: False\n",
      "Final Error Check..."
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 557/557 [12:43<00:00,  1.37s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "intervals : []\n",
      "Error Status: False\n",
      "assigned dfl[249] generic residue numbers for the receptor...\n",
      "Writing files with generic numbers on receptors.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "p.assign_generic_numbers_r(f=p.table, pdb_ids=[], overwrite=True, folder='data/processed/')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "80a4fdff",
   "metadata": {
    "hidden": true
   },
   "source": [
    "The processed dataframes are safed to the specified folder (*data/processed/*, and the file is named '*\\<pdb id>_r.pkl*'."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9cd8f168",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "## GPROTEIN GENERIC RESIDUE NUMBERS"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "97bdb0ef",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Very similar to how assigning generic residue numbers to receptors work, however gproteins are (EVEN) more difficult to number due to a few reasons:\n",
    "- First, gpcrdb does not provide a webservice to query generic position numbers for gproteins.\n",
    "- Second, we do not know which, if any at all, chain of the pdb structure is a gprotein.\n",
    "- Third, there're gproteins in the form of helix-5-like-peptides fused to the gpcr, chimeras, mini-Gproteins, etc\n",
    "- And of course all of the problems we faced with gpcrs in terms of aligning them to the uniprot sequence. (Just worse since often there are no sifts mappings provided at all.)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b2f42f4f",
   "metadata": {
    "hidden": true
   },
   "source": [
    "How did we solve these issues?\n",
    "- First issue: This could be easily solved, since the number of gproteins is far less than that of receptors, and gpcrdb does provide an old excel sheet with generic residue numbers for all of them (the file was downloaded and manually updated from .xls to .xlsx). The file is stored in \"*data/alignments/residue_table.xlsx*\". \n",
    "- Second issue: since there is a limited number of gproteins we do sequence alignments of ALL chains to ALL gprotein (uniprot) sequences and, using a scoring function as well as a threshold, we detect the BEST gprotein fit if it is present.\n",
    "- Third issue: This is not solved per-se, but generally two facts somewhat make our solution work: using alignments to find gproteins does make the type of gprotein we're looking at irrelevant, since IF we manage to find a good enough alignment we simply use that; Also many gproteins are similar, so even if not the correct gprotein is picked from the alignment (often happens in chimeras) due to their overall similarity the results seem consistently fine.\n",
    "- And... Pain."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "ae38d101",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Quick note: The function expects as an input files that have already been processed by assigning generic residue numbers to the receptor (this is useful to check if what we're trying to label is the gpcr chain). Thus you should always either load data from the processed folder with the flag *mode='r'* or if you start by processing raws first run \"*assign_generic_numbers_r*\"."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "b00b7e4d",
   "metadata": {
    "hidden": true
   },
   "source": [
    "To make sure everything you want to assign gprotein-generic-residue-numbers to, let's reinitialize/load our data from the correct folder. Note that the flag 'mode' in the *p.read_pkl()* function can be:\n",
    "- '' / an empty string: This loads the raw dataframes from the raw folder without gen. position numbers (requires *folder='data/raw/'*)!\n",
    "- 'r': This loads the dataframes where generic position numbers have been assigned for the receptors (but not the gproteins)!\n",
    "- 'rg': This loads the dataframes where generic position numbers have been assigned for both gpcr and gprotein!"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 18,
   "id": "2bb50282",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading files with generic numbers on receptors.\n"
     ]
    }
   ],
   "source": [
    "del p\n",
    "p = CifProcessor()\n",
    "p.read_pkl_metainfo()\n",
    "p.read_pkl(mode='r', folder='data/processed/')\n",
    "f_act = p.make_filter(State='Active', Cl='Rhodopsin', gprotein=True)\n",
    "f_act = f_act[f_act['Method']=='cryo-EM']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 19,
   "id": "95397c69",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  0%|                                                                                                                             | 0/67 [00:00<?, ?it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "assinging res_nums to gprot of 6WHA\n",
      "No mapping (no uniprot-seq) information found! ====> Assigning labels ourself!\n",
      "_assign_res_nums_mini_g\n",
      "start_uni 12 > end_uni 58\n",
      "start_pdb 5 > end_pdb 51\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 5 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "start_uni 206 > end_uni 225\n",
      "start_pdb 68 > end_pdb 87\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 68 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 69 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 70 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 71 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 72 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 73 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 74 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 75 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 76 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 77 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 78 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 79 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 80 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 81 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 82 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 83 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 84 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 85 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 86 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 87 <> gen pos G.s3h2.01\n",
      "start_uni 231 > end_uni 253\n",
      "start_pdb 93 > end_pdb 115\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 93 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 94 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 95 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 96 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 97 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 98 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 99 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 100 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 101 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 102 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 103 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 104 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 105 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 106 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 107 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 108 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 109 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 110 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 111 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 112 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 113 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 114 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 115 <> gen pos G.s4h3.04\n",
      "start_uni 264 > end_uni 321\n",
      "start_pdb 116 > end_pdb 173\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 116 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 117 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 118 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 119 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 120 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 121 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 122 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 123 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 124 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 125 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 126 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 127 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 128 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 129 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 130 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 131 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 132 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 133 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 134 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 135 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 136 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 137 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 138 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 139 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 140 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 141 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 142 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 143 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 144 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 145 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 146 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 147 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 148 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 149 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 150 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 151 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 152 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 153 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 154 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 155 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 156 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 157 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 158 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 159 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 160 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 161 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 162 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 163 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 164 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 165 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 166 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 167 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 168 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 169 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 170 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 171 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 172 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 173 <> gen pos G.hgh4.11\n",
      "start_uni 331 > end_uni 365\n",
      "start_pdb 183 > end_pdb 217\n",
      "mini g setting index += 1\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 183 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 184 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 185 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 186 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 187 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 188 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 189 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 190 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 191 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 192 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 193 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 194 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 195 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 196 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 197 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 198 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 199 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 200 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 201 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 202 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 203 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 204 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 205 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 206 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 207 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 208 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 209 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 210 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 211 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 212 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 213 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 214 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 215 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 216 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 217 <> gen pos G.s6h5.02\n",
      "start_uni 367 > end_uni 394\n",
      "start_pdb 219 > end_pdb 246\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 219 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 220 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 221 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 222 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 223 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 224 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 225 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 226 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 227 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 228 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 229 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 230 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 231 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 232 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 233 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 234 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 235 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 236 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 237 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 238 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 239 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 240 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 241 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 242 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 243 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 244 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 245 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 246 <> gen pos G.H5.26"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  1%|█▋                                                                                                                   | 1/67 [00:01<02:04,  1.89s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "assinging res_nums to gprot of 7L1U\n",
      "No mapping (no uniprot-seq) information found! ====> Assigning labels ourself!\n",
      "_assign_res_nums_mini_g\n",
      "start_uni 11 > end_uni 60\n",
      "start_pdb 2 > end_pdb 51\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 2 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 3 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 4 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 5 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 6 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 7 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 8 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 9 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 10 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 11 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 12 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 13 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 14 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 15 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 16 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 17 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 18 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 19 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 20 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 21 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 22 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 23 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 24 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 25 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 26 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 27 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 28 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 29 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 30 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 31 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 32 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 33 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 34 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 35 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 36 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 37 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 38 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 39 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 40 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 41 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 42 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 43 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 44 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 45 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 46 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 47 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 48 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 49 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 50 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 51 <> gen pos G.H1.08\n",
      "start_uni 207 > end_uni 253\n",
      "start_pdb 67 > end_pdb 113\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 67 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 68 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 69 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 70 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 71 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 72 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 73 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 74 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 75 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 76 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 77 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 78 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 79 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 80 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 81 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 82 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 83 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 84 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 85 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 86 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 87 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 88 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 89 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 90 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 91 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 92 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 93 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 94 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 95 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 96 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 97 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 98 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 99 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 100 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 101 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 102 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 103 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 104 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 105 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 106 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 107 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 108 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 109 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 110 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 111 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 112 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 113 <> gen pos G.s4h3.04\n",
      "start_uni 264 > end_uni 394\n",
      "start_pdb 114 > end_pdb 244\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 114 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 115 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 116 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 117 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 118 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 119 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 120 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 121 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 122 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 123 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 124 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 125 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 126 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 127 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 128 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 129 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 130 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 131 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 132 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 133 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 134 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 135 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 136 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 137 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 138 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 139 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 140 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 141 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 142 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 143 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 144 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 145 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 146 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 147 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 148 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 149 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 150 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 151 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 152 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 153 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 154 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 155 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 156 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 157 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 158 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 159 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 160 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 161 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 162 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 163 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 164 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 165 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 166 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 167 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 168 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 169 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 170 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 171 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 172 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 173 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 174 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 175 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 176 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 177 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 178 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 179 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 180 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 181 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 182 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 183 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 184 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 185 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 186 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 187 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 188 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 189 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 190 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 191 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 192 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 193 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 194 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 195 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 196 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 197 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 198 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 199 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 200 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 201 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 202 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 203 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 204 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 205 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 206 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 207 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 208 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 209 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 210 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 211 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 212 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 213 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 214 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 215 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 216 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 217 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 218 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 219 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 220 <> gen pos G.H5.02"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  3%|███▍                                                                                                                 | 2/67 [00:03<01:55,  1.78s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 221 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 222 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 223 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 224 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 225 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 226 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 227 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 228 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 229 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 230 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 231 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 232 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 233 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 234 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 235 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 236 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 237 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 238 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 239 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 240 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 241 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 242 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 243 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 244 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7E32\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  4%|█████▏                                                                                                               | 3/67 [00:04<01:22,  1.28s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7JV5\n",
      "Mapping found!\n",
      "start_uni 2 > end_uni 394\n",
      "start_pdb 1 > end_pdb 393\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 8 <> gen pos G.HN.26\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 9 <> gen pos G.HN.27\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 10 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 11 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 12 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 13 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 14 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 15 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 16 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 17 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 18 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 19 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 20 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 21 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 22 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 23 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 24 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 25 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 26 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 27 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 28 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 29 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 30 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 31 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 32 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 33 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 34 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 35 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 36 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 37 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 38 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 39 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 40 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 41 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 42 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 43 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 44 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 45 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 46 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 47 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 48 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 49 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 50 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 51 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 52 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 53 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 54 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 55 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 56 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 57 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 58 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 59 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 60 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 61 <> gen pos G.H1.10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 203 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 204 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 205 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 206 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 207 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 208 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 209 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 210 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 211 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 212 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 213 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 214 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 215 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 216 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 217 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 218 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 219 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 220 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 221 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 222 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 223 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 224 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 225 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 226 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 227 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 228 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 229 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 230 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 231 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 232 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 233 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 234 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 235 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 236 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 237 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 238 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 239 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 240 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 241 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 242 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 243 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 244 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 245 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 246 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 247 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 248 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 249 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 250 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 251 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 252 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 260 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 261 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 262 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 263 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 264 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 265 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 266 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 267 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 268 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 269 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 270 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 271 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 272 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 273 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 274 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 275 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 276 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 277 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 278 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 279 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 280 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 281 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 282 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 283 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 284 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 285 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 286 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 287 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 288 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 289 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 290 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 291 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 292 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 293 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 294 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 295 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 296 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 297 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 298 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 299 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 300 <> gen pos G.HG.08"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  6%|██████▉                                                                                                              | 4/67 [00:05<01:07,  1.07s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 301 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 302 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 303 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 304 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 305 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 306 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 307 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 308 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 309 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 310 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 311 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 312 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 313 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 314 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 315 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 316 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 317 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 318 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 319 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 320 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 321 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 322 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 323 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 324 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 325 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 326 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 327 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 328 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 329 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 330 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 331 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 332 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 333 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 334 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 335 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 336 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 337 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 338 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 339 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 340 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 341 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 342 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 343 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 344 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 345 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 346 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 347 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 348 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 349 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 350 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 351 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 352 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 353 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 354 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 355 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 356 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 357 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 358 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 359 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 360 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 361 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 362 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 363 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 364 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 365 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 366 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 367 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 368 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 369 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 370 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 371 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 372 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 373 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 374 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 375 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 376 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 377 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 378 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 379 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 380 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 381 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 382 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 383 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 384 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 385 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 386 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 387 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 388 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 389 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 390 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 391 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 392 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 393 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6WWZ\n",
      "Mapping found!\n",
      "start_uni 4 > end_uni 57\n",
      "start_pdb 16 > end_pdb 69\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 17 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 18 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 19 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 20 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 21 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 22 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 23 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 24 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 25 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 26 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 27 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 28 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 29 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 30 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 31 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 32 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 33 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 34 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 35 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 36 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 37 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 38 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 39 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 40 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 41 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 42 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 43 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 44 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 45 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 46 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 47 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 48 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 49 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 50 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 51 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 52 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 53 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 54 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 55 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 56 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 57 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 58 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 59 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 60 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 61 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 62 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 63 <> gen pos G.H1.06\n",
      "start_uni 182 > end_uni 354\n",
      "start_pdb 78 > end_pdb 250\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 79 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 80 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 81 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 82 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 83 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 84 <> gen pos G.S2.04\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 85 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 86 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 87 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 88 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 89 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 90 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 91 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 92 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 93 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 94 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 95 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 96 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 97 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 98 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 99 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 100 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 101 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 102 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 103 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 104 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 105 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 106 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 107 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 108 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 109 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 110 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 111 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 112 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 113 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 114 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 115 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 116 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 117 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 118 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 119 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 120 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 121 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 122 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 123 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 124 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 125 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 126 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 127 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 138 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 139 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 140 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 141 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 142 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 143 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 144 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 145 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 146 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 147 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 148 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 149 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 150 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 151 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 152 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 153 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 154 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 155 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 156 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 157 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 158 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 159 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 160 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 161 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 162 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 163 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 164 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 165 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 166 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 167 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 168 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 169 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 170 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 171 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 172 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 173 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 174 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 175 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 176 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 177 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 178 <> gen pos G.HG.13"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  7%|████████▋                                                                                                            | 5/67 [00:05<00:55,  1.11it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 179 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 180 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 181 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 182 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 183 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 184 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 185 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 186 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 187 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 188 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 189 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 190 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 191 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 192 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 193 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 194 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 195 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 196 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 197 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 198 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 199 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 200 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 201 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 202 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 203 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 204 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 205 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 206 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 207 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 208 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 209 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 210 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 211 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 212 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 213 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 214 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 215 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 216 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 217 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 218 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 219 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 220 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 221 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 222 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 223 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 224 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 225 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 226 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 227 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 228 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 229 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 230 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 231 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 232 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 233 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 234 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 235 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 236 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 237 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 238 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 239 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 240 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 241 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 242 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 243 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 244 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 245 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 246 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 247 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 248 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 249 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 250 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7CX3\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 394\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.HG.13"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      "  9%|██████████▍                                                                                                          | 6/67 [00:06<00:51,  1.19it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 365 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 366 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 367 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 368 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 369 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 370 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 381 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 382 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 383 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 384 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 385 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 386 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 387 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 388 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 389 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 390 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 391 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 392 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 393 <> gen pos G.H5.25\n",
      "assinging res_nums to gprot of 7CKW\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 394\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 61 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 62 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 63 <> gen pos G.H1.11\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.HG.14"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 10%|████████████▏                                                                                                        | 7/67 [00:07<00:48,  1.24it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 365 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 366 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 367 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 368 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 369 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 370 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 381 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 382 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 383 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 384 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 385 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 386 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 387 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 388 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 389 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 390 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 391 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 392 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 393 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 394 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6KPF\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 3 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 12%|█████████████▉                                                                                                       | 8/67 [00:07<00:45,  1.30it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7L0Q\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 1 <> idx_seq: 2 <> gen pos G.HN.02\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 3 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 63 <> idx_seq: 64 <> gen pos H.HA.02\n",
      "Found corr. gen num! idx_uni: 64 <> idx_seq: 65 <> gen pos H.HA.03\n",
      "Found corr. gen num! idx_uni: 65 <> idx_seq: 66 <> gen pos H.HA.04\n",
      "Found corr. gen num! idx_uni: 66 <> idx_seq: 67 <> gen pos H.HA.05\n",
      "Found corr. gen num! idx_uni: 67 <> idx_seq: 68 <> gen pos H.HA.06\n",
      "Found corr. gen num! idx_uni: 68 <> idx_seq: 69 <> gen pos H.HA.07\n",
      "Found corr. gen num! idx_uni: 69 <> idx_seq: 70 <> gen pos H.HA.08\n",
      "Found corr. gen num! idx_uni: 70 <> idx_seq: 71 <> gen pos H.HA.09\n",
      "Found corr. gen num! idx_uni: 71 <> idx_seq: 72 <> gen pos H.HA.10\n",
      "Found corr. gen num! idx_uni: 72 <> idx_seq: 73 <> gen pos H.HA.11\n",
      "Found corr. gen num! idx_uni: 73 <> idx_seq: 74 <> gen pos H.HA.12\n",
      "Found corr. gen num! idx_uni: 74 <> idx_seq: 75 <> gen pos H.HA.13\n",
      "Found corr. gen num! idx_uni: 75 <> idx_seq: 76 <> gen pos H.HA.14\n",
      "Found corr. gen num! idx_uni: 76 <> idx_seq: 77 <> gen pos H.HA.15\n",
      "Found corr. gen num! idx_uni: 77 <> idx_seq: 78 <> gen pos H.HA.16\n",
      "Found corr. gen num! idx_uni: 78 <> idx_seq: 79 <> gen pos H.HA.17\n",
      "Found corr. gen num! idx_uni: 79 <> idx_seq: 80 <> gen pos H.HA.18\n",
      "Found corr. gen num! idx_uni: 80 <> idx_seq: 81 <> gen pos H.HA.19\n",
      "Found corr. gen num! idx_uni: 81 <> idx_seq: 82 <> gen pos H.HA.20\n",
      "Found corr. gen num! idx_uni: 82 <> idx_seq: 83 <> gen pos H.HA.21\n",
      "Found corr. gen num! idx_uni: 83 <> idx_seq: 84 <> gen pos H.HA.22\n",
      "Found corr. gen num! idx_uni: 84 <> idx_seq: 85 <> gen pos H.HA.23\n",
      "Found corr. gen num! idx_uni: 85 <> idx_seq: 86 <> gen pos H.HA.24\n",
      "Found corr. gen num! idx_uni: 86 <> idx_seq: 87 <> gen pos H.HA.25\n",
      "Found corr. gen num! idx_uni: 87 <> idx_seq: 88 <> gen pos H.HA.26\n",
      "Found corr. gen num! idx_uni: 88 <> idx_seq: 89 <> gen pos H.HA.27\n",
      "Found corr. gen num! idx_uni: 89 <> idx_seq: 90 <> gen pos H.HA.28\n",
      "Found corr. gen num! idx_uni: 90 <> idx_seq: 91 <> gen pos H.HA.29\n",
      "Found corr. gen num! idx_uni: 91 <> idx_seq: 92 <> gen pos H.hahb.01\n",
      "Found corr. gen num! idx_uni: 92 <> idx_seq: 93 <> gen pos H.hahb.04\n",
      "Found corr. gen num! idx_uni: 93 <> idx_seq: 94 <> gen pos H.hahb.05\n",
      "Found corr. gen num! idx_uni: 94 <> idx_seq: 95 <> gen pos H.hahb.06\n",
      "Found corr. gen num! idx_uni: 95 <> idx_seq: 96 <> gen pos H.hahb.07\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 96 <> idx_seq: 97 <> gen pos H.hahb.08\n",
      "Found corr. gen num! idx_uni: 97 <> idx_seq: 98 <> gen pos H.hahb.09\n",
      "Found corr. gen num! idx_uni: 98 <> idx_seq: 99 <> gen pos H.HB.01\n",
      "Found corr. gen num! idx_uni: 99 <> idx_seq: 100 <> gen pos H.HB.02\n",
      "Found corr. gen num! idx_uni: 100 <> idx_seq: 101 <> gen pos H.HB.03\n",
      "Found corr. gen num! idx_uni: 101 <> idx_seq: 102 <> gen pos H.HB.04\n",
      "Found corr. gen num! idx_uni: 102 <> idx_seq: 103 <> gen pos H.HB.05\n",
      "Found corr. gen num! idx_uni: 103 <> idx_seq: 104 <> gen pos H.HB.06\n",
      "Found corr. gen num! idx_uni: 104 <> idx_seq: 105 <> gen pos H.HB.07\n",
      "Found corr. gen num! idx_uni: 105 <> idx_seq: 106 <> gen pos H.HB.08\n",
      "Found corr. gen num! idx_uni: 106 <> idx_seq: 107 <> gen pos H.HB.09\n",
      "Found corr. gen num! idx_uni: 107 <> idx_seq: 108 <> gen pos H.HB.10\n",
      "Found corr. gen num! idx_uni: 108 <> idx_seq: 109 <> gen pos H.HB.11\n",
      "Found corr. gen num! idx_uni: 109 <> idx_seq: 110 <> gen pos H.HB.12\n",
      "Found corr. gen num! idx_uni: 110 <> idx_seq: 111 <> gen pos H.HB.13\n",
      "Found corr. gen num! idx_uni: 111 <> idx_seq: 112 <> gen pos H.HB.14\n",
      "Found corr. gen num! idx_uni: 112 <> idx_seq: 113 <> gen pos H.hbhc.01\n",
      "Found corr. gen num! idx_uni: 113 <> idx_seq: 114 <> gen pos H.hbhc.02\n",
      "Found corr. gen num! idx_uni: 114 <> idx_seq: 115 <> gen pos H.hbhc.03\n",
      "Found corr. gen num! idx_uni: 115 <> idx_seq: 116 <> gen pos H.hbhc.04\n",
      "Found corr. gen num! idx_uni: 116 <> idx_seq: 117 <> gen pos H.hbhc.12\n",
      "Found corr. gen num! idx_uni: 117 <> idx_seq: 118 <> gen pos H.hbhc.13\n",
      "Found corr. gen num! idx_uni: 118 <> idx_seq: 119 <> gen pos H.hbhc.14\n",
      "Found corr. gen num! idx_uni: 119 <> idx_seq: 120 <> gen pos H.hbhc.15\n",
      "Found corr. gen num! idx_uni: 120 <> idx_seq: 121 <> gen pos H.HC.01\n",
      "Found corr. gen num! idx_uni: 121 <> idx_seq: 122 <> gen pos H.HC.02\n",
      "Found corr. gen num! idx_uni: 122 <> idx_seq: 123 <> gen pos H.HC.03\n",
      "Found corr. gen num! idx_uni: 123 <> idx_seq: 124 <> gen pos H.HC.04\n",
      "Found corr. gen num! idx_uni: 124 <> idx_seq: 125 <> gen pos H.HC.05\n",
      "Found corr. gen num! idx_uni: 125 <> idx_seq: 126 <> gen pos H.HC.06\n",
      "Found corr. gen num! idx_uni: 126 <> idx_seq: 127 <> gen pos H.HC.07\n",
      "Found corr. gen num! idx_uni: 127 <> idx_seq: 128 <> gen pos H.HC.08\n",
      "Found corr. gen num! idx_uni: 128 <> idx_seq: 129 <> gen pos H.HC.09\n",
      "Found corr. gen num! idx_uni: 129 <> idx_seq: 130 <> gen pos H.HC.10\n",
      "Found corr. gen num! idx_uni: 130 <> idx_seq: 131 <> gen pos H.HC.11\n",
      "Found corr. gen num! idx_uni: 131 <> idx_seq: 132 <> gen pos H.HC.12\n",
      "Found corr. gen num! idx_uni: 132 <> idx_seq: 133 <> gen pos H.hchd.01\n",
      "Found corr. gen num! idx_uni: 133 <> idx_seq: 134 <> gen pos H.HD.01\n",
      "Found corr. gen num! idx_uni: 134 <> idx_seq: 135 <> gen pos H.HD.02\n",
      "Found corr. gen num! idx_uni: 135 <> idx_seq: 136 <> gen pos H.HD.03\n",
      "Found corr. gen num! idx_uni: 136 <> idx_seq: 137 <> gen pos H.HD.04\n",
      "Found corr. gen num! idx_uni: 137 <> idx_seq: 138 <> gen pos H.HD.05\n",
      "Found corr. gen num! idx_uni: 138 <> idx_seq: 139 <> gen pos H.HD.06\n",
      "Found corr. gen num! idx_uni: 139 <> idx_seq: 140 <> gen pos H.HD.07\n",
      "Found corr. gen num! idx_uni: 140 <> idx_seq: 141 <> gen pos H.HD.08\n",
      "Found corr. gen num! idx_uni: 141 <> idx_seq: 142 <> gen pos H.HD.09\n",
      "Found corr. gen num! idx_uni: 142 <> idx_seq: 143 <> gen pos H.HD.10\n",
      "Found corr. gen num! idx_uni: 143 <> idx_seq: 144 <> gen pos H.HD.11\n",
      "Found corr. gen num! idx_uni: 144 <> idx_seq: 145 <> gen pos H.HD.12\n",
      "Found corr. gen num! idx_uni: 145 <> idx_seq: 146 <> gen pos H.hdhe.01\n",
      "Found corr. gen num! idx_uni: 146 <> idx_seq: 147 <> gen pos H.hdhe.02\n",
      "Found corr. gen num! idx_uni: 147 <> idx_seq: 148 <> gen pos H.hdhe.03\n",
      "Found corr. gen num! idx_uni: 148 <> idx_seq: 149 <> gen pos H.hdhe.04\n",
      "Found corr. gen num! idx_uni: 149 <> idx_seq: 150 <> gen pos H.hdhe.05\n",
      "Found corr. gen num! idx_uni: 150 <> idx_seq: 151 <> gen pos H.HE.01\n",
      "Found corr. gen num! idx_uni: 151 <> idx_seq: 152 <> gen pos H.HE.02\n",
      "Found corr. gen num! idx_uni: 152 <> idx_seq: 153 <> gen pos H.HE.03\n",
      "Found corr. gen num! idx_uni: 153 <> idx_seq: 154 <> gen pos H.HE.04\n",
      "Found corr. gen num! idx_uni: 154 <> idx_seq: 155 <> gen pos H.HE.05\n",
      "Found corr. gen num! idx_uni: 155 <> idx_seq: 156 <> gen pos H.HE.06\n",
      "Found corr. gen num! idx_uni: 156 <> idx_seq: 157 <> gen pos H.HE.07\n",
      "Found corr. gen num! idx_uni: 157 <> idx_seq: 158 <> gen pos H.HE.08\n",
      "Found corr. gen num! idx_uni: 158 <> idx_seq: 159 <> gen pos H.HE.09\n",
      "Found corr. gen num! idx_uni: 159 <> idx_seq: 160 <> gen pos H.HE.10\n",
      "Found corr. gen num! idx_uni: 160 <> idx_seq: 161 <> gen pos H.HE.11\n",
      "Found corr. gen num! idx_uni: 161 <> idx_seq: 162 <> gen pos H.HE.12\n",
      "Found corr. gen num! idx_uni: 162 <> idx_seq: 163 <> gen pos H.HE.13\n",
      "Found corr. gen num! idx_uni: 163 <> idx_seq: 164 <> gen pos H.hehf.01\n",
      "Found corr. gen num! idx_uni: 164 <> idx_seq: 165 <> gen pos H.hehf.02\n",
      "Found corr. gen num! idx_uni: 165 <> idx_seq: 166 <> gen pos H.hehf.03\n",
      "Found corr. gen num! idx_uni: 166 <> idx_seq: 167 <> gen pos H.hehf.04\n",
      "Found corr. gen num! idx_uni: 167 <> idx_seq: 168 <> gen pos H.hehf.05\n",
      "Found corr. gen num! idx_uni: 168 <> idx_seq: 169 <> gen pos H.hehf.06\n",
      "Found corr. gen num! idx_uni: 169 <> idx_seq: 170 <> gen pos H.hehf.07\n",
      "Found corr. gen num! idx_uni: 170 <> idx_seq: 171 <> gen pos H.HF.01\n",
      "Found corr. gen num! idx_uni: 171 <> idx_seq: 172 <> gen pos H.HF.02\n",
      "Found corr. gen num! idx_uni: 172 <> idx_seq: 173 <> gen pos H.HF.03\n",
      "Found corr. gen num! idx_uni: 173 <> idx_seq: 174 <> gen pos H.HF.04\n",
      "Found corr. gen num! idx_uni: 174 <> idx_seq: 175 <> gen pos H.HF.05\n",
      "Found corr. gen num! idx_uni: 175 <> idx_seq: 176 <> gen pos H.HF.06\n",
      "Found corr. gen num! idx_uni: 176 <> idx_seq: 177 <> gen pos G.hfs2.01\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 13%|███████████████▋                                                                                                     | 9/67 [00:08<00:43,  1.33it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7DHR\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 394\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 15%|█████████████████▎                                                                                                  | 10/67 [00:09<00:41,  1.38it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 381 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 382 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 383 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 384 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 385 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 386 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 387 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 388 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 389 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 390 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 391 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 392 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 393 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 394 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6CMO\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 1 <> idx_seq: 2 <> gen pos G.HN.02\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 3 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.12\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.h1ha.01\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.h1ha.02\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.h1ha.03\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 61 <> gen pos G.h1ha.04\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 62 <> gen pos G.h1ha.05\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 63 <> gen pos H.HA.01\n",
      "Found corr. gen num! idx_uni: 63 <> idx_seq: 64 <> gen pos H.HA.02\n",
      "Found corr. gen num! idx_uni: 64 <> idx_seq: 65 <> gen pos H.HA.03\n",
      "Found corr. gen num! idx_uni: 65 <> idx_seq: 66 <> gen pos H.HA.04\n",
      "Found corr. gen num! idx_uni: 66 <> idx_seq: 67 <> gen pos H.HA.05\n",
      "Found corr. gen num! idx_uni: 67 <> idx_seq: 68 <> gen pos H.HA.06\n",
      "Found corr. gen num! idx_uni: 68 <> idx_seq: 69 <> gen pos H.HA.07\n",
      "Found corr. gen num! idx_uni: 69 <> idx_seq: 70 <> gen pos H.HA.08\n",
      "Found corr. gen num! idx_uni: 70 <> idx_seq: 71 <> gen pos H.HA.09\n",
      "Found corr. gen num! idx_uni: 71 <> idx_seq: 72 <> gen pos H.HA.10\n",
      "Found corr. gen num! idx_uni: 72 <> idx_seq: 73 <> gen pos H.HA.11\n",
      "Found corr. gen num! idx_uni: 73 <> idx_seq: 74 <> gen pos H.HA.12\n",
      "Found corr. gen num! idx_uni: 74 <> idx_seq: 75 <> gen pos H.HA.13\n",
      "Found corr. gen num! idx_uni: 75 <> idx_seq: 76 <> gen pos H.HA.14\n",
      "Found corr. gen num! idx_uni: 76 <> idx_seq: 77 <> gen pos H.HA.15\n",
      "Found corr. gen num! idx_uni: 77 <> idx_seq: 78 <> gen pos H.HA.16\n",
      "Found corr. gen num! idx_uni: 78 <> idx_seq: 79 <> gen pos H.HA.17\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 79 <> idx_seq: 80 <> gen pos H.HA.18\n",
      "Found corr. gen num! idx_uni: 80 <> idx_seq: 81 <> gen pos H.HA.19\n",
      "Found corr. gen num! idx_uni: 81 <> idx_seq: 82 <> gen pos H.HA.20\n",
      "Found corr. gen num! idx_uni: 82 <> idx_seq: 83 <> gen pos H.HA.21\n",
      "Found corr. gen num! idx_uni: 83 <> idx_seq: 84 <> gen pos H.HA.22\n",
      "Found corr. gen num! idx_uni: 84 <> idx_seq: 85 <> gen pos H.HA.23\n",
      "Found corr. gen num! idx_uni: 85 <> idx_seq: 86 <> gen pos H.HA.24\n",
      "Found corr. gen num! idx_uni: 86 <> idx_seq: 87 <> gen pos H.HA.25\n",
      "Found corr. gen num! idx_uni: 87 <> idx_seq: 88 <> gen pos H.HA.26\n",
      "Found corr. gen num! idx_uni: 88 <> idx_seq: 89 <> gen pos H.HA.27\n",
      "Found corr. gen num! idx_uni: 89 <> idx_seq: 90 <> gen pos H.HA.28\n",
      "Found corr. gen num! idx_uni: 90 <> idx_seq: 91 <> gen pos H.HA.29\n",
      "Found corr. gen num! idx_uni: 91 <> idx_seq: 92 <> gen pos H.hahb.01\n",
      "Found corr. gen num! idx_uni: 92 <> idx_seq: 93 <> gen pos H.hahb.04\n",
      "Found corr. gen num! idx_uni: 93 <> idx_seq: 94 <> gen pos H.hahb.05\n",
      "Found corr. gen num! idx_uni: 94 <> idx_seq: 95 <> gen pos H.hahb.06\n",
      "Found corr. gen num! idx_uni: 95 <> idx_seq: 96 <> gen pos H.hahb.07\n",
      "Found corr. gen num! idx_uni: 96 <> idx_seq: 97 <> gen pos H.hahb.08\n",
      "Found corr. gen num! idx_uni: 97 <> idx_seq: 98 <> gen pos H.hahb.09\n",
      "Found corr. gen num! idx_uni: 98 <> idx_seq: 99 <> gen pos H.HB.01\n",
      "Found corr. gen num! idx_uni: 99 <> idx_seq: 100 <> gen pos H.HB.02\n",
      "Found corr. gen num! idx_uni: 100 <> idx_seq: 101 <> gen pos H.HB.03\n",
      "Found corr. gen num! idx_uni: 101 <> idx_seq: 102 <> gen pos H.HB.04\n",
      "Found corr. gen num! idx_uni: 102 <> idx_seq: 103 <> gen pos H.HB.05\n",
      "Found corr. gen num! idx_uni: 103 <> idx_seq: 104 <> gen pos H.HB.06\n",
      "Found corr. gen num! idx_uni: 104 <> idx_seq: 105 <> gen pos H.HB.07\n",
      "Found corr. gen num! idx_uni: 105 <> idx_seq: 106 <> gen pos H.HB.08\n",
      "Found corr. gen num! idx_uni: 106 <> idx_seq: 107 <> gen pos H.HB.09\n",
      "Found corr. gen num! idx_uni: 107 <> idx_seq: 108 <> gen pos H.HB.10\n",
      "Found corr. gen num! idx_uni: 108 <> idx_seq: 109 <> gen pos H.HB.11\n",
      "Found corr. gen num! idx_uni: 109 <> idx_seq: 110 <> gen pos H.HB.12\n",
      "Found corr. gen num! idx_uni: 110 <> idx_seq: 111 <> gen pos H.HB.13\n",
      "Found corr. gen num! idx_uni: 111 <> idx_seq: 112 <> gen pos H.HB.14\n",
      "Found corr. gen num! idx_uni: 112 <> idx_seq: 113 <> gen pos H.hbhc.01\n",
      "Found corr. gen num! idx_uni: 113 <> idx_seq: 114 <> gen pos H.hbhc.02\n",
      "Found corr. gen num! idx_uni: 114 <> idx_seq: 115 <> gen pos H.hbhc.03\n",
      "Found corr. gen num! idx_uni: 115 <> idx_seq: 116 <> gen pos H.hbhc.04\n",
      "Found corr. gen num! idx_uni: 116 <> idx_seq: 117 <> gen pos H.hbhc.12\n",
      "Found corr. gen num! idx_uni: 117 <> idx_seq: 118 <> gen pos H.hbhc.13\n",
      "Found corr. gen num! idx_uni: 118 <> idx_seq: 119 <> gen pos H.hbhc.14\n",
      "Found corr. gen num! idx_uni: 119 <> idx_seq: 120 <> gen pos H.hbhc.15\n",
      "Found corr. gen num! idx_uni: 120 <> idx_seq: 121 <> gen pos H.HC.01\n",
      "Found corr. gen num! idx_uni: 121 <> idx_seq: 122 <> gen pos H.HC.02\n",
      "Found corr. gen num! idx_uni: 122 <> idx_seq: 123 <> gen pos H.HC.03\n",
      "Found corr. gen num! idx_uni: 123 <> idx_seq: 124 <> gen pos H.HC.04\n",
      "Found corr. gen num! idx_uni: 124 <> idx_seq: 125 <> gen pos H.HC.05\n",
      "Found corr. gen num! idx_uni: 125 <> idx_seq: 126 <> gen pos H.HC.06\n",
      "Found corr. gen num! idx_uni: 126 <> idx_seq: 127 <> gen pos H.HC.07\n",
      "Found corr. gen num! idx_uni: 127 <> idx_seq: 128 <> gen pos H.HC.08\n",
      "Found corr. gen num! idx_uni: 128 <> idx_seq: 129 <> gen pos H.HC.09\n",
      "Found corr. gen num! idx_uni: 129 <> idx_seq: 130 <> gen pos H.HC.10\n",
      "Found corr. gen num! idx_uni: 130 <> idx_seq: 131 <> gen pos H.HC.11\n",
      "Found corr. gen num! idx_uni: 131 <> idx_seq: 132 <> gen pos H.HC.12\n",
      "Found corr. gen num! idx_uni: 132 <> idx_seq: 133 <> gen pos H.hchd.01\n",
      "Found corr. gen num! idx_uni: 133 <> idx_seq: 134 <> gen pos H.HD.01\n",
      "Found corr. gen num! idx_uni: 134 <> idx_seq: 135 <> gen pos H.HD.02\n",
      "Found corr. gen num! idx_uni: 135 <> idx_seq: 136 <> gen pos H.HD.03\n",
      "Found corr. gen num! idx_uni: 136 <> idx_seq: 137 <> gen pos H.HD.04\n",
      "Found corr. gen num! idx_uni: 137 <> idx_seq: 138 <> gen pos H.HD.05\n",
      "Found corr. gen num! idx_uni: 138 <> idx_seq: 139 <> gen pos H.HD.06\n",
      "Found corr. gen num! idx_uni: 139 <> idx_seq: 140 <> gen pos H.HD.07\n",
      "Found corr. gen num! idx_uni: 140 <> idx_seq: 141 <> gen pos H.HD.08\n",
      "Found corr. gen num! idx_uni: 141 <> idx_seq: 142 <> gen pos H.HD.09\n",
      "Found corr. gen num! idx_uni: 142 <> idx_seq: 143 <> gen pos H.HD.10\n",
      "Found corr. gen num! idx_uni: 143 <> idx_seq: 144 <> gen pos H.HD.11\n",
      "Found corr. gen num! idx_uni: 144 <> idx_seq: 145 <> gen pos H.HD.12\n",
      "Found corr. gen num! idx_uni: 145 <> idx_seq: 146 <> gen pos H.hdhe.01\n",
      "Found corr. gen num! idx_uni: 146 <> idx_seq: 147 <> gen pos H.hdhe.02\n",
      "Found corr. gen num! idx_uni: 147 <> idx_seq: 148 <> gen pos H.hdhe.03\n",
      "Found corr. gen num! idx_uni: 148 <> idx_seq: 149 <> gen pos H.hdhe.04\n",
      "Found corr. gen num! idx_uni: 149 <> idx_seq: 150 <> gen pos H.hdhe.05\n",
      "Found corr. gen num! idx_uni: 150 <> idx_seq: 151 <> gen pos H.HE.01\n",
      "Found corr. gen num! idx_uni: 151 <> idx_seq: 152 <> gen pos H.HE.02\n",
      "Found corr. gen num! idx_uni: 152 <> idx_seq: 153 <> gen pos H.HE.03\n",
      "Found corr. gen num! idx_uni: 153 <> idx_seq: 154 <> gen pos H.HE.04\n",
      "Found corr. gen num! idx_uni: 154 <> idx_seq: 155 <> gen pos H.HE.05\n",
      "Found corr. gen num! idx_uni: 155 <> idx_seq: 156 <> gen pos H.HE.06\n",
      "Found corr. gen num! idx_uni: 156 <> idx_seq: 157 <> gen pos H.HE.07\n",
      "Found corr. gen num! idx_uni: 157 <> idx_seq: 158 <> gen pos H.HE.08\n",
      "Found corr. gen num! idx_uni: 158 <> idx_seq: 159 <> gen pos H.HE.09\n",
      "Found corr. gen num! idx_uni: 159 <> idx_seq: 160 <> gen pos H.HE.10\n",
      "Found corr. gen num! idx_uni: 160 <> idx_seq: 161 <> gen pos H.HE.11\n",
      "Found corr. gen num! idx_uni: 161 <> idx_seq: 162 <> gen pos H.HE.12\n",
      "Found corr. gen num! idx_uni: 162 <> idx_seq: 163 <> gen pos H.HE.13\n",
      "Found corr. gen num! idx_uni: 163 <> idx_seq: 164 <> gen pos H.hehf.01\n",
      "Found corr. gen num! idx_uni: 164 <> idx_seq: 165 <> gen pos H.hehf.02\n",
      "Found corr. gen num! idx_uni: 165 <> idx_seq: 166 <> gen pos H.hehf.03\n",
      "Found corr. gen num! idx_uni: 166 <> idx_seq: 167 <> gen pos H.hehf.04\n",
      "Found corr. gen num! idx_uni: 167 <> idx_seq: 168 <> gen pos H.hehf.05\n",
      "Found corr. gen num! idx_uni: 168 <> idx_seq: 169 <> gen pos H.hehf.06\n",
      "Found corr. gen num! idx_uni: 169 <> idx_seq: 170 <> gen pos H.hehf.07\n",
      "Found corr. gen num! idx_uni: 170 <> idx_seq: 171 <> gen pos H.HF.01\n",
      "Found corr. gen num! idx_uni: 171 <> idx_seq: 172 <> gen pos H.HF.02\n",
      "Found corr. gen num! idx_uni: 172 <> idx_seq: 173 <> gen pos H.HF.03\n",
      "Found corr. gen num! idx_uni: 173 <> idx_seq: 174 <> gen pos H.HF.04\n",
      "Found corr. gen num! idx_uni: 174 <> idx_seq: 175 <> gen pos H.HF.05\n",
      "Found corr. gen num! idx_uni: 175 <> idx_seq: 176 <> gen pos H.HF.06\n",
      "Found corr. gen num! idx_uni: 176 <> idx_seq: 177 <> gen pos G.hfs2.01\n",
      "Found corr. gen num! idx_uni: 177 <> idx_seq: 178 <> gen pos G.hfs2.02\n",
      "Found corr. gen num! idx_uni: 178 <> idx_seq: 179 <> gen pos G.hfs2.03\n",
      "Found corr. gen num! idx_uni: 179 <> idx_seq: 180 <> gen pos G.hfs2.04\n",
      "Found corr. gen num! idx_uni: 180 <> idx_seq: 181 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 16%|███████████████████                                                                                                 | 11/67 [00:10<00:43,  1.29it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6OIK\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.06"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 18%|████████████████████▊                                                                                               | 12/67 [00:10<00:41,  1.34it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6QNO\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 23 > end_pdb 376\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 26 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 27 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 28 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 29 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 30 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 31 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 32 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 33 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 34 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 35 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 36 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 37 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 38 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 39 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 40 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 41 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 42 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 43 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 44 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 45 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 46 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 47 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 48 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 49 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 50 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 51 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 52 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 53 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 54 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 55 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 56 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 57 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 58 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 59 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 60 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 61 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 62 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 63 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 64 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 65 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 66 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 67 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 68 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 69 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 70 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 71 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 72 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 73 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 74 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 75 <> gen pos G.H1.08\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 205 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 206 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 207 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 208 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 209 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 210 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 211 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 212 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 213 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 214 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 215 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 216 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 217 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 218 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 219 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 220 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 221 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 222 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 223 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 224 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 225 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 226 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 227 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 228 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 229 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 230 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 231 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 232 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 233 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 234 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 235 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 236 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 237 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 238 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 239 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 240 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 241 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 242 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 243 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 244 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 245 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 246 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 247 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 248 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 249 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 250 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 251 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 252 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 263 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 264 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 265 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 266 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 267 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 268 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 269 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 270 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 271 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 272 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 273 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 274 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 275 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 276 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 277 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 278 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 279 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 280 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 281 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 282 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 283 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 284 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 285 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 286 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 287 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 288 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 289 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 290 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 291 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 292 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 293 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 294 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 295 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 296 <> gen pos G.HG.04"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 19%|██████████████████████▌                                                                                             | 13/67 [00:11<00:40,  1.34it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 297 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 298 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 299 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 300 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 301 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 302 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 303 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 304 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 305 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 306 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 307 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 308 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 309 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 310 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 311 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 312 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 313 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 314 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 315 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 316 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 317 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 318 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 319 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 320 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 321 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 322 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 323 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 324 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 325 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 326 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 327 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 328 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 329 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 330 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 331 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 332 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 333 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 334 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 335 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 336 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 337 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 338 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 339 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 340 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 341 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 342 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 343 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 344 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 345 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 346 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 347 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 348 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 349 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 350 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 351 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 352 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 353 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 354 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 355 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 356 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 357 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 358 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 359 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 360 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 361 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 362 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 363 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 364 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 365 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 366 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 367 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 368 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 369 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 370 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 371 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 372 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 373 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 374 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 375 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 376 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6PT0\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 1 <> idx_seq: 2 <> gen pos G.HN.02\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 3 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.12\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.h1ha.01\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.h1ha.02\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.h1ha.03\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 61 <> gen pos G.h1ha.04\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 62 <> gen pos G.h1ha.05\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 63 <> gen pos H.HA.01\n",
      "Found corr. gen num! idx_uni: 63 <> idx_seq: 64 <> gen pos H.HA.02\n",
      "Found corr. gen num! idx_uni: 64 <> idx_seq: 65 <> gen pos H.HA.03\n",
      "Found corr. gen num! idx_uni: 65 <> idx_seq: 66 <> gen pos H.HA.04\n",
      "Found corr. gen num! idx_uni: 66 <> idx_seq: 67 <> gen pos H.HA.05\n",
      "Found corr. gen num! idx_uni: 67 <> idx_seq: 68 <> gen pos H.HA.06\n",
      "Found corr. gen num! idx_uni: 68 <> idx_seq: 69 <> gen pos H.HA.07\n",
      "Found corr. gen num! idx_uni: 69 <> idx_seq: 70 <> gen pos H.HA.08\n",
      "Found corr. gen num! idx_uni: 70 <> idx_seq: 71 <> gen pos H.HA.09\n",
      "Found corr. gen num! idx_uni: 71 <> idx_seq: 72 <> gen pos H.HA.10\n",
      "Found corr. gen num! idx_uni: 72 <> idx_seq: 73 <> gen pos H.HA.11\n",
      "Found corr. gen num! idx_uni: 73 <> idx_seq: 74 <> gen pos H.HA.12\n",
      "Found corr. gen num! idx_uni: 74 <> idx_seq: 75 <> gen pos H.HA.13\n",
      "Found corr. gen num! idx_uni: 75 <> idx_seq: 76 <> gen pos H.HA.14\n",
      "Found corr. gen num! idx_uni: 76 <> idx_seq: 77 <> gen pos H.HA.15\n",
      "Found corr. gen num! idx_uni: 77 <> idx_seq: 78 <> gen pos H.HA.16\n",
      "Found corr. gen num! idx_uni: 78 <> idx_seq: 79 <> gen pos H.HA.17\n",
      "Found corr. gen num! idx_uni: 79 <> idx_seq: 80 <> gen pos H.HA.18\n",
      "Found corr. gen num! idx_uni: 80 <> idx_seq: 81 <> gen pos H.HA.19\n",
      "Found corr. gen num! idx_uni: 81 <> idx_seq: 82 <> gen pos H.HA.20\n",
      "Found corr. gen num! idx_uni: 82 <> idx_seq: 83 <> gen pos H.HA.21\n",
      "Found corr. gen num! idx_uni: 83 <> idx_seq: 84 <> gen pos H.HA.22\n",
      "Found corr. gen num! idx_uni: 84 <> idx_seq: 85 <> gen pos H.HA.23\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 85 <> idx_seq: 86 <> gen pos H.HA.24\n",
      "Found corr. gen num! idx_uni: 86 <> idx_seq: 87 <> gen pos H.HA.25\n",
      "Found corr. gen num! idx_uni: 87 <> idx_seq: 88 <> gen pos H.HA.26\n",
      "Found corr. gen num! idx_uni: 88 <> idx_seq: 89 <> gen pos H.HA.27\n",
      "Found corr. gen num! idx_uni: 89 <> idx_seq: 90 <> gen pos H.HA.28\n",
      "Found corr. gen num! idx_uni: 90 <> idx_seq: 91 <> gen pos H.HA.29\n",
      "Found corr. gen num! idx_uni: 91 <> idx_seq: 92 <> gen pos H.hahb.01\n",
      "Found corr. gen num! idx_uni: 92 <> idx_seq: 93 <> gen pos H.hahb.04\n",
      "Found corr. gen num! idx_uni: 93 <> idx_seq: 94 <> gen pos H.hahb.05\n",
      "Found corr. gen num! idx_uni: 94 <> idx_seq: 95 <> gen pos H.hahb.06\n",
      "Found corr. gen num! idx_uni: 95 <> idx_seq: 96 <> gen pos H.hahb.07\n",
      "Found corr. gen num! idx_uni: 96 <> idx_seq: 97 <> gen pos H.hahb.08\n",
      "Found corr. gen num! idx_uni: 97 <> idx_seq: 98 <> gen pos H.hahb.09\n",
      "Found corr. gen num! idx_uni: 98 <> idx_seq: 99 <> gen pos H.HB.01\n",
      "Found corr. gen num! idx_uni: 99 <> idx_seq: 100 <> gen pos H.HB.02\n",
      "Found corr. gen num! idx_uni: 100 <> idx_seq: 101 <> gen pos H.HB.03\n",
      "Found corr. gen num! idx_uni: 101 <> idx_seq: 102 <> gen pos H.HB.04\n",
      "Found corr. gen num! idx_uni: 102 <> idx_seq: 103 <> gen pos H.HB.05\n",
      "Found corr. gen num! idx_uni: 103 <> idx_seq: 104 <> gen pos H.HB.06\n",
      "Found corr. gen num! idx_uni: 104 <> idx_seq: 105 <> gen pos H.HB.07\n",
      "Found corr. gen num! idx_uni: 105 <> idx_seq: 106 <> gen pos H.HB.08\n",
      "Found corr. gen num! idx_uni: 106 <> idx_seq: 107 <> gen pos H.HB.09\n",
      "Found corr. gen num! idx_uni: 107 <> idx_seq: 108 <> gen pos H.HB.10\n",
      "Found corr. gen num! idx_uni: 108 <> idx_seq: 109 <> gen pos H.HB.11\n",
      "Found corr. gen num! idx_uni: 109 <> idx_seq: 110 <> gen pos H.HB.12\n",
      "Found corr. gen num! idx_uni: 110 <> idx_seq: 111 <> gen pos H.HB.13\n",
      "Found corr. gen num! idx_uni: 111 <> idx_seq: 112 <> gen pos H.HB.14\n",
      "Found corr. gen num! idx_uni: 112 <> idx_seq: 113 <> gen pos H.hbhc.01\n",
      "Found corr. gen num! idx_uni: 113 <> idx_seq: 114 <> gen pos H.hbhc.02\n",
      "Found corr. gen num! idx_uni: 114 <> idx_seq: 115 <> gen pos H.hbhc.03\n",
      "Found corr. gen num! idx_uni: 115 <> idx_seq: 116 <> gen pos H.hbhc.04\n",
      "Found corr. gen num! idx_uni: 116 <> idx_seq: 117 <> gen pos H.hbhc.12\n",
      "Found corr. gen num! idx_uni: 117 <> idx_seq: 118 <> gen pos H.hbhc.13\n",
      "Found corr. gen num! idx_uni: 118 <> idx_seq: 119 <> gen pos H.hbhc.14\n",
      "Found corr. gen num! idx_uni: 119 <> idx_seq: 120 <> gen pos H.hbhc.15\n",
      "Found corr. gen num! idx_uni: 120 <> idx_seq: 121 <> gen pos H.HC.01\n",
      "Found corr. gen num! idx_uni: 121 <> idx_seq: 122 <> gen pos H.HC.02\n",
      "Found corr. gen num! idx_uni: 122 <> idx_seq: 123 <> gen pos H.HC.03\n",
      "Found corr. gen num! idx_uni: 123 <> idx_seq: 124 <> gen pos H.HC.04\n",
      "Found corr. gen num! idx_uni: 124 <> idx_seq: 125 <> gen pos H.HC.05\n",
      "Found corr. gen num! idx_uni: 125 <> idx_seq: 126 <> gen pos H.HC.06\n",
      "Found corr. gen num! idx_uni: 126 <> idx_seq: 127 <> gen pos H.HC.07\n",
      "Found corr. gen num! idx_uni: 127 <> idx_seq: 128 <> gen pos H.HC.08\n",
      "Found corr. gen num! idx_uni: 128 <> idx_seq: 129 <> gen pos H.HC.09\n",
      "Found corr. gen num! idx_uni: 129 <> idx_seq: 130 <> gen pos H.HC.10\n",
      "Found corr. gen num! idx_uni: 130 <> idx_seq: 131 <> gen pos H.HC.11\n",
      "Found corr. gen num! idx_uni: 131 <> idx_seq: 132 <> gen pos H.HC.12\n",
      "Found corr. gen num! idx_uni: 132 <> idx_seq: 133 <> gen pos H.hchd.01\n",
      "Found corr. gen num! idx_uni: 133 <> idx_seq: 134 <> gen pos H.HD.01\n",
      "Found corr. gen num! idx_uni: 134 <> idx_seq: 135 <> gen pos H.HD.02\n",
      "Found corr. gen num! idx_uni: 135 <> idx_seq: 136 <> gen pos H.HD.03\n",
      "Found corr. gen num! idx_uni: 136 <> idx_seq: 137 <> gen pos H.HD.04\n",
      "Found corr. gen num! idx_uni: 137 <> idx_seq: 138 <> gen pos H.HD.05\n",
      "Found corr. gen num! idx_uni: 138 <> idx_seq: 139 <> gen pos H.HD.06\n",
      "Found corr. gen num! idx_uni: 139 <> idx_seq: 140 <> gen pos H.HD.07\n",
      "Found corr. gen num! idx_uni: 140 <> idx_seq: 141 <> gen pos H.HD.08\n",
      "Found corr. gen num! idx_uni: 141 <> idx_seq: 142 <> gen pos H.HD.09\n",
      "Found corr. gen num! idx_uni: 142 <> idx_seq: 143 <> gen pos H.HD.10\n",
      "Found corr. gen num! idx_uni: 143 <> idx_seq: 144 <> gen pos H.HD.11\n",
      "Found corr. gen num! idx_uni: 144 <> idx_seq: 145 <> gen pos H.HD.12\n",
      "Found corr. gen num! idx_uni: 145 <> idx_seq: 146 <> gen pos H.hdhe.01\n",
      "Found corr. gen num! idx_uni: 146 <> idx_seq: 147 <> gen pos H.hdhe.02\n",
      "Found corr. gen num! idx_uni: 147 <> idx_seq: 148 <> gen pos H.hdhe.03\n",
      "Found corr. gen num! idx_uni: 148 <> idx_seq: 149 <> gen pos H.hdhe.04\n",
      "Found corr. gen num! idx_uni: 149 <> idx_seq: 150 <> gen pos H.hdhe.05\n",
      "Found corr. gen num! idx_uni: 150 <> idx_seq: 151 <> gen pos H.HE.01\n",
      "Found corr. gen num! idx_uni: 151 <> idx_seq: 152 <> gen pos H.HE.02\n",
      "Found corr. gen num! idx_uni: 152 <> idx_seq: 153 <> gen pos H.HE.03\n",
      "Found corr. gen num! idx_uni: 153 <> idx_seq: 154 <> gen pos H.HE.04\n",
      "Found corr. gen num! idx_uni: 154 <> idx_seq: 155 <> gen pos H.HE.05\n",
      "Found corr. gen num! idx_uni: 155 <> idx_seq: 156 <> gen pos H.HE.06\n",
      "Found corr. gen num! idx_uni: 156 <> idx_seq: 157 <> gen pos H.HE.07\n",
      "Found corr. gen num! idx_uni: 157 <> idx_seq: 158 <> gen pos H.HE.08\n",
      "Found corr. gen num! idx_uni: 158 <> idx_seq: 159 <> gen pos H.HE.09\n",
      "Found corr. gen num! idx_uni: 159 <> idx_seq: 160 <> gen pos H.HE.10\n",
      "Found corr. gen num! idx_uni: 160 <> idx_seq: 161 <> gen pos H.HE.11\n",
      "Found corr. gen num! idx_uni: 161 <> idx_seq: 162 <> gen pos H.HE.12\n",
      "Found corr. gen num! idx_uni: 162 <> idx_seq: 163 <> gen pos H.HE.13\n",
      "Found corr. gen num! idx_uni: 163 <> idx_seq: 164 <> gen pos H.hehf.01\n",
      "Found corr. gen num! idx_uni: 164 <> idx_seq: 165 <> gen pos H.hehf.02\n",
      "Found corr. gen num! idx_uni: 165 <> idx_seq: 166 <> gen pos H.hehf.03\n",
      "Found corr. gen num! idx_uni: 166 <> idx_seq: 167 <> gen pos H.hehf.04\n",
      "Found corr. gen num! idx_uni: 167 <> idx_seq: 168 <> gen pos H.hehf.05\n",
      "Found corr. gen num! idx_uni: 168 <> idx_seq: 169 <> gen pos H.hehf.06\n",
      "Found corr. gen num! idx_uni: 169 <> idx_seq: 170 <> gen pos H.hehf.07\n",
      "Found corr. gen num! idx_uni: 170 <> idx_seq: 171 <> gen pos H.HF.01\n",
      "Found corr. gen num! idx_uni: 171 <> idx_seq: 172 <> gen pos H.HF.02\n",
      "Found corr. gen num! idx_uni: 172 <> idx_seq: 173 <> gen pos H.HF.03\n",
      "Found corr. gen num! idx_uni: 173 <> idx_seq: 174 <> gen pos H.HF.04\n",
      "Found corr. gen num! idx_uni: 174 <> idx_seq: 175 <> gen pos H.HF.05\n",
      "Found corr. gen num! idx_uni: 175 <> idx_seq: 176 <> gen pos H.HF.06\n",
      "Found corr. gen num! idx_uni: 176 <> idx_seq: 177 <> gen pos G.hfs2.01\n",
      "Found corr. gen num! idx_uni: 177 <> idx_seq: 178 <> gen pos G.hfs2.02\n",
      "Found corr. gen num! idx_uni: 178 <> idx_seq: 179 <> gen pos G.hfs2.03\n",
      "Found corr. gen num! idx_uni: 179 <> idx_seq: 180 <> gen pos G.hfs2.04\n",
      "Found corr. gen num! idx_uni: 180 <> idx_seq: 181 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 21%|████████████████████████▏                                                                                           | 14/67 [00:12<00:40,  1.30it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6D9H\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 355\n",
      "start_pdb 1 > end_pdb 355\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.03"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 22%|█████████████████████████▉                                                                                          | 15/67 [00:12<00:37,  1.38it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6NI3\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 380\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.26\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.27\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos H.hehf.05\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos H.hehf.06\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos H.hehf.07\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos H.HF.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos H.HF.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos H.HF.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos H.HF.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos H.HF.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos H.HF.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.hfs2.01\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.hfs2.02\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.hfs2.03\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.hfs2.04\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 365 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 366 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 367 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 368 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 369 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 370 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 24%|███████████████████████████▋                                                                                        | 16/67 [00:13<00:37,  1.37it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "assinging res_nums to gprot of 6LFO\n",
      "Mapping found!\n",
      "start_uni 2 > end_uni 354\n",
      "start_pdb 1 > end_pdb 353\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 2 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 3 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 4 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 5 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 6 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 7 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 8 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 9 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 10 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 11 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 12 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 13 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 14 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 15 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 16 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 17 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 18 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 19 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 20 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 21 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 22 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 23 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 24 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 25 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 26 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 27 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 28 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 29 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 30 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 31 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 32 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 33 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 34 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 35 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 36 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 37 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 38 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 39 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 40 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 41 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 42 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 43 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 44 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 45 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 46 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 47 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 48 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 49 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 50 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 51 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 52 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 53 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 54 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 55 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 178 <> idx_seq: 178 <> gen pos G.hfs2.03\n",
      "Found corr. gen num! idx_uni: 179 <> idx_seq: 179 <> gen pos G.hfs2.04\n",
      "Found corr. gen num! idx_uni: 180 <> idx_seq: 180 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 181 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 182 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 183 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 184 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 185 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 186 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 187 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 188 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 189 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 190 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 191 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 192 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 193 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 194 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 195 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 196 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 197 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 198 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 199 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 200 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 201 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 202 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 203 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 204 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 205 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 206 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 207 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 208 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 209 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 210 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 211 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 212 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 213 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 214 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 215 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 216 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 217 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 218 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 219 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 220 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 221 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 222 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 223 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 224 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 225 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 226 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 227 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 228 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 229 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 230 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 231 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 232 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 233 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 234 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 235 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 236 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 237 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 238 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 239 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 240 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 241 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 242 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 243 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 244 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 245 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 246 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 247 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 248 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 249 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 250 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 251 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 252 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 253 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 254 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 255 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 256 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 257 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 258 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 259 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 260 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 261 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 262 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 263 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 264 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 265 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 266 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 267 <> gen pos G.S5.06\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 25%|█████████████████████████████▍                                                                                      | 17/67 [00:14<00:36,  1.38it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 268 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 269 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 270 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 271 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 272 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 273 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 274 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 275 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 276 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 277 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 278 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 279 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 280 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 281 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 282 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 283 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 284 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 285 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 286 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 287 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 288 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 289 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 290 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 291 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 292 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 293 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 294 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 295 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 296 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 297 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 298 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 299 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 300 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 301 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 302 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 303 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 304 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 305 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 306 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 307 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 308 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 309 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 310 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 311 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 312 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 313 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 314 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 315 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 316 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 317 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 318 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 319 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 320 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 321 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 322 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 323 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 324 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 325 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 326 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 327 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 328 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 329 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 330 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 331 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 332 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 333 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 334 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 335 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 336 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 337 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 338 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 339 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 340 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 341 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 342 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 343 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 344 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 345 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 346 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 347 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 348 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 349 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 350 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 351 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 352 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 353 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6OSA\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 27%|███████████████████████████████▏                                                                                    | 18/67 [00:14<00:34,  1.43it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7KH0\n",
      "Mapping found!\n",
      "start_uni 27 > end_uni 394\n",
      "start_pdb 19 > end_pdb 372\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 20 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 21 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 22 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 23 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 24 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 25 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 26 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 27 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 28 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 29 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 30 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 31 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 32 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 33 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 34 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 35 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 36 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 37 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 38 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 39 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 40 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 41 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 42 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 43 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 44 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 45 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 46 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 47 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 48 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 49 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 50 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 51 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 52 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 183 <> gen pos H.hehf.05\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 184 <> gen pos H.hehf.06\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 185 <> gen pos H.hehf.07\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 186 <> gen pos H.HF.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 187 <> gen pos H.HF.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 188 <> gen pos H.HF.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 189 <> gen pos H.HF.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 190 <> gen pos H.HF.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 191 <> gen pos H.HF.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 192 <> gen pos G.hfs2.01\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 193 <> gen pos G.hfs2.02\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 194 <> gen pos G.hfs2.03\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 195 <> gen pos G.hfs2.04\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 196 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 197 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 198 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 199 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 200 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 201 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 202 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 203 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 204 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 205 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 206 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 207 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 208 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 209 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 210 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 211 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 212 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 213 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 214 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 215 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 216 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 217 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 218 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 219 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 220 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 221 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 222 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 223 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 224 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 225 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 226 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 227 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 228 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 229 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 240 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 241 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 242 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 243 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 244 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 245 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 246 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 247 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 248 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 249 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 250 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 251 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 252 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 253 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 254 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 255 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 256 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 257 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 258 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 259 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 260 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 261 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 262 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 263 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 264 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 265 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 266 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 267 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 268 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 269 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 270 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 271 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 272 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 273 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 274 <> gen pos G.H3.18\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 275 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 276 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 277 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 278 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 279 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 280 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 281 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 285 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 286 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 287 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 288 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 289 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 290 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 291 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 292 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 293 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 294 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 295 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 296 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 297 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 298 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 299 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 300 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 301 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 302 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 303 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 304 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 305 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 306 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 307 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 308 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 309 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 310 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 311 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 312 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 313 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 314 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 315 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 316 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 317 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 318 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 319 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 320 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 321 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 322 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 323 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 324 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 325 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 326 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 327 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 328 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 329 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 330 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 331 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 332 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 333 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 334 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 335 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 336 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 337 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 338 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 339 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 340 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 341 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 342 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 343 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 344 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 345 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 346 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 347 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 348 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 349 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 350 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 351 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 352 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 353 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 354 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 355 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 356 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 357 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 358 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 359 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 360 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 361 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 362 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 363 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 364 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 365 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 366 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 367 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 368 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 369 <> gen pos G.H5.09"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 28%|████████████████████████████████▉                                                                                   | 19/67 [00:15<00:34,  1.41it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 370 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 371 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 372 <> gen pos G.H5.12\n",
      "assinging res_nums to gprot of 7JVR\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 30%|██████████████████████████████████▋                                                                                 | 20/67 [00:16<00:33,  1.41it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6G79\n",
      "Mapping found!\n",
      "start_uni 4 > end_uni 57\n",
      "start_pdb 1 > end_pdb 54\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 2 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 3 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 4 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 5 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 6 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 7 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 8 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 9 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 10 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 11 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 12 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 13 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 14 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 15 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 16 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 17 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 18 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 19 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 20 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 21 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 22 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 23 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 24 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 25 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 26 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 27 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 28 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 29 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 30 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 31 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 32 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 33 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 34 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 35 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 36 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 37 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 38 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 39 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 40 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 41 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 42 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 43 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 44 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 45 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 46 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 47 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 48 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 49 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 50 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 51 <> gen pos G.H1.09\n",
      "start_uni 182 > end_uni 354\n",
      "start_pdb 63 > end_pdb 225\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 64 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 65 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 66 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 67 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 68 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 69 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 70 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 71 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 72 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 73 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 74 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 75 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 76 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 77 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 78 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 79 <> gen pos G.S3.04\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 80 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 81 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 82 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 83 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 84 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 85 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 86 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 87 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 88 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 89 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 90 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 91 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 92 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 93 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 94 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 95 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 96 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 97 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 98 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 99 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 100 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 101 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 102 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 103 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 104 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 105 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 106 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 107 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 108 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 109 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 110 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 111 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 112 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 113 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 114 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 115 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 116 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 117 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 118 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 119 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 120 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 121 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 122 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 123 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 124 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 125 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 126 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 127 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 128 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 129 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 130 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 131 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 132 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 133 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 134 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 135 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 136 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 137 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 138 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 139 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 140 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 141 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 142 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 143 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 144 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 145 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 146 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 147 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 148 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 149 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 150 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 151 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 152 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 153 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 154 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 155 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 156 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 157 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 158 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 159 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 160 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 164 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 165 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 166 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 167 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 168 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 169 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 170 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 171 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 172 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 173 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 174 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 175 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 176 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 177 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 178 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 179 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 180 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 181 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 182 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 183 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 184 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 185 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 186 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 187 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 188 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 189 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 190 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 191 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 192 <> gen pos G.h4s6.01"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 31%|████████████████████████████████████▎                                                                               | 21/67 [00:16<00:29,  1.55it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 193 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 194 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 195 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 196 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 197 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 198 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 199 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 200 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 201 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 202 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 203 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 204 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 205 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 206 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 207 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 208 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 209 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 210 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 211 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 212 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 213 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 214 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 215 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 216 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 217 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 218 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 219 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 220 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 221 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 222 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 223 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 224 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 225 <> gen pos G.H5.16\n",
      "assinging res_nums to gprot of 7CMV\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.12\n",
      "Found corr. gen num! idx_uni: 180 <> idx_seq: 181 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 33%|██████████████████████████████████████                                                                              | 22/67 [00:17<00:29,  1.50it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7JJO\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 5 > end_pdb 384\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 19 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 20 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 21 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 22 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 23 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 24 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 25 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 26 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 27 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 28 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 29 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 30 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 31 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 32 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 33 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 34 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 35 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 36 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 37 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 38 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 39 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 40 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 41 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 42 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 43 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 44 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 45 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 46 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 47 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 48 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 49 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 50 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 51 <> gen pos G.s1h1.01\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 195 <> gen pos H.hehf.05\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 196 <> gen pos H.hehf.06\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 197 <> gen pos H.hehf.07\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 198 <> gen pos H.HF.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 199 <> gen pos H.HF.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 200 <> gen pos H.HF.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 201 <> gen pos H.HF.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 202 <> gen pos H.HF.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 203 <> gen pos H.HF.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 204 <> gen pos G.hfs2.01\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 205 <> gen pos G.hfs2.02\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 206 <> gen pos G.hfs2.03\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 207 <> gen pos G.hfs2.04\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 208 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 209 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 210 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 211 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 212 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 213 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 214 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 215 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 216 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 217 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 218 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 219 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 220 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 221 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 222 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 223 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 224 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 225 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 226 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 227 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 228 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 229 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 230 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 231 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 232 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 233 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 234 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 235 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 236 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 237 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 238 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 239 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 240 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 241 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 242 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 251 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 252 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 253 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 254 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 255 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 256 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 257 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 258 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 259 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 260 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 261 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 262 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 263 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 264 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 265 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 266 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 267 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 268 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 269 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 270 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 271 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 272 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 273 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 274 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 275 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 276 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 277 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 278 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 279 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 280 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 281 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 282 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 283 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 284 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 285 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 286 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 287 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 288 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 289 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 290 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 291 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 292 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 297 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 298 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 299 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 300 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 301 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 302 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 303 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 304 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 305 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 306 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 307 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 308 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 309 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 310 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 311 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 321 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 322 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 323 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 324 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 325 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 326 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 327 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 328 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 329 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 330 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 331 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 332 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 333 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 334 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 335 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 336 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 337 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 338 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 339 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 340 <> gen pos G.H4.05"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 34%|███████████████████████████████████████▊                                                                            | 23/67 [00:18<00:33,  1.32it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 341 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 342 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 343 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 344 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 345 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 346 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 347 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 348 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 349 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 350 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 351 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 352 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 353 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 354 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 357 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 358 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 359 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 360 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 361 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 362 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 363 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 364 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 365 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 366 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 367 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 368 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 369 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 370 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 371 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 372 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 373 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 374 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 375 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 376 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 377 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 378 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 379 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 380 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 381 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 382 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 383 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 384 <> gen pos G.H5.12\n",
      "assinging res_nums to gprot of 6OY9\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 350\n",
      "start_pdb 10 > end_pdb 359\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 15 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 16 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 17 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 18 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 19 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 20 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 21 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 22 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 23 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 24 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 25 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 26 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 27 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 28 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 29 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 30 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 31 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 32 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 33 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 34 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 35 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 36 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 37 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 38 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 39 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 40 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 41 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 42 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 43 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 44 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 45 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 46 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 47 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 48 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 49 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 50 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 51 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 52 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 53 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 54 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 55 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 56 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 57 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 58 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 59 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 60 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 61 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 62 <> gen pos G.H1.12\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 63 <> gen pos G.h1ha.01\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 64 <> gen pos G.h1ha.02\n",
      "Found corr. gen num! idx_uni: 177 <> idx_seq: 187 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 178 <> idx_seq: 188 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 179 <> idx_seq: 189 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 180 <> idx_seq: 190 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 191 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 192 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 193 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 194 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 195 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 196 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 197 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 198 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 199 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 200 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 201 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 202 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 203 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 204 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 205 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 206 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 207 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 208 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 209 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 210 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 211 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 212 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 213 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 214 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 215 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 216 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 217 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 218 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 219 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 220 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 221 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 222 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 223 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 224 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 225 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 226 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 227 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 228 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 229 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 230 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 231 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 232 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 233 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 234 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 235 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 236 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 237 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 238 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 239 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 240 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 241 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 242 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 243 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 244 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 245 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 246 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 247 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 248 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 249 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 250 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 251 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 252 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 253 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 254 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 255 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 256 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 257 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 258 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 259 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 260 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 261 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 262 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 263 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 264 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 265 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 266 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 267 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 268 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 269 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 270 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 271 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 272 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 273 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 274 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 275 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 276 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 277 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 278 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 279 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 280 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 281 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 282 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 283 <> gen pos G.HG.08"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 36%|█████████████████████████████████████████▌                                                                          | 24/67 [00:19<00:31,  1.38it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 284 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 285 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 286 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 287 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 288 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 289 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 290 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 291 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 292 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 293 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 294 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 295 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 296 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 297 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 298 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 299 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 300 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 301 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 302 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 303 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 304 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 305 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 306 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 307 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 308 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 309 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 310 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 311 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 312 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 313 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 314 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 315 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 316 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 317 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 318 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 319 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 320 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 321 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 322 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 323 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 324 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 325 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 326 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 327 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 328 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 329 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 330 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 331 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 332 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 333 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 334 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 335 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 336 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 337 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 338 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 339 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 340 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 341 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 342 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 343 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 344 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 345 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 346 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 347 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 348 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 349 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 350 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 351 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 352 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 353 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 354 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 355 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 356 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 357 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 358 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 359 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7LJD\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 246\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.26\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.27\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 61 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 62 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 63 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 63 <> idx_seq: 64 <> gen pos G.H1.12\n",
      "Found corr. gen num! idx_uni: 64 <> idx_seq: 65 <> gen pos G.h1ha.01\n",
      "Found corr. gen num! idx_uni: 65 <> idx_seq: 66 <> gen pos G.h1ha.02\n",
      "Found corr. gen num! idx_uni: 66 <> idx_seq: 67 <> gen pos G.h1ha.03\n",
      "Found corr. gen num! idx_uni: 67 <> idx_seq: 68 <> gen pos G.h1ha.04\n",
      "Found corr. gen num! idx_uni: 68 <> idx_seq: 69 <> gen pos G.h1ha.05\n",
      "Found corr. gen num! idx_uni: 69 <> idx_seq: 70 <> gen pos G.h1ha.06\n",
      "Found corr. gen num! idx_uni: 70 <> idx_seq: 71 <> gen pos G.h1ha.07\n",
      "Found corr. gen num! idx_uni: 71 <> idx_seq: 72 <> gen pos G.h1ha.08\n",
      "Found corr. gen num! idx_uni: 72 <> idx_seq: 73 <> gen pos G.h1ha.09\n",
      "Found corr. gen num! idx_uni: 73 <> idx_seq: 74 <> gen pos G.h1ha.10\n",
      "Found corr. gen num! idx_uni: 74 <> idx_seq: 75 <> gen pos G.h1ha.11\n",
      "Found corr. gen num! idx_uni: 75 <> idx_seq: 76 <> gen pos G.h1ha.12\n",
      "Found corr. gen num! idx_uni: 76 <> idx_seq: 77 <> gen pos G.h1ha.13\n",
      "Found corr. gen num! idx_uni: 77 <> idx_seq: 78 <> gen pos G.h1ha.14\n",
      "Found corr. gen num! idx_uni: 78 <> idx_seq: 79 <> gen pos G.h1ha.15\n",
      "Found corr. gen num! idx_uni: 79 <> idx_seq: 80 <> gen pos G.h1ha.16\n",
      "Found corr. gen num! idx_uni: 80 <> idx_seq: 81 <> gen pos G.h1ha.17\n",
      "Found corr. gen num! idx_uni: 81 <> idx_seq: 82 <> gen pos G.h1ha.18\n",
      "Found corr. gen num! idx_uni: 82 <> idx_seq: 83 <> gen pos G.h1ha.19\n",
      "Found corr. gen num! idx_uni: 83 <> idx_seq: 84 <> gen pos G.h1ha.20\n",
      "Found corr. gen num! idx_uni: 84 <> idx_seq: 85 <> gen pos H.HA.01\n",
      "Found corr. gen num! idx_uni: 85 <> idx_seq: 86 <> gen pos H.HA.02\n",
      "Found corr. gen num! idx_uni: 86 <> idx_seq: 87 <> gen pos H.HA.03\n",
      "Found corr. gen num! idx_uni: 87 <> idx_seq: 88 <> gen pos H.HA.04\n",
      "Found corr. gen num! idx_uni: 88 <> idx_seq: 89 <> gen pos H.HA.05\n",
      "Found corr. gen num! idx_uni: 89 <> idx_seq: 90 <> gen pos H.HA.06\n",
      "Found corr. gen num! idx_uni: 90 <> idx_seq: 91 <> gen pos H.HA.07\n",
      "Found corr. gen num! idx_uni: 91 <> idx_seq: 92 <> gen pos H.HA.08\n",
      "Found corr. gen num! idx_uni: 92 <> idx_seq: 93 <> gen pos H.HA.09\n",
      "Found corr. gen num! idx_uni: 93 <> idx_seq: 94 <> gen pos H.HA.10\n",
      "Found corr. gen num! idx_uni: 94 <> idx_seq: 95 <> gen pos H.HA.11\n",
      "Found corr. gen num! idx_uni: 95 <> idx_seq: 96 <> gen pos H.HA.12\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 96 <> idx_seq: 97 <> gen pos H.HA.13\n",
      "Found corr. gen num! idx_uni: 97 <> idx_seq: 98 <> gen pos H.HA.14\n",
      "Found corr. gen num! idx_uni: 98 <> idx_seq: 99 <> gen pos H.HA.15\n",
      "Found corr. gen num! idx_uni: 99 <> idx_seq: 100 <> gen pos H.HA.16\n",
      "Found corr. gen num! idx_uni: 100 <> idx_seq: 101 <> gen pos H.HA.17\n",
      "Found corr. gen num! idx_uni: 101 <> idx_seq: 102 <> gen pos H.HA.18\n",
      "Found corr. gen num! idx_uni: 102 <> idx_seq: 103 <> gen pos H.HA.19\n",
      "Found corr. gen num! idx_uni: 103 <> idx_seq: 104 <> gen pos H.HA.20\n",
      "Found corr. gen num! idx_uni: 104 <> idx_seq: 105 <> gen pos H.HA.21\n",
      "Found corr. gen num! idx_uni: 105 <> idx_seq: 106 <> gen pos H.HA.22\n",
      "Found corr. gen num! idx_uni: 106 <> idx_seq: 107 <> gen pos H.HA.23\n",
      "Found corr. gen num! idx_uni: 107 <> idx_seq: 108 <> gen pos H.HA.24\n",
      "Found corr. gen num! idx_uni: 108 <> idx_seq: 109 <> gen pos H.HA.25\n",
      "Found corr. gen num! idx_uni: 109 <> idx_seq: 110 <> gen pos H.HA.26\n",
      "Found corr. gen num! idx_uni: 110 <> idx_seq: 111 <> gen pos H.HA.27\n",
      "Found corr. gen num! idx_uni: 111 <> idx_seq: 112 <> gen pos H.HA.28\n",
      "Found corr. gen num! idx_uni: 112 <> idx_seq: 113 <> gen pos H.HA.29\n",
      "Found corr. gen num! idx_uni: 113 <> idx_seq: 114 <> gen pos H.hahb.01\n",
      "Found corr. gen num! idx_uni: 114 <> idx_seq: 115 <> gen pos H.hahb.02\n",
      "Found corr. gen num! idx_uni: 115 <> idx_seq: 116 <> gen pos H.hahb.03\n",
      "Found corr. gen num! idx_uni: 116 <> idx_seq: 117 <> gen pos H.hahb.04\n",
      "Found corr. gen num! idx_uni: 117 <> idx_seq: 118 <> gen pos H.hahb.05\n",
      "Found corr. gen num! idx_uni: 118 <> idx_seq: 119 <> gen pos H.hahb.06\n",
      "Found corr. gen num! idx_uni: 119 <> idx_seq: 120 <> gen pos H.hahb.07\n",
      "Found corr. gen num! idx_uni: 120 <> idx_seq: 121 <> gen pos H.hahb.08\n",
      "Found corr. gen num! idx_uni: 121 <> idx_seq: 122 <> gen pos H.hahb.09\n",
      "Found corr. gen num! idx_uni: 122 <> idx_seq: 123 <> gen pos H.HB.01\n",
      "Found corr. gen num! idx_uni: 123 <> idx_seq: 124 <> gen pos H.HB.02\n",
      "Found corr. gen num! idx_uni: 124 <> idx_seq: 125 <> gen pos H.HB.03\n",
      "Found corr. gen num! idx_uni: 125 <> idx_seq: 126 <> gen pos H.HB.04\n",
      "Found corr. gen num! idx_uni: 126 <> idx_seq: 127 <> gen pos H.HB.05\n",
      "Found corr. gen num! idx_uni: 127 <> idx_seq: 128 <> gen pos H.HB.06\n",
      "Found corr. gen num! idx_uni: 128 <> idx_seq: 129 <> gen pos H.HB.07\n",
      "Found corr. gen num! idx_uni: 129 <> idx_seq: 130 <> gen pos H.HB.08\n",
      "Found corr. gen num! idx_uni: 130 <> idx_seq: 131 <> gen pos H.HB.09\n",
      "Found corr. gen num! idx_uni: 131 <> idx_seq: 132 <> gen pos H.HB.10\n",
      "Found corr. gen num! idx_uni: 132 <> idx_seq: 133 <> gen pos H.HB.11\n",
      "Found corr. gen num! idx_uni: 133 <> idx_seq: 134 <> gen pos H.HB.12\n",
      "Found corr. gen num! idx_uni: 134 <> idx_seq: 135 <> gen pos H.HB.13\n",
      "Found corr. gen num! idx_uni: 135 <> idx_seq: 136 <> gen pos H.HB.14\n",
      "Found corr. gen num! idx_uni: 136 <> idx_seq: 137 <> gen pos H.hbhc.01\n",
      "Found corr. gen num! idx_uni: 137 <> idx_seq: 138 <> gen pos H.hbhc.02\n",
      "Found corr. gen num! idx_uni: 138 <> idx_seq: 139 <> gen pos H.hbhc.03\n",
      "Found corr. gen num! idx_uni: 139 <> idx_seq: 140 <> gen pos H.hbhc.04\n",
      "Found corr. gen num! idx_uni: 140 <> idx_seq: 141 <> gen pos H.hbhc.13\n",
      "Found corr. gen num! idx_uni: 141 <> idx_seq: 142 <> gen pos H.hbhc.14\n",
      "Found corr. gen num! idx_uni: 142 <> idx_seq: 143 <> gen pos H.hbhc.15\n",
      "Found corr. gen num! idx_uni: 143 <> idx_seq: 144 <> gen pos H.HC.01\n",
      "Found corr. gen num! idx_uni: 144 <> idx_seq: 145 <> gen pos H.HC.02\n",
      "Found corr. gen num! idx_uni: 145 <> idx_seq: 146 <> gen pos H.HC.03\n",
      "Found corr. gen num! idx_uni: 146 <> idx_seq: 147 <> gen pos H.HC.04\n",
      "Found corr. gen num! idx_uni: 147 <> idx_seq: 148 <> gen pos H.HC.05\n",
      "Found corr. gen num! idx_uni: 148 <> idx_seq: 149 <> gen pos H.HC.06\n",
      "Found corr. gen num! idx_uni: 149 <> idx_seq: 150 <> gen pos H.HC.07\n",
      "Found corr. gen num! idx_uni: 150 <> idx_seq: 151 <> gen pos H.HC.08\n",
      "Found corr. gen num! idx_uni: 151 <> idx_seq: 152 <> gen pos H.HC.09\n",
      "Found corr. gen num! idx_uni: 152 <> idx_seq: 153 <> gen pos H.HC.10\n",
      "Found corr. gen num! idx_uni: 153 <> idx_seq: 154 <> gen pos H.HC.11\n",
      "Found corr. gen num! idx_uni: 154 <> idx_seq: 155 <> gen pos H.HC.12\n",
      "Found corr. gen num! idx_uni: 155 <> idx_seq: 156 <> gen pos H.hchd.01\n",
      "Found corr. gen num! idx_uni: 156 <> idx_seq: 157 <> gen pos H.HD.01\n",
      "Found corr. gen num! idx_uni: 157 <> idx_seq: 158 <> gen pos H.HD.02\n",
      "Found corr. gen num! idx_uni: 158 <> idx_seq: 159 <> gen pos H.HD.03\n",
      "Found corr. gen num! idx_uni: 159 <> idx_seq: 160 <> gen pos H.HD.04\n",
      "Found corr. gen num! idx_uni: 160 <> idx_seq: 161 <> gen pos H.HD.05\n",
      "Found corr. gen num! idx_uni: 161 <> idx_seq: 162 <> gen pos H.HD.06\n",
      "Found corr. gen num! idx_uni: 162 <> idx_seq: 163 <> gen pos H.HD.07\n",
      "Found corr. gen num! idx_uni: 163 <> idx_seq: 164 <> gen pos H.HD.08\n",
      "Found corr. gen num! idx_uni: 164 <> idx_seq: 165 <> gen pos H.HD.09\n",
      "Found corr. gen num! idx_uni: 165 <> idx_seq: 166 <> gen pos H.HD.10\n",
      "Found corr. gen num! idx_uni: 166 <> idx_seq: 167 <> gen pos H.HD.11\n",
      "Found corr. gen num! idx_uni: 167 <> idx_seq: 168 <> gen pos H.HD.12\n",
      "Found corr. gen num! idx_uni: 168 <> idx_seq: 169 <> gen pos H.hdhe.01\n",
      "Found corr. gen num! idx_uni: 169 <> idx_seq: 170 <> gen pos H.hdhe.02\n",
      "Found corr. gen num! idx_uni: 170 <> idx_seq: 171 <> gen pos H.hdhe.03\n",
      "Found corr. gen num! idx_uni: 171 <> idx_seq: 172 <> gen pos H.hdhe.04\n",
      "Found corr. gen num! idx_uni: 172 <> idx_seq: 173 <> gen pos H.hdhe.05\n",
      "Found corr. gen num! idx_uni: 173 <> idx_seq: 174 <> gen pos H.HE.01\n",
      "Found corr. gen num! idx_uni: 174 <> idx_seq: 175 <> gen pos H.HE.02\n",
      "Found corr. gen num! idx_uni: 175 <> idx_seq: 176 <> gen pos H.HE.03\n",
      "Found corr. gen num! idx_uni: 176 <> idx_seq: 177 <> gen pos H.HE.04\n",
      "Found corr. gen num! idx_uni: 177 <> idx_seq: 178 <> gen pos H.HE.05\n",
      "Found corr. gen num! idx_uni: 178 <> idx_seq: 179 <> gen pos H.HE.06\n",
      "Found corr. gen num! idx_uni: 179 <> idx_seq: 180 <> gen pos H.HE.07\n",
      "Found corr. gen num! idx_uni: 180 <> idx_seq: 181 <> gen pos H.HE.08\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos H.HE.09\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos H.HE.10\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos H.HE.11\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos H.HE.12\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos H.HE.13\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos H.hehf.01\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos H.hehf.02\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos H.hehf.03\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos H.hehf.04\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos H.hehf.05\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos H.hehf.06\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos H.hehf.07\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos H.HF.01"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 37%|███████████████████████████████████████████▎                                                                        | 25/67 [00:19<00:28,  1.49it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos H.HF.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos H.HF.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos H.HF.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos H.HF.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos H.HF.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.hfs2.01\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.hfs2.02\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.hfs2.03\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.hfs2.04\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "assinging res_nums to gprot of 6DDF\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 39%|█████████████████████████████████████████████                                                                       | 26/67 [00:20<00:26,  1.53it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6DDE\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 40%|██████████████████████████████████████████████▋                                                                     | 27/67 [00:21<00:26,  1.52it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7MBY\n",
      "Mapping found!\n",
      "start_uni 2 > end_uni 57\n",
      "start_pdb 9 > end_pdb 64\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "start_uni 204 > end_uni 251\n",
      "start_pdb 73 > end_pdb 120\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 76 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 77 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 78 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 79 <> gen pos G.S2.04\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 80 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 81 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 82 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 83 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 84 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 85 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 86 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 87 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 88 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 89 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 90 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 91 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 92 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 93 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 94 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 100 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 101 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 102 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 103 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 104 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 105 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 106 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 107 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 108 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 109 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 110 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 111 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 112 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 113 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 114 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 115 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 116 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 117 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 118 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 119 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 120 <> gen pos G.s4h3.02\n",
      "start_uni 229 > end_uni 242\n",
      "start_pdb 121 > end_pdb 124\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 122 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 123 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 124 <> gen pos G.s4h3.06\n",
      "start_uni 266 > end_uni 346\n",
      "start_pdb 125 > end_pdb 205\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 126 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 127 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 128 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 129 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 130 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 131 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 132 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 133 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 134 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 135 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 136 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 137 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 138 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 139 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 140 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 141 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 142 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 143 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 144 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 145 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 146 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 147 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 148 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 149 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 150 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 151 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 152 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 153 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 154 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 155 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 156 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 157 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 158 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 159 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 160 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 161 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 162 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 165 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 166 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 167 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 168 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 169 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 170 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 171 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 172 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 173 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 174 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 175 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 176 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 177 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 178 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 179 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 189 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 190 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 191 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 192 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 193 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 194 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 195 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 196 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 197 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 198 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 199 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 200 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 201 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 202 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 203 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 204 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 205 <> gen pos G.H4.16\n",
      "start_uni 309 > end_uni 309\n",
      "start_pdb 206 > end_pdb 206\n",
      "start_uni 348 > end_uni 357\n",
      "start_pdb 207 > end_pdb 216\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 208 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 209 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 210 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 211 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 212 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 213 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 214 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 215 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 216 <> gen pos G.h4s6.13\n",
      "start_uni 319 > end_uni 319\n",
      "start_pdb 217 > end_pdb 217\n",
      "start_uni 359 > end_uni 386\n",
      "start_pdb 218 > end_pdb 245\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 219 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 220 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 221 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 222 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 223 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 224 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 225 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 226 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 227 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 228 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 229 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 230 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 231 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 232 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 233 <> gen pos G.H5.06"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 42%|████████████████████████████████████████████████▍                                                                   | 28/67 [00:22<00:30,  1.27it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 234 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 235 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 236 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 237 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 238 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 239 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 240 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 241 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 242 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 243 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 244 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 245 <> gen pos G.H5.18\n",
      "7MBY unlabelled: 8\n",
      "label_seq_id 253\n",
      "assigning 253 of gprotein gen. number: G.H5.19\n",
      "label_seq_id 254\n",
      "label_seq_id 255\n",
      "label_seq_id 256\n",
      "label_seq_id 257\n",
      "label_seq_id 258\n",
      "label_seq_id 259\n",
      "label_seq_id 260\n",
      "start_uni 347 > end_uni 348\n",
      "start_pdb 246 > end_pdb 247\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 247 <> gen pos G.H5.20\n",
      "start_uni 389 > end_uni 394\n",
      "start_pdb 248 > end_pdb 253\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 249 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 250 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 251 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 252 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 253 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7L0P\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 1 <> idx_seq: 2 <> gen pos G.HN.02\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 3 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 43%|██████████████████████████████████████████████████▏                                                                 | 29/67 [00:22<00:28,  1.35it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7L0S\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 1 <> idx_seq: 2 <> gen pos G.HN.02\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 3 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 63 <> idx_seq: 64 <> gen pos H.HA.02\n",
      "Found corr. gen num! idx_uni: 64 <> idx_seq: 65 <> gen pos H.HA.03\n",
      "Found corr. gen num! idx_uni: 65 <> idx_seq: 66 <> gen pos H.HA.04\n",
      "Found corr. gen num! idx_uni: 66 <> idx_seq: 67 <> gen pos H.HA.05\n",
      "Found corr. gen num! idx_uni: 67 <> idx_seq: 68 <> gen pos H.HA.06\n",
      "Found corr. gen num! idx_uni: 68 <> idx_seq: 69 <> gen pos H.HA.07\n",
      "Found corr. gen num! idx_uni: 69 <> idx_seq: 70 <> gen pos H.HA.08\n",
      "Found corr. gen num! idx_uni: 70 <> idx_seq: 71 <> gen pos H.HA.09\n",
      "Found corr. gen num! idx_uni: 71 <> idx_seq: 72 <> gen pos H.HA.10\n",
      "Found corr. gen num! idx_uni: 72 <> idx_seq: 73 <> gen pos H.HA.11\n",
      "Found corr. gen num! idx_uni: 73 <> idx_seq: 74 <> gen pos H.HA.12\n",
      "Found corr. gen num! idx_uni: 74 <> idx_seq: 75 <> gen pos H.HA.13\n",
      "Found corr. gen num! idx_uni: 75 <> idx_seq: 76 <> gen pos H.HA.14\n",
      "Found corr. gen num! idx_uni: 76 <> idx_seq: 77 <> gen pos H.HA.15\n",
      "Found corr. gen num! idx_uni: 77 <> idx_seq: 78 <> gen pos H.HA.16\n",
      "Found corr. gen num! idx_uni: 78 <> idx_seq: 79 <> gen pos H.HA.17\n",
      "Found corr. gen num! idx_uni: 79 <> idx_seq: 80 <> gen pos H.HA.18\n",
      "Found corr. gen num! idx_uni: 80 <> idx_seq: 81 <> gen pos H.HA.19\n",
      "Found corr. gen num! idx_uni: 81 <> idx_seq: 82 <> gen pos H.HA.20\n",
      "Found corr. gen num! idx_uni: 82 <> idx_seq: 83 <> gen pos H.HA.21\n",
      "Found corr. gen num! idx_uni: 83 <> idx_seq: 84 <> gen pos H.HA.22\n",
      "Found corr. gen num! idx_uni: 84 <> idx_seq: 85 <> gen pos H.HA.23\n",
      "Found corr. gen num! idx_uni: 85 <> idx_seq: 86 <> gen pos H.HA.24\n",
      "Found corr. gen num! idx_uni: 86 <> idx_seq: 87 <> gen pos H.HA.25\n",
      "Found corr. gen num! idx_uni: 87 <> idx_seq: 88 <> gen pos H.HA.26\n",
      "Found corr. gen num! idx_uni: 88 <> idx_seq: 89 <> gen pos H.HA.27\n",
      "Found corr. gen num! idx_uni: 89 <> idx_seq: 90 <> gen pos H.HA.28\n",
      "Found corr. gen num! idx_uni: 90 <> idx_seq: 91 <> gen pos H.HA.29\n",
      "Found corr. gen num! idx_uni: 91 <> idx_seq: 92 <> gen pos H.hahb.01\n",
      "Found corr. gen num! idx_uni: 92 <> idx_seq: 93 <> gen pos H.hahb.04\n",
      "Found corr. gen num! idx_uni: 93 <> idx_seq: 94 <> gen pos H.hahb.05\n",
      "Found corr. gen num! idx_uni: 94 <> idx_seq: 95 <> gen pos H.hahb.06\n",
      "Found corr. gen num! idx_uni: 95 <> idx_seq: 96 <> gen pos H.hahb.07\n",
      "Found corr. gen num! idx_uni: 96 <> idx_seq: 97 <> gen pos H.hahb.08\n",
      "Found corr. gen num! idx_uni: 97 <> idx_seq: 98 <> gen pos H.hahb.09\n",
      "Found corr. gen num! idx_uni: 98 <> idx_seq: 99 <> gen pos H.HB.01\n",
      "Found corr. gen num! idx_uni: 99 <> idx_seq: 100 <> gen pos H.HB.02\n",
      "Found corr. gen num! idx_uni: 100 <> idx_seq: 101 <> gen pos H.HB.03\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 101 <> idx_seq: 102 <> gen pos H.HB.04\n",
      "Found corr. gen num! idx_uni: 102 <> idx_seq: 103 <> gen pos H.HB.05\n",
      "Found corr. gen num! idx_uni: 103 <> idx_seq: 104 <> gen pos H.HB.06\n",
      "Found corr. gen num! idx_uni: 104 <> idx_seq: 105 <> gen pos H.HB.07\n",
      "Found corr. gen num! idx_uni: 105 <> idx_seq: 106 <> gen pos H.HB.08\n",
      "Found corr. gen num! idx_uni: 106 <> idx_seq: 107 <> gen pos H.HB.09\n",
      "Found corr. gen num! idx_uni: 107 <> idx_seq: 108 <> gen pos H.HB.10\n",
      "Found corr. gen num! idx_uni: 108 <> idx_seq: 109 <> gen pos H.HB.11\n",
      "Found corr. gen num! idx_uni: 109 <> idx_seq: 110 <> gen pos H.HB.12\n",
      "Found corr. gen num! idx_uni: 110 <> idx_seq: 111 <> gen pos H.HB.13\n",
      "Found corr. gen num! idx_uni: 111 <> idx_seq: 112 <> gen pos H.HB.14\n",
      "Found corr. gen num! idx_uni: 112 <> idx_seq: 113 <> gen pos H.hbhc.01\n",
      "Found corr. gen num! idx_uni: 113 <> idx_seq: 114 <> gen pos H.hbhc.02\n",
      "Found corr. gen num! idx_uni: 114 <> idx_seq: 115 <> gen pos H.hbhc.03\n",
      "Found corr. gen num! idx_uni: 115 <> idx_seq: 116 <> gen pos H.hbhc.04\n",
      "Found corr. gen num! idx_uni: 116 <> idx_seq: 117 <> gen pos H.hbhc.12\n",
      "Found corr. gen num! idx_uni: 117 <> idx_seq: 118 <> gen pos H.hbhc.13\n",
      "Found corr. gen num! idx_uni: 118 <> idx_seq: 119 <> gen pos H.hbhc.14\n",
      "Found corr. gen num! idx_uni: 119 <> idx_seq: 120 <> gen pos H.hbhc.15\n",
      "Found corr. gen num! idx_uni: 120 <> idx_seq: 121 <> gen pos H.HC.01\n",
      "Found corr. gen num! idx_uni: 121 <> idx_seq: 122 <> gen pos H.HC.02\n",
      "Found corr. gen num! idx_uni: 122 <> idx_seq: 123 <> gen pos H.HC.03\n",
      "Found corr. gen num! idx_uni: 123 <> idx_seq: 124 <> gen pos H.HC.04\n",
      "Found corr. gen num! idx_uni: 124 <> idx_seq: 125 <> gen pos H.HC.05\n",
      "Found corr. gen num! idx_uni: 125 <> idx_seq: 126 <> gen pos H.HC.06\n",
      "Found corr. gen num! idx_uni: 126 <> idx_seq: 127 <> gen pos H.HC.07\n",
      "Found corr. gen num! idx_uni: 127 <> idx_seq: 128 <> gen pos H.HC.08\n",
      "Found corr. gen num! idx_uni: 128 <> idx_seq: 129 <> gen pos H.HC.09\n",
      "Found corr. gen num! idx_uni: 129 <> idx_seq: 130 <> gen pos H.HC.10\n",
      "Found corr. gen num! idx_uni: 130 <> idx_seq: 131 <> gen pos H.HC.11\n",
      "Found corr. gen num! idx_uni: 131 <> idx_seq: 132 <> gen pos H.HC.12\n",
      "Found corr. gen num! idx_uni: 132 <> idx_seq: 133 <> gen pos H.hchd.01\n",
      "Found corr. gen num! idx_uni: 133 <> idx_seq: 134 <> gen pos H.HD.01\n",
      "Found corr. gen num! idx_uni: 134 <> idx_seq: 135 <> gen pos H.HD.02\n",
      "Found corr. gen num! idx_uni: 135 <> idx_seq: 136 <> gen pos H.HD.03\n",
      "Found corr. gen num! idx_uni: 136 <> idx_seq: 137 <> gen pos H.HD.04\n",
      "Found corr. gen num! idx_uni: 137 <> idx_seq: 138 <> gen pos H.HD.05\n",
      "Found corr. gen num! idx_uni: 138 <> idx_seq: 139 <> gen pos H.HD.06\n",
      "Found corr. gen num! idx_uni: 139 <> idx_seq: 140 <> gen pos H.HD.07\n",
      "Found corr. gen num! idx_uni: 140 <> idx_seq: 141 <> gen pos H.HD.08\n",
      "Found corr. gen num! idx_uni: 141 <> idx_seq: 142 <> gen pos H.HD.09\n",
      "Found corr. gen num! idx_uni: 142 <> idx_seq: 143 <> gen pos H.HD.10\n",
      "Found corr. gen num! idx_uni: 143 <> idx_seq: 144 <> gen pos H.HD.11\n",
      "Found corr. gen num! idx_uni: 144 <> idx_seq: 145 <> gen pos H.HD.12\n",
      "Found corr. gen num! idx_uni: 145 <> idx_seq: 146 <> gen pos H.hdhe.01\n",
      "Found corr. gen num! idx_uni: 146 <> idx_seq: 147 <> gen pos H.hdhe.02\n",
      "Found corr. gen num! idx_uni: 147 <> idx_seq: 148 <> gen pos H.hdhe.03\n",
      "Found corr. gen num! idx_uni: 148 <> idx_seq: 149 <> gen pos H.hdhe.04\n",
      "Found corr. gen num! idx_uni: 149 <> idx_seq: 150 <> gen pos H.hdhe.05\n",
      "Found corr. gen num! idx_uni: 150 <> idx_seq: 151 <> gen pos H.HE.01\n",
      "Found corr. gen num! idx_uni: 151 <> idx_seq: 152 <> gen pos H.HE.02\n",
      "Found corr. gen num! idx_uni: 152 <> idx_seq: 153 <> gen pos H.HE.03\n",
      "Found corr. gen num! idx_uni: 153 <> idx_seq: 154 <> gen pos H.HE.04\n",
      "Found corr. gen num! idx_uni: 154 <> idx_seq: 155 <> gen pos H.HE.05\n",
      "Found corr. gen num! idx_uni: 155 <> idx_seq: 156 <> gen pos H.HE.06\n",
      "Found corr. gen num! idx_uni: 156 <> idx_seq: 157 <> gen pos H.HE.07\n",
      "Found corr. gen num! idx_uni: 157 <> idx_seq: 158 <> gen pos H.HE.08\n",
      "Found corr. gen num! idx_uni: 158 <> idx_seq: 159 <> gen pos H.HE.09\n",
      "Found corr. gen num! idx_uni: 159 <> idx_seq: 160 <> gen pos H.HE.10\n",
      "Found corr. gen num! idx_uni: 160 <> idx_seq: 161 <> gen pos H.HE.11\n",
      "Found corr. gen num! idx_uni: 161 <> idx_seq: 162 <> gen pos H.HE.12\n",
      "Found corr. gen num! idx_uni: 162 <> idx_seq: 163 <> gen pos H.HE.13\n",
      "Found corr. gen num! idx_uni: 163 <> idx_seq: 164 <> gen pos H.hehf.01\n",
      "Found corr. gen num! idx_uni: 164 <> idx_seq: 165 <> gen pos H.hehf.02\n",
      "Found corr. gen num! idx_uni: 165 <> idx_seq: 166 <> gen pos H.hehf.03\n",
      "Found corr. gen num! idx_uni: 166 <> idx_seq: 167 <> gen pos H.hehf.04\n",
      "Found corr. gen num! idx_uni: 167 <> idx_seq: 168 <> gen pos H.hehf.05\n",
      "Found corr. gen num! idx_uni: 168 <> idx_seq: 169 <> gen pos H.hehf.06\n",
      "Found corr. gen num! idx_uni: 169 <> idx_seq: 170 <> gen pos H.hehf.07\n",
      "Found corr. gen num! idx_uni: 170 <> idx_seq: 171 <> gen pos H.HF.01\n",
      "Found corr. gen num! idx_uni: 171 <> idx_seq: 172 <> gen pos H.HF.02\n",
      "Found corr. gen num! idx_uni: 172 <> idx_seq: 173 <> gen pos H.HF.03\n",
      "Found corr. gen num! idx_uni: 173 <> idx_seq: 174 <> gen pos H.HF.04\n",
      "Found corr. gen num! idx_uni: 174 <> idx_seq: 175 <> gen pos H.HF.05\n",
      "Found corr. gen num! idx_uni: 175 <> idx_seq: 176 <> gen pos H.HF.06\n",
      "Found corr. gen num! idx_uni: 176 <> idx_seq: 177 <> gen pos G.hfs2.01\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 45%|███████████████████████████████████████████████████▉                                                                | 30/67 [00:23<00:27,  1.36it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7D7M\n",
      "Mapping found!\n",
      "start_uni 5 > end_uni 64\n",
      "start_pdb 1 > end_pdb 60\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 8 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 9 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 10 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 11 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 12 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 13 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 14 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 15 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 16 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 17 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 18 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 19 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 20 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 21 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 22 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 23 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 24 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 25 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 26 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 27 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 28 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 29 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 30 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 31 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 32 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 33 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 34 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 35 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 36 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 37 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 38 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 39 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 40 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 41 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 42 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 43 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 44 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 45 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 46 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 47 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 48 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 49 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 50 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 51 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 52 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 53 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 54 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 55 <> gen pos G.H1.07\n",
      "start_uni 204 > end_uni 394\n",
      "start_pdb 69 > end_pdb 249\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 70 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 71 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 72 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 73 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 74 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 75 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 76 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 77 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 78 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 79 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 80 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 81 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 82 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 83 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 84 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 85 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 86 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 87 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 88 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 89 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 90 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 91 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 92 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 93 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 94 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 95 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 96 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 97 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 98 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 99 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 100 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 101 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 102 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 103 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 104 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 105 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 106 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 107 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 108 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 109 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 110 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 111 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 112 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 113 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 114 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 115 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 116 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 117 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 118 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 119 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 120 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 121 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 122 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 123 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 124 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 125 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 126 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 127 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 128 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 129 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 130 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 131 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 132 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 133 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 134 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 135 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 136 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 137 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 138 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 139 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 140 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 141 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 142 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 143 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 144 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 145 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 146 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 147 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 148 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 149 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 150 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 151 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 152 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 153 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 154 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 155 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 156 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 157 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 158 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 159 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 160 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 161 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 162 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 163 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 164 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 165 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 166 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 167 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 168 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 169 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 170 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 171 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 172 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 173 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 174 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 175 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 176 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 177 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 178 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 179 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 180 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 181 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 182 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 183 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 184 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 185 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 186 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 187 <> gen pos G.hgh4.12"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 46%|█████████████████████████████████████████████████████▋                                                              | 31/67 [00:24<00:24,  1.45it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 188 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 189 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 190 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 191 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 192 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 193 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 194 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 195 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 196 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 197 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 198 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 199 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 200 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 201 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 202 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 203 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 204 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 205 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 206 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 207 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 208 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 209 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 210 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 211 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 212 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 213 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 214 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 215 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 216 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 217 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 218 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 219 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 220 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 221 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 222 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 223 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 224 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 225 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 226 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 227 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 228 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 229 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 230 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 231 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 232 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 233 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 234 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 235 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 236 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 237 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 238 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 239 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 240 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 241 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 242 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 243 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 244 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 245 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 246 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 247 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 248 <> gen pos G.H5.15\n",
      "assinging res_nums to gprot of 6K41\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.16"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 48%|███████████████████████████████████████████████████████▍                                                            | 32/67 [00:24<00:23,  1.47it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6OMM\n",
      "Mapping found!\n",
      "start_uni 2 > end_uni 354\n",
      "start_pdb 1 > end_pdb 353\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 4 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 5 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 6 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 7 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 8 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 9 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 10 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 11 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 12 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 13 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 14 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 15 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 16 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 17 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 18 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 19 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 20 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 21 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 22 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 23 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 24 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 25 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 26 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 27 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 28 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 29 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 30 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 31 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 32 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 33 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 34 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 35 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 36 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 37 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 38 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 39 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 40 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 41 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 42 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 43 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 44 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 45 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 46 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 47 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 48 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 49 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 50 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 51 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 52 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 53 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 54 <> gen pos G.H1.10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 181 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 182 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 183 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 184 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 185 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 186 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 187 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 188 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 189 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 190 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 191 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 192 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 193 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 194 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 195 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 196 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 197 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 198 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 199 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 200 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 201 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 202 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 203 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 204 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 205 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 206 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 207 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 208 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 209 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 210 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 211 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 212 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 213 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 214 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 215 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 216 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 217 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 218 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 219 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 220 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 221 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 222 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 223 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 224 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 225 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 226 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 227 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 228 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 229 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 230 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 231 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 232 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 240 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 241 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 242 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 243 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 244 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 245 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 246 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 247 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 248 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 249 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 250 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 251 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 252 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 253 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 254 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 255 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 256 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 257 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 258 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 259 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 260 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 261 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 262 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 263 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 264 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 265 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 266 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 267 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 268 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 269 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 270 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 271 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 272 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 273 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 274 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 275 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 276 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 277 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 278 <> gen pos G.HG.09"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 49%|█████████████████████████████████████████████████████████▏                                                          | 33/67 [00:25<00:23,  1.46it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 279 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 280 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 281 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 282 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 283 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 284 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 285 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 286 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 287 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 288 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 289 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 290 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 291 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 292 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 293 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 294 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 295 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 296 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 297 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 298 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 299 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 300 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 301 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 302 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 303 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 304 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 305 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 306 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 307 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 308 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 309 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 310 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 311 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 312 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 313 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 314 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 315 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 316 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 317 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 318 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 319 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 320 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 321 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 322 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 323 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 324 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 325 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 326 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 327 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 328 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 329 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 330 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 331 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 332 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 333 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 334 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 335 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 336 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 337 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 338 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 339 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 340 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 341 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 342 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 343 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 344 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 345 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 346 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 347 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 348 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 349 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 350 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 351 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 352 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 353 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7AUE\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 3\n",
      "start_pdb 1 > end_pdb 3\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 3 <> gen pos G.HN.09\n",
      "start_uni 39 > end_uni 394\n",
      "start_pdb 32 > end_pdb 373\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 185 <> gen pos H.hehf.06\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 186 <> gen pos H.hehf.07\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 187 <> gen pos H.HF.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 188 <> gen pos H.HF.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 189 <> gen pos H.HF.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 190 <> gen pos H.HF.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 191 <> gen pos H.HF.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 192 <> gen pos H.HF.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 193 <> gen pos G.hfs2.01\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 194 <> gen pos G.hfs2.02\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 195 <> gen pos G.hfs2.03\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 196 <> gen pos G.hfs2.04\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 197 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 198 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 199 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 200 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 201 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 202 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 203 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 204 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 207 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 208 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 209 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 210 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 211 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 212 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 213 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 214 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 215 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 216 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 217 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 218 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 219 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 220 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 221 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 222 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 223 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 224 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 225 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 226 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 227 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 228 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 229 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 230 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 231 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 232 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 233 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 234 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 235 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 243 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 244 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 245 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 246 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 247 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 248 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 249 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 250 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 251 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 252 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 253 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 254 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 255 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 256 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 257 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 258 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 259 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 260 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 261 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 262 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 263 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 264 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 265 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 266 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 267 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 268 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 269 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 270 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 271 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 272 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 273 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 274 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 275 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 276 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 277 <> gen pos G.h3s5.02\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 278 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 279 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 280 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 281 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 282 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 283 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 284 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 285 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 286 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 287 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 288 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 289 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 290 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 291 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 292 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 293 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 294 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 295 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 296 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 297 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 298 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 299 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 300 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 301 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 302 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 303 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 304 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 305 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 306 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 307 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 308 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 309 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 310 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 311 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 312 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 313 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 314 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 315 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 316 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 317 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 318 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 319 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 320 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 321 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 322 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 323 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 324 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 325 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 326 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 327 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 328 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 329 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 330 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 331 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 332 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 333 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 334 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 335 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 336 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 337 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 338 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 339 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 340 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 341 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 342 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 343 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 344 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 345 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 346 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 347 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 348 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 349 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 350 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 351 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 352 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 353 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 354 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 355 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 356 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 357 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 358 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 359 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 360 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 361 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 362 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 363 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 364 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 365 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 366 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 367 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 368 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 369 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 370 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 371 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 372 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 373 <> gen pos G.H5.12\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 51%|██████████████████████████████████████████████████████████▊                                                         | 34/67 [00:26<00:23,  1.43it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "assinging res_nums to gprot of 7CKY\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 394\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 61 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 62 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 63 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.HG.13"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 52%|████████████████████████████████████████████████████████████▌                                                       | 35/67 [00:26<00:22,  1.41it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 365 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 366 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 367 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 368 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 369 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 370 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 381 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 382 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 383 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 384 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 385 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 386 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 387 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 388 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 389 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 390 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 391 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 392 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 393 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 394 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7E33\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 54%|██████████████████████████████████████████████████████████████▎                                                     | 36/67 [00:27<00:21,  1.43it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6LFM\n",
      "Mapping found!\n",
      "start_uni 2 > end_uni 354\n",
      "start_pdb 1 > end_pdb 353\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 2 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 3 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 4 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 5 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 6 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 7 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 8 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 9 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 10 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 11 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 12 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 13 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 14 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 15 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 16 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 17 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 18 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 19 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 20 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 21 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 22 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 23 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 24 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 25 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 26 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 27 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 28 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 29 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 30 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 31 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 32 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 33 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 34 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 35 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 36 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 37 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 38 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 39 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 40 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 41 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 42 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 43 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 44 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 45 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 46 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 47 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 48 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 49 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 50 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 51 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 52 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 53 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 54 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 55 <> gen pos G.H1.11\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 178 <> idx_seq: 178 <> gen pos G.hfs2.03\n",
      "Found corr. gen num! idx_uni: 179 <> idx_seq: 179 <> gen pos G.hfs2.04\n",
      "Found corr. gen num! idx_uni: 180 <> idx_seq: 180 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 181 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 182 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 183 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 184 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 185 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 186 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 187 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 188 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 189 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 190 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 191 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 192 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 193 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 194 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 195 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 196 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 197 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 198 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 199 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 200 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 201 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 202 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 203 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 204 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 205 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 206 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 207 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 208 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 209 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 210 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 211 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 212 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 213 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 214 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 215 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 216 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 217 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 218 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 219 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 220 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 221 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 222 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 223 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 224 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 225 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 226 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 227 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 228 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 229 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 230 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 231 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 232 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 233 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 234 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 235 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 236 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 237 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 238 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 239 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 240 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 241 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 242 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 243 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 244 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 245 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 246 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 247 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 248 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 249 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 250 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 251 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 252 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 253 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 254 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 255 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 256 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 257 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 258 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 259 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 260 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 261 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 262 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 263 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 264 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 265 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 266 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 267 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 268 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 269 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 270 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 271 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 272 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 273 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 274 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 275 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 276 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 277 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 278 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 279 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 280 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 281 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 282 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 283 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 284 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 285 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 286 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 287 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 288 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 289 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 290 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 291 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 292 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 293 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 294 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 295 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 296 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 297 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 298 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 299 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 300 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 301 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 302 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 303 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 304 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 305 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 306 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 307 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 308 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 309 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 310 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 311 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 312 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 313 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 314 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 315 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 316 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 317 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 318 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 319 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 320 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 321 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 322 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 323 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 324 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 325 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 326 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 327 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 328 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 329 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 330 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 331 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 332 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 333 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 334 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 335 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 336 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 337 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 338 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 339 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 340 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 341 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 342 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 343 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 344 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 345 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 346 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 347 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 348 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 349 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 350 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 351 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 352 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 353 <> gen pos G.H5.26\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 55%|████████████████████████████████████████████████████████████████                                                    | 37/67 [00:28<00:21,  1.41it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "assinging res_nums to gprot of 7LJC\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 246\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.26\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.27\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 61 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 62 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 63 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 63 <> idx_seq: 64 <> gen pos G.H1.12\n",
      "Found corr. gen num! idx_uni: 64 <> idx_seq: 65 <> gen pos G.h1ha.01\n",
      "Found corr. gen num! idx_uni: 65 <> idx_seq: 66 <> gen pos G.h1ha.02\n",
      "Found corr. gen num! idx_uni: 66 <> idx_seq: 67 <> gen pos G.h1ha.03\n",
      "Found corr. gen num! idx_uni: 67 <> idx_seq: 68 <> gen pos G.h1ha.04\n",
      "Found corr. gen num! idx_uni: 68 <> idx_seq: 69 <> gen pos G.h1ha.05\n",
      "Found corr. gen num! idx_uni: 69 <> idx_seq: 70 <> gen pos G.h1ha.06\n",
      "Found corr. gen num! idx_uni: 70 <> idx_seq: 71 <> gen pos G.h1ha.07\n",
      "Found corr. gen num! idx_uni: 71 <> idx_seq: 72 <> gen pos G.h1ha.08\n",
      "Found corr. gen num! idx_uni: 72 <> idx_seq: 73 <> gen pos G.h1ha.09\n",
      "Found corr. gen num! idx_uni: 73 <> idx_seq: 74 <> gen pos G.h1ha.10\n",
      "Found corr. gen num! idx_uni: 74 <> idx_seq: 75 <> gen pos G.h1ha.11\n",
      "Found corr. gen num! idx_uni: 75 <> idx_seq: 76 <> gen pos G.h1ha.12\n",
      "Found corr. gen num! idx_uni: 76 <> idx_seq: 77 <> gen pos G.h1ha.13\n",
      "Found corr. gen num! idx_uni: 77 <> idx_seq: 78 <> gen pos G.h1ha.14\n",
      "Found corr. gen num! idx_uni: 78 <> idx_seq: 79 <> gen pos G.h1ha.15\n",
      "Found corr. gen num! idx_uni: 79 <> idx_seq: 80 <> gen pos G.h1ha.16\n",
      "Found corr. gen num! idx_uni: 80 <> idx_seq: 81 <> gen pos G.h1ha.17\n",
      "Found corr. gen num! idx_uni: 81 <> idx_seq: 82 <> gen pos G.h1ha.18\n",
      "Found corr. gen num! idx_uni: 82 <> idx_seq: 83 <> gen pos G.h1ha.19\n",
      "Found corr. gen num! idx_uni: 83 <> idx_seq: 84 <> gen pos G.h1ha.20\n",
      "Found corr. gen num! idx_uni: 84 <> idx_seq: 85 <> gen pos H.HA.01\n",
      "Found corr. gen num! idx_uni: 85 <> idx_seq: 86 <> gen pos H.HA.02\n",
      "Found corr. gen num! idx_uni: 86 <> idx_seq: 87 <> gen pos H.HA.03\n",
      "Found corr. gen num! idx_uni: 87 <> idx_seq: 88 <> gen pos H.HA.04\n",
      "Found corr. gen num! idx_uni: 88 <> idx_seq: 89 <> gen pos H.HA.05\n",
      "Found corr. gen num! idx_uni: 89 <> idx_seq: 90 <> gen pos H.HA.06\n",
      "Found corr. gen num! idx_uni: 90 <> idx_seq: 91 <> gen pos H.HA.07\n",
      "Found corr. gen num! idx_uni: 91 <> idx_seq: 92 <> gen pos H.HA.08\n",
      "Found corr. gen num! idx_uni: 92 <> idx_seq: 93 <> gen pos H.HA.09\n",
      "Found corr. gen num! idx_uni: 93 <> idx_seq: 94 <> gen pos H.HA.10\n",
      "Found corr. gen num! idx_uni: 94 <> idx_seq: 95 <> gen pos H.HA.11\n",
      "Found corr. gen num! idx_uni: 95 <> idx_seq: 96 <> gen pos H.HA.12\n",
      "Found corr. gen num! idx_uni: 96 <> idx_seq: 97 <> gen pos H.HA.13\n",
      "Found corr. gen num! idx_uni: 97 <> idx_seq: 98 <> gen pos H.HA.14\n",
      "Found corr. gen num! idx_uni: 98 <> idx_seq: 99 <> gen pos H.HA.15\n",
      "Found corr. gen num! idx_uni: 99 <> idx_seq: 100 <> gen pos H.HA.16\n",
      "Found corr. gen num! idx_uni: 100 <> idx_seq: 101 <> gen pos H.HA.17\n",
      "Found corr. gen num! idx_uni: 101 <> idx_seq: 102 <> gen pos H.HA.18\n",
      "Found corr. gen num! idx_uni: 102 <> idx_seq: 103 <> gen pos H.HA.19\n",
      "Found corr. gen num! idx_uni: 103 <> idx_seq: 104 <> gen pos H.HA.20\n",
      "Found corr. gen num! idx_uni: 104 <> idx_seq: 105 <> gen pos H.HA.21\n",
      "Found corr. gen num! idx_uni: 105 <> idx_seq: 106 <> gen pos H.HA.22\n",
      "Found corr. gen num! idx_uni: 106 <> idx_seq: 107 <> gen pos H.HA.23\n",
      "Found corr. gen num! idx_uni: 107 <> idx_seq: 108 <> gen pos H.HA.24\n",
      "Found corr. gen num! idx_uni: 108 <> idx_seq: 109 <> gen pos H.HA.25\n",
      "Found corr. gen num! idx_uni: 109 <> idx_seq: 110 <> gen pos H.HA.26\n",
      "Found corr. gen num! idx_uni: 110 <> idx_seq: 111 <> gen pos H.HA.27\n",
      "Found corr. gen num! idx_uni: 111 <> idx_seq: 112 <> gen pos H.HA.28\n",
      "Found corr. gen num! idx_uni: 112 <> idx_seq: 113 <> gen pos H.HA.29\n",
      "Found corr. gen num! idx_uni: 113 <> idx_seq: 114 <> gen pos H.hahb.01\n",
      "Found corr. gen num! idx_uni: 114 <> idx_seq: 115 <> gen pos H.hahb.02\n",
      "Found corr. gen num! idx_uni: 115 <> idx_seq: 116 <> gen pos H.hahb.03\n",
      "Found corr. gen num! idx_uni: 116 <> idx_seq: 117 <> gen pos H.hahb.04\n",
      "Found corr. gen num! idx_uni: 117 <> idx_seq: 118 <> gen pos H.hahb.05\n",
      "Found corr. gen num! idx_uni: 118 <> idx_seq: 119 <> gen pos H.hahb.06\n",
      "Found corr. gen num! idx_uni: 119 <> idx_seq: 120 <> gen pos H.hahb.07\n",
      "Found corr. gen num! idx_uni: 120 <> idx_seq: 121 <> gen pos H.hahb.08\n",
      "Found corr. gen num! idx_uni: 121 <> idx_seq: 122 <> gen pos H.hahb.09\n",
      "Found corr. gen num! idx_uni: 122 <> idx_seq: 123 <> gen pos H.HB.01\n",
      "Found corr. gen num! idx_uni: 123 <> idx_seq: 124 <> gen pos H.HB.02\n",
      "Found corr. gen num! idx_uni: 124 <> idx_seq: 125 <> gen pos H.HB.03\n",
      "Found corr. gen num! idx_uni: 125 <> idx_seq: 126 <> gen pos H.HB.04\n",
      "Found corr. gen num! idx_uni: 126 <> idx_seq: 127 <> gen pos H.HB.05\n",
      "Found corr. gen num! idx_uni: 127 <> idx_seq: 128 <> gen pos H.HB.06\n",
      "Found corr. gen num! idx_uni: 128 <> idx_seq: 129 <> gen pos H.HB.07\n",
      "Found corr. gen num! idx_uni: 129 <> idx_seq: 130 <> gen pos H.HB.08\n",
      "Found corr. gen num! idx_uni: 130 <> idx_seq: 131 <> gen pos H.HB.09\n",
      "Found corr. gen num! idx_uni: 131 <> idx_seq: 132 <> gen pos H.HB.10\n",
      "Found corr. gen num! idx_uni: 132 <> idx_seq: 133 <> gen pos H.HB.11\n",
      "Found corr. gen num! idx_uni: 133 <> idx_seq: 134 <> gen pos H.HB.12\n",
      "Found corr. gen num! idx_uni: 134 <> idx_seq: 135 <> gen pos H.HB.13\n",
      "Found corr. gen num! idx_uni: 135 <> idx_seq: 136 <> gen pos H.HB.14\n",
      "Found corr. gen num! idx_uni: 136 <> idx_seq: 137 <> gen pos H.hbhc.01\n",
      "Found corr. gen num! idx_uni: 137 <> idx_seq: 138 <> gen pos H.hbhc.02\n",
      "Found corr. gen num! idx_uni: 138 <> idx_seq: 139 <> gen pos H.hbhc.03\n",
      "Found corr. gen num! idx_uni: 139 <> idx_seq: 140 <> gen pos H.hbhc.04\n",
      "Found corr. gen num! idx_uni: 140 <> idx_seq: 141 <> gen pos H.hbhc.13\n",
      "Found corr. gen num! idx_uni: 141 <> idx_seq: 142 <> gen pos H.hbhc.14\n",
      "Found corr. gen num! idx_uni: 142 <> idx_seq: 143 <> gen pos H.hbhc.15\n",
      "Found corr. gen num! idx_uni: 143 <> idx_seq: 144 <> gen pos H.HC.01\n",
      "Found corr. gen num! idx_uni: 144 <> idx_seq: 145 <> gen pos H.HC.02\n",
      "Found corr. gen num! idx_uni: 145 <> idx_seq: 146 <> gen pos H.HC.03\n",
      "Found corr. gen num! idx_uni: 146 <> idx_seq: 147 <> gen pos H.HC.04\n",
      "Found corr. gen num! idx_uni: 147 <> idx_seq: 148 <> gen pos H.HC.05\n",
      "Found corr. gen num! idx_uni: 148 <> idx_seq: 149 <> gen pos H.HC.06\n",
      "Found corr. gen num! idx_uni: 149 <> idx_seq: 150 <> gen pos H.HC.07\n",
      "Found corr. gen num! idx_uni: 150 <> idx_seq: 151 <> gen pos H.HC.08\n",
      "Found corr. gen num! idx_uni: 151 <> idx_seq: 152 <> gen pos H.HC.09\n",
      "Found corr. gen num! idx_uni: 152 <> idx_seq: 153 <> gen pos H.HC.10\n",
      "Found corr. gen num! idx_uni: 153 <> idx_seq: 154 <> gen pos H.HC.11\n",
      "Found corr. gen num! idx_uni: 154 <> idx_seq: 155 <> gen pos H.HC.12\n",
      "Found corr. gen num! idx_uni: 155 <> idx_seq: 156 <> gen pos H.hchd.01\n",
      "Found corr. gen num! idx_uni: 156 <> idx_seq: 157 <> gen pos H.HD.01\n",
      "Found corr. gen num! idx_uni: 157 <> idx_seq: 158 <> gen pos H.HD.02\n",
      "Found corr. gen num! idx_uni: 158 <> idx_seq: 159 <> gen pos H.HD.03\n",
      "Found corr. gen num! idx_uni: 159 <> idx_seq: 160 <> gen pos H.HD.04\n",
      "Found corr. gen num! idx_uni: 160 <> idx_seq: 161 <> gen pos H.HD.05\n",
      "Found corr. gen num! idx_uni: 161 <> idx_seq: 162 <> gen pos H.HD.06\n",
      "Found corr. gen num! idx_uni: 162 <> idx_seq: 163 <> gen pos H.HD.07\n",
      "Found corr. gen num! idx_uni: 163 <> idx_seq: 164 <> gen pos H.HD.08\n",
      "Found corr. gen num! idx_uni: 164 <> idx_seq: 165 <> gen pos H.HD.09\n",
      "Found corr. gen num! idx_uni: 165 <> idx_seq: 166 <> gen pos H.HD.10\n",
      "Found corr. gen num! idx_uni: 166 <> idx_seq: 167 <> gen pos H.HD.11\n",
      "Found corr. gen num! idx_uni: 167 <> idx_seq: 168 <> gen pos H.HD.12\n",
      "Found corr. gen num! idx_uni: 168 <> idx_seq: 169 <> gen pos H.hdhe.01\n",
      "Found corr. gen num! idx_uni: 169 <> idx_seq: 170 <> gen pos H.hdhe.02\n",
      "Found corr. gen num! idx_uni: 170 <> idx_seq: 171 <> gen pos H.hdhe.03\n",
      "Found corr. gen num! idx_uni: 171 <> idx_seq: 172 <> gen pos H.hdhe.04\n",
      "Found corr. gen num! idx_uni: 172 <> idx_seq: 173 <> gen pos H.hdhe.05\n",
      "Found corr. gen num! idx_uni: 173 <> idx_seq: 174 <> gen pos H.HE.01\n",
      "Found corr. gen num! idx_uni: 174 <> idx_seq: 175 <> gen pos H.HE.02\n",
      "Found corr. gen num! idx_uni: 175 <> idx_seq: 176 <> gen pos H.HE.03\n",
      "Found corr. gen num! idx_uni: 176 <> idx_seq: 177 <> gen pos H.HE.04\n",
      "Found corr. gen num! idx_uni: 177 <> idx_seq: 178 <> gen pos H.HE.05\n",
      "Found corr. gen num! idx_uni: 178 <> idx_seq: 179 <> gen pos H.HE.06\n",
      "Found corr. gen num! idx_uni: 179 <> idx_seq: 180 <> gen pos H.HE.07\n",
      "Found corr. gen num! idx_uni: 180 <> idx_seq: 181 <> gen pos H.HE.08\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos H.HE.09\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos H.HE.10\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos H.HE.11\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos H.HE.12\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos H.HE.13"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 57%|█████████████████████████████████████████████████████████████████▊                                                  | 38/67 [00:28<00:19,  1.52it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos H.hehf.01\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos H.hehf.02\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos H.hehf.03\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos H.hehf.04\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos H.hehf.05\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos H.hehf.06\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos H.hehf.07\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos H.HF.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos H.HF.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos H.HF.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos H.HF.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos H.HF.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos H.HF.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.hfs2.01\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.hfs2.02\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.hfs2.03\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.hfs2.04\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "assinging res_nums to gprot of 6OS9\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 58%|███████████████████████████████████████████████████████████████████▌                                                | 39/67 [00:29<00:18,  1.50it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7CX4\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 394\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 60%|█████████████████████████████████████████████████████████████████████▎                                              | 40/67 [00:30<00:18,  1.47it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 365 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 366 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 367 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 368 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 369 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 370 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 381 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 382 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 383 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 384 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 385 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 386 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 387 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 388 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 389 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 390 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 391 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 392 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 393 <> gen pos G.H5.25\n",
      "assinging res_nums to gprot of 7E2Y\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 61%|██████████████████████████████████████████████████████████████████████▉                                             | 41/67 [00:30<00:17,  1.51it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6GDG\n",
      "Mapping found!\n",
      "start_uni 6 > end_uni 64\n",
      "start_pdb 1 > end_pdb 59\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 7 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 8 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 9 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 10 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 11 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 12 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 13 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 14 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 15 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 16 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 17 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 18 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 19 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 20 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 21 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 22 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 23 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 24 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 25 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 26 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 27 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 28 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 29 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 30 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 31 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 32 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 33 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 34 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 35 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 36 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 37 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 38 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 39 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 40 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 41 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 42 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 43 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 44 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 45 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 46 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 47 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 48 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 49 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 50 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 51 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 52 <> gen pos G.H1.05\n",
      "start_uni 204 > end_uni 394\n",
      "start_pdb 68 > end_pdb 248\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 70 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 71 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 72 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 73 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 74 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 75 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 76 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 77 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 78 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 79 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 80 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 81 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 82 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 83 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 84 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 85 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 86 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 87 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 88 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 89 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 90 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 91 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 92 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 93 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 94 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 95 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 96 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 97 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 98 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 99 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 100 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 101 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 102 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 103 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 104 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 105 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 106 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 107 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 108 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 109 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 110 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 111 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 112 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 113 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 114 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 115 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 116 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 117 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 118 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 119 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 120 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 121 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 122 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 123 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 124 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 125 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 126 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 127 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 128 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 129 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 130 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 131 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 132 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 133 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 134 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 135 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 136 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 137 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 138 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 139 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 140 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 141 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 142 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 143 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 144 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 145 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 146 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 147 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 148 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 149 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 150 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 151 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 152 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 153 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 154 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 155 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 156 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 157 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 158 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 159 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 160 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 161 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 162 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 163 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 164 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 165 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 166 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 167 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 168 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 169 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 170 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 171 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 172 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 173 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 174 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 175 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 176 <> gen pos G.hgh4.02"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 63%|████████████████████████████████████████████████████████████████████████▋                                           | 42/67 [00:31<00:15,  1.57it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 177 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 178 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 179 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 180 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 181 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 182 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 183 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 184 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 185 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 186 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 187 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 188 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 189 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 190 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 191 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 192 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 193 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 194 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 195 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 196 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 197 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 198 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 199 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 200 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 201 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 202 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 203 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 204 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 205 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 206 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 207 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 208 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 209 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 210 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 211 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 212 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 213 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 214 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 215 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 216 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 217 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 218 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 219 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 220 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 221 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 222 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 223 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 224 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 225 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 226 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 227 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 228 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 229 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 230 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 231 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 232 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 233 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 234 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 235 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 236 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 237 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 238 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 239 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 240 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 241 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 242 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 243 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 244 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 245 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 246 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 247 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 248 <> gen pos G.H5.16\n",
      "assinging res_nums to gprot of 6N4B\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 3 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 64%|██████████████████████████████████████████████████████████████████████████▍                                         | 43/67 [00:32<00:15,  1.54it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6K42\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.11\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 66%|████████████████████████████████████████████████████████████████████████████▏                                       | 44/67 [00:32<00:15,  1.52it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7CKZ\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 394\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 61 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 62 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 63 <> gen pos G.H1.11\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 67%|█████████████████████████████████████████████████████████████████████████████▉                                      | 45/67 [00:33<00:14,  1.47it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 365 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 366 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 367 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 368 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 369 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 370 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 381 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 382 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 383 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 384 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 385 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 386 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 387 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 388 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 389 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 390 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 391 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 392 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 393 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 394 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7DFL\n",
      "No mapping (no uniprot-seq) information found! ====> Assigning labels ourself!\n",
      "_assign_res_nums_mini_g\n",
      "start_uni 3 > end_uni 57\n",
      "start_pdb 1 > end_pdb 55\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 1 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 2 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 3 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 4 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 5 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 6 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 7 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 8 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 9 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 10 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 11 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 12 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 13 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 14 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 15 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 16 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 17 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 18 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 19 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 20 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 21 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 22 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 23 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 24 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 25 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 26 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 27 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 28 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 29 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 30 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 31 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 32 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 33 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 34 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 35 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 36 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 37 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 38 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 39 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 40 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 41 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 42 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 43 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 44 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 45 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 46 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 47 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 48 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 49 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 50 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 51 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 52 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 53 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 54 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 55 <> gen pos G.H1.12\n",
      "start_uni 181 > end_uni 236\n",
      "start_pdb 56 > end_pdb 237\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 180 <> idx_seq: 56 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 179 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 180 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 181 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 182 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 183 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 184 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 185 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 186 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 187 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 188 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 189 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 190 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 191 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 192 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 193 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 194 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 195 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 196 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 197 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 198 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 199 <> gen pos G.s3h2.01\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 200 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 201 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 202 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 203 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 204 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 205 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 206 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 207 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 208 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 209 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 210 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 211 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 212 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 213 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 214 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 215 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 216 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 217 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 218 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 219 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 220 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 221 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 222 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 223 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 224 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 225 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 226 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 227 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 228 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 229 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 230 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 235 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 236 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 237 <> gen pos G.s4h3.10\n",
      "start_uni 241 > end_uni 300\n",
      "start_pdb 238 > end_pdb 297\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 238 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 239 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 240 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 241 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 242 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 243 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 244 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 245 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 246 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 247 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 248 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 249 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 250 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 251 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 252 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 253 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 254 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 255 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 256 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 257 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 258 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 259 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 260 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 261 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 262 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 263 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 264 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 265 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 266 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 267 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 268 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 269 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 270 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 271 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 272 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 273 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 274 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 275 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 276 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 277 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 278 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 279 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 280 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 281 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 282 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 283 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 284 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 285 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 286 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 287 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 288 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 289 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 290 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 291 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 292 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 293 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 294 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 295 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 296 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 297 <> gen pos G.H4.07\n",
      "start_uni 302 > end_uni 316\n",
      "start_pdb 299 > end_pdb 313\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 299 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 300 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 301 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 302 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 303 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 304 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 305 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 306 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 307 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 308 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 309 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 310 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 311 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 312 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 313 <> gen pos G.h4s6.10\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 69%|███████████████████████████████████████████████████████████████████████████████▋                                    | 46/67 [00:35<00:21,  1.03s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "start_uni 318 > end_uni 355\n",
      "start_pdb 314 > end_pdb 351\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 314 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 315 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 316 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 317 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 318 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 319 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 320 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 321 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 322 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 323 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 324 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 325 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 326 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 327 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 328 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 329 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 330 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 331 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 332 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 333 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 334 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 335 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 336 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 337 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 338 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 339 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 340 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 341 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 342 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 343 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 344 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 345 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 346 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 347 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 348 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 349 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 350 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7L0R\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 1 <> idx_seq: 2 <> gen pos G.HN.02\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 3 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 70%|█████████████████████████████████████████████████████████████████████████████████▎                                  | 47/67 [00:36<00:18,  1.11it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7MBX\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 394\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.24\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.26\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.27\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 61 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 62 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 63 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 63 <> idx_seq: 64 <> gen pos G.H1.12\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.HG.13"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 72%|███████████████████████████████████████████████████████████████████████████████████                                 | 48/67 [00:36<00:16,  1.17it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 365 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 366 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 367 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 368 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 369 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 370 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 381 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 382 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 383 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 384 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 385 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 386 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 387 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 388 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 389 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 390 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 391 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 392 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 393 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 394 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6LI3\n",
      "Mapping found!\n",
      "start_uni 6 > end_uni 64\n",
      "start_pdb 1 > end_pdb 59\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 4 <> gen pos G.HN.26\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 5 <> gen pos G.HN.27\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 6 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 7 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 8 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 9 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 10 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 11 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 12 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 13 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 14 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 15 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 16 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 17 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 18 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 19 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 20 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 21 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 22 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 23 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 24 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 25 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 26 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 27 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 28 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 29 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 30 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 31 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 32 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 33 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 34 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 35 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 36 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 37 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 38 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 39 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 40 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 41 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 42 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 43 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 44 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 45 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 46 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 47 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 48 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 49 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 50 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 51 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 52 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 53 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 54 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 55 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 56 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 57 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 58 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 63 <> idx_seq: 59 <> gen pos G.H1.12\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "start_uni 204 > end_uni 394\n",
      "start_pdb 68 > end_pdb 248\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 69 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 70 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 71 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 72 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 73 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 74 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 75 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 76 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 77 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 78 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 79 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 80 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 81 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 82 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 83 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 84 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 85 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 86 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 87 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 88 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 89 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 90 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 91 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 92 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 93 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 94 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 95 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 96 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 97 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 98 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 99 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 100 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 101 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 102 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 103 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 104 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 105 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 106 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 107 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 108 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 109 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 110 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 111 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 112 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 113 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 114 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 115 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 116 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 117 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 118 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 119 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 120 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 121 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 122 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 123 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 124 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 125 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 126 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 127 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 128 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 129 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 130 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 131 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 132 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 133 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 134 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 135 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 136 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 137 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 138 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 139 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 140 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 141 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 142 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 143 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 144 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 145 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 146 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 147 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 148 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 149 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 150 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 151 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 152 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 153 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 154 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 155 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 156 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 157 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 158 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 159 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 160 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 161 <> gen pos G.HG.04"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 73%|████████████████████████████████████████████████████████████████████████████████████▊                               | 49/67 [00:37<00:14,  1.28it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 162 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 163 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 164 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 165 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 166 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 167 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 168 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 169 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 170 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 171 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 172 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 173 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 174 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 175 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 176 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 177 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 178 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 179 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 180 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 181 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 182 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 183 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 184 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 185 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 186 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 187 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 188 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 189 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 190 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 191 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 192 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 193 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 194 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 195 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 196 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 197 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 198 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 199 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 200 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 201 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 202 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 203 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 204 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 205 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 206 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 207 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 208 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 209 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 210 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 211 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 212 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 213 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 214 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 215 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 216 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 217 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 218 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 219 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 220 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 221 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 222 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 223 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 224 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 225 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 226 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 227 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 228 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 229 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 230 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 231 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 232 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 233 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 234 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 235 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 236 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 237 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 238 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 239 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 240 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 241 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 242 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 243 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 244 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 245 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 246 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 247 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 248 <> gen pos G.H5.16\n",
      "assinging res_nums to gprot of 7CFM\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 394\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 75%|██████████████████████████████████████████████████████████████████████████████████████▌                             | 50/67 [00:38<00:12,  1.31it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 365 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 366 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 367 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 368 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 369 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 370 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 381 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 382 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 383 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 384 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 385 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 386 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 387 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 388 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 389 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 390 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 391 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 392 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 393 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 394 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7CFN\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 394\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.26\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.27\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.HG.12"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 76%|████████████████████████████████████████████████████████████████████████████████████████▎                           | 51/67 [00:38<00:12,  1.33it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 365 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 366 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 367 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 368 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 369 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 370 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 381 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 382 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 383 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 384 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 385 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 386 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 387 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 388 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 389 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 390 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 391 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 392 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 393 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 394 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7L1V\n",
      "No mapping (no uniprot-seq) information found! ====> Assigning labels ourself!\n",
      "_assign_res_nums_mini_g\n",
      "start_uni 11 > end_uni 60\n",
      "start_pdb 2 > end_pdb 51\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 2 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 3 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 4 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 5 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 6 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 7 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 8 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 9 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 10 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 11 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 12 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 13 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 14 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 15 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 16 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 17 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 18 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 19 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 20 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 21 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 22 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 23 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 24 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 25 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 26 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 27 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 28 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 29 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 30 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 31 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 32 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 33 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 34 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 35 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 36 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 37 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 38 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 39 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 40 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 41 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 42 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 43 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 44 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 45 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 46 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 47 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 48 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 49 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 50 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 51 <> gen pos G.H1.08\n",
      "start_uni 207 > end_uni 253\n",
      "start_pdb 67 > end_pdb 113\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 67 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 68 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 69 <> gen pos G.S2.03\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 70 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 71 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 72 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 73 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 74 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 75 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 76 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 77 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 78 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 79 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 80 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 81 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 82 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 83 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 84 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 85 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 86 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 87 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 88 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 89 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 90 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 91 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 92 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 93 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 94 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 95 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 96 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 97 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 98 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 99 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 100 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 101 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 102 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 103 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 104 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 105 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 106 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 107 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 108 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 109 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 110 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 111 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 112 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 113 <> gen pos G.s4h3.04\n",
      "start_uni 264 > end_uni 394\n",
      "start_pdb 114 > end_pdb 244\n",
      "mini g setting index += 1\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 114 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 115 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 116 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 117 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 118 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 119 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 120 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 121 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 122 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 123 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 124 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 125 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 126 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 127 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 128 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 129 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 130 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 131 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 132 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 133 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 134 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 135 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 136 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 137 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 138 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 139 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 140 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 141 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 142 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 143 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 144 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 145 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 146 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 147 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 148 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 149 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 150 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 151 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 152 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 153 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 154 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 155 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 156 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 157 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 158 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 159 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 160 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 161 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 162 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 163 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 164 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 165 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 166 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 167 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 168 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 169 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 170 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 171 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 172 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 173 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 174 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 175 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 176 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 177 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 178 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 179 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 180 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 181 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 182 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 183 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 184 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 185 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 186 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 187 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 188 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 189 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 190 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 191 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 192 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 193 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 194 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 195 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 196 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 197 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 198 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 199 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 200 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 201 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 202 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 203 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 204 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 205 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 206 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 207 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 208 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 209 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 210 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 211 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 212 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 213 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 214 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 215 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 216 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 217 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 218 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 219 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 220 <> gen pos G.H5.02"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 78%|██████████████████████████████████████████████████████████████████████████████████████████                          | 52/67 [00:40<00:15,  1.06s/it]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 221 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 222 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 223 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 224 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 225 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 226 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 227 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 228 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 229 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 230 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 231 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 232 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 233 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 234 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 235 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 236 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 237 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 238 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 239 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 240 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 241 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 242 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 243 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 244 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7BW0\n",
      "Mapping found!\n",
      "start_uni 26 > end_uni 394\n",
      "start_pdb 19 > end_pdb 373\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 188 <> gen pos H.HF.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 189 <> gen pos H.HF.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 190 <> gen pos H.HF.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 191 <> gen pos H.HF.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 192 <> gen pos H.HF.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 193 <> gen pos G.hfs2.01\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 194 <> gen pos G.hfs2.02\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 195 <> gen pos G.hfs2.03\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 196 <> gen pos G.hfs2.04\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 197 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 198 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 199 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 200 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 201 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 202 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 203 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 204 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 205 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 206 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 207 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 208 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 209 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 210 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 211 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 212 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 213 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 214 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 215 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 216 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 217 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 218 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 219 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 220 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 221 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 222 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 223 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 224 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 225 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 226 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 227 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 228 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 229 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 230 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 231 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 232 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 233 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 234 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 242 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 243 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 244 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 245 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 246 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 247 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 248 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 249 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 250 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 251 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 252 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 253 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 254 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 255 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 256 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 257 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 258 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 259 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 260 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 261 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 262 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 263 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 264 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 265 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 266 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 267 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 268 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 269 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 270 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 271 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 272 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 273 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 274 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 275 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 276 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 277 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 278 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 279 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 280 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 281 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 282 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 283 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 284 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 285 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 286 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 287 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 288 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 289 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 290 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 291 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 292 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 293 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 294 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 295 <> gen pos G.HG.09"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 79%|███████████████████████████████████████████████████████████████████████████████████████████▊                        | 53/67 [00:41<00:13,  1.07it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 296 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 297 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 298 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 299 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 300 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 301 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 302 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 303 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 304 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 305 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 306 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 307 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 308 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 309 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 310 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 311 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 312 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 313 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 314 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 315 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 316 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 317 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 318 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 319 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 320 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 321 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 322 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 323 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 324 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 325 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 326 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 327 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 328 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 329 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 330 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 331 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 332 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 333 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 334 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 335 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 336 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 337 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 338 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 339 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 340 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 341 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 342 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 343 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 344 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 345 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 346 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 347 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 348 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 349 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 350 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 351 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 352 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 353 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 354 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 355 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 356 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 357 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 358 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 359 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 360 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 361 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 362 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 363 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 364 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 365 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 366 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 367 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 368 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 369 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 370 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 371 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 372 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 373 <> gen pos G.H5.12\n",
      "assinging res_nums to gprot of 7E2Z\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 81%|█████████████████████████████████████████████████████████████████████████████████████████████▍                      | 54/67 [00:41<00:10,  1.19it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6OIJ\n",
      "Mapping found!\n",
      "start_uni 7 > end_uni 7\n",
      "start_pdb 1 > end_pdb 1\n",
      "start_uni 37 > end_uni 359\n",
      "start_pdb 31 > end_pdb 353\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 56 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 57 <> gen pos G.H1.12\n",
      "Found corr. gen num! idx_uni: 63 <> idx_seq: 58 <> gen pos G.h1ha.01\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 181 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 182 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 183 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 184 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 185 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 186 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 187 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 188 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 189 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 190 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 191 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 192 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 193 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 194 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 195 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 196 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 197 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 198 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 199 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 200 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 201 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 202 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 203 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 204 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 205 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 206 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 207 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 208 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 209 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 210 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 211 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 212 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 213 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 214 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 215 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 216 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 217 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 218 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 219 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 220 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 221 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 222 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 223 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 224 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 225 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 226 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 227 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 228 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 229 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 230 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 231 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 232 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 237 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 238 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 239 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 240 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 241 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 242 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 243 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 244 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 245 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 246 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 247 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 248 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 249 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 250 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 251 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 252 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 253 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 254 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 255 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 256 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 257 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 258 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 259 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 260 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 261 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 262 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 263 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 264 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 265 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 266 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 267 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 268 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 269 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 270 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 271 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 272 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 273 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 274 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 275 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 276 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 277 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 278 <> gen pos G.HG.09"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 82%|███████████████████████████████████████████████████████████████████████████████████████████████▏                    | 55/67 [00:42<00:09,  1.26it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 279 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 280 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 281 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 282 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 283 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 284 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 285 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 286 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 287 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 288 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 289 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 290 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 291 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 292 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 293 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 294 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 295 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 296 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 297 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 298 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 299 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 300 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 301 <> gen pos G.H4.09\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 311 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 312 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 313 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 314 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 315 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 316 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 317 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 318 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 319 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 320 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 321 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 322 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 323 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 324 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 325 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 326 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 327 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 328 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 329 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 330 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 331 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 332 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 333 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 334 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 335 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 336 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 337 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 338 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 339 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 340 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 341 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 342 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 343 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 344 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 345 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 346 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 347 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 348 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 349 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 350 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 351 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 352 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 353 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7CKX\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 394\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 61 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 62 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 63 <> gen pos G.H1.11\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 84%|████████████████████████████████████████████████████████████████████████████████████████████████▉                   | 56/67 [00:43<00:08,  1.29it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 365 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 366 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 367 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 368 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 369 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 370 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 381 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 382 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 383 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 384 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 385 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 386 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 387 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 388 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 389 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 390 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 391 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 392 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 393 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 394 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6VMS\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 1 <> idx_seq: 2 <> gen pos G.HN.02\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 3 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 85%|██████████████████████████████████████████████████████████████████████████████████████████████████▋                 | 57/67 [00:43<00:07,  1.33it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7BZ2\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 2 > end_pdb 395\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 11 <> gen pos G.HN.27\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 12 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 13 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 14 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 15 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 16 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 17 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 18 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 19 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 20 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 21 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 22 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 23 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 24 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 25 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 26 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 27 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 28 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 29 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 30 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 31 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 32 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 33 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 34 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 35 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 36 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 37 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 38 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 39 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 40 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 41 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 42 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 43 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 44 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 45 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 46 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 47 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 48 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 49 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 50 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 51 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 52 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 53 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 54 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 55 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 56 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 57 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 58 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 59 <> gen pos G.H1.06\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 209 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 210 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 211 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 212 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 213 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 214 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 215 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 216 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 217 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 218 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 219 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 220 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 221 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 222 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 223 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 224 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 225 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 226 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 227 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 228 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 229 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 230 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 231 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 232 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 233 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 234 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 235 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 236 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 237 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 238 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 239 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 240 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 241 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 242 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 243 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 244 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 245 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 246 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 247 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 248 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 249 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 250 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 265 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 266 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 267 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 268 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 269 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 270 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 271 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 272 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 273 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 274 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 275 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 276 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 277 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 278 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 279 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 280 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 281 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 282 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 283 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 284 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 285 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 286 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 287 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 288 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 289 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 290 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 291 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 292 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 293 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 294 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 295 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 309 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 310 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 311 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 312 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 313 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 314 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 315 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 316 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 317 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 318 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 319 <> gen pos G.hgh4.08"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 87%|████████████████████████████████████████████████████████████████████████████████████████████████████▍               | 58/67 [00:44<00:06,  1.37it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 320 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 321 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 332 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 333 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 334 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 335 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 336 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 337 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 338 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 339 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 340 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 341 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 342 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 343 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 344 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 345 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 346 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 347 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 348 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 349 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 350 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 351 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 352 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 353 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 355 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 356 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 357 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 358 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 359 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 360 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 361 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 362 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 363 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 364 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 365 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 366 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 367 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 368 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 369 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 370 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 371 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 372 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 373 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 374 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 375 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 376 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 377 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 378 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 379 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 380 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 381 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 382 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 383 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 384 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 385 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 386 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 387 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 388 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 389 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 390 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 391 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 392 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 393 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 394 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 395 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7DHI\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 2 > end_pdb 395\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 13 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 14 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 15 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 16 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 17 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 18 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 19 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 20 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 21 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 22 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 23 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 24 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 25 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 26 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 27 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 28 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 29 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 30 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 31 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 32 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 33 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 34 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 35 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 36 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 37 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 38 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 39 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 40 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 41 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 42 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 43 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 44 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 45 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 46 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 47 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 48 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 49 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 50 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 51 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 52 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 53 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 54 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 55 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 56 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 57 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 58 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 209 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 210 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 211 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 212 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 213 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 214 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 215 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 216 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 217 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 218 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 219 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 220 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 221 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 222 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 223 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 224 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 225 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 226 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 227 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 228 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 229 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 230 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 231 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 232 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 233 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 234 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 235 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 236 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 237 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 238 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 239 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 240 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 241 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 242 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 243 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 244 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 245 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 246 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 247 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 248 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 249 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 250 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 251 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 252 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 263 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 264 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 265 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 266 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 267 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 268 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 269 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 270 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 271 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 272 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 273 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 274 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 275 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 276 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 277 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 278 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 279 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 280 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 281 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 282 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 283 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 284 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 285 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 286 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 287 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 288 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 289 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 290 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 291 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 292 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 293 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 294 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 295 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 296 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 297 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 298 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 299 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 300 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 301 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 302 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 303 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 304 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 309 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 310 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 311 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 312 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 313 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 314 <> gen pos G.hgh4.03"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 88%|██████████████████████████████████████████████████████████████████████████████████████████████████████▏             | 59/67 [00:45<00:05,  1.38it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 315 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 316 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 317 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 318 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 319 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 320 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 321 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 322 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 323 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 324 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 325 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 326 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 327 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 331 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 332 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 333 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 334 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 335 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 336 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 337 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 338 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 339 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 340 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 341 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 342 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 343 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 344 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 345 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 346 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 347 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 348 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 349 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 350 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 351 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 352 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 353 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 354 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 355 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 356 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 357 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 358 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 359 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 360 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 361 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 362 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 363 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 364 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 365 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 366 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 367 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 368 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 369 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 370 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 371 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 372 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 373 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 374 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 375 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 376 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 377 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 378 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 379 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 380 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 381 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 382 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 383 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 384 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 385 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 386 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 387 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 388 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 389 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 390 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 391 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 392 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 393 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 394 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 395 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6OYA\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 350\n",
      "start_pdb 10 > end_pdb 359\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 15 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 16 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 17 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 18 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 19 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 20 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 21 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 22 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 23 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 24 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 25 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 26 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 27 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 28 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 29 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 30 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 31 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 32 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 33 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 34 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 35 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 36 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 37 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 38 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 39 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 40 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 41 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 42 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 43 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 44 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 45 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 46 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 47 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 48 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 49 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 50 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 51 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 52 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 53 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 54 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 55 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 56 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 57 <> gen pos G.H1.07\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 177 <> idx_seq: 187 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 178 <> idx_seq: 188 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 179 <> idx_seq: 189 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 180 <> idx_seq: 190 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 191 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 192 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 193 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 194 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 195 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 196 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 197 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 198 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 199 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 200 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 201 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 202 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 203 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 204 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 205 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 206 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 207 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 208 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 209 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 210 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 211 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 212 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 213 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 214 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 215 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 216 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 217 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 218 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 219 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 220 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 221 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 222 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 223 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 224 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 225 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 226 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 227 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 228 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 229 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 230 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 231 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 232 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 233 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 234 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 235 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 247 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 248 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 249 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 250 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 251 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 252 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 253 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 254 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 255 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 256 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 257 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 258 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 259 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 260 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 261 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 262 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 263 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 264 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 265 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 266 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 267 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 268 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 269 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 270 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 271 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 272 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 273 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 274 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 275 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 276 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 277 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 278 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 279 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 280 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 281 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 282 <> gen pos G.HG.07"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 90%|███████████████████████████████████████████████████████████████████████████████████████████████████████▉            | 60/67 [00:46<00:04,  1.42it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 283 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 284 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 285 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 286 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 287 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 288 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 289 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 290 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 291 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 292 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 293 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 294 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 295 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 296 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 297 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 298 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 299 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 300 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 301 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 302 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 303 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 304 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 305 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 306 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 307 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 308 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 309 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 310 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 311 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 312 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 313 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 314 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 315 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 316 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 317 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 318 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 319 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 320 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 321 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 322 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 323 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 324 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 325 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 326 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 327 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 328 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 329 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 330 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 331 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 332 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 333 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 334 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 335 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 336 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 337 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 338 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 339 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 340 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 341 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 342 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 343 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 344 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 345 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 346 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 347 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 348 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 349 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 350 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 351 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 352 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 353 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 354 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 355 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 356 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 357 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 358 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 359 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7CRH\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 394\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 61 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 62 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 63 <> gen pos G.H1.11\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.HG.11"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 91%|█████████████████████████████████████████████████████████████████████████████████████████████████████████▌          | 61/67 [00:46<00:04,  1.41it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 365 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 366 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 367 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 368 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 369 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 370 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 381 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 382 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 383 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 384 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 385 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 386 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 387 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 388 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 389 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 390 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 391 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 392 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 393 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 394 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7JVQ\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 246\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.26\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.27\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 61 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 62 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 62 <> idx_seq: 63 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 63 <> idx_seq: 64 <> gen pos G.H1.12\n",
      "Found corr. gen num! idx_uni: 64 <> idx_seq: 65 <> gen pos G.h1ha.01\n",
      "Found corr. gen num! idx_uni: 65 <> idx_seq: 66 <> gen pos G.h1ha.02\n",
      "Found corr. gen num! idx_uni: 66 <> idx_seq: 67 <> gen pos G.h1ha.03\n",
      "Found corr. gen num! idx_uni: 67 <> idx_seq: 68 <> gen pos G.h1ha.04\n",
      "Found corr. gen num! idx_uni: 68 <> idx_seq: 69 <> gen pos G.h1ha.05\n",
      "Found corr. gen num! idx_uni: 69 <> idx_seq: 70 <> gen pos G.h1ha.06\n",
      "Found corr. gen num! idx_uni: 70 <> idx_seq: 71 <> gen pos G.h1ha.07\n",
      "Found corr. gen num! idx_uni: 71 <> idx_seq: 72 <> gen pos G.h1ha.08\n",
      "Found corr. gen num! idx_uni: 72 <> idx_seq: 73 <> gen pos G.h1ha.09\n",
      "Found corr. gen num! idx_uni: 73 <> idx_seq: 74 <> gen pos G.h1ha.10\n",
      "Found corr. gen num! idx_uni: 74 <> idx_seq: 75 <> gen pos G.h1ha.11\n",
      "Found corr. gen num! idx_uni: 75 <> idx_seq: 76 <> gen pos G.h1ha.12\n",
      "Found corr. gen num! idx_uni: 76 <> idx_seq: 77 <> gen pos G.h1ha.13\n",
      "Found corr. gen num! idx_uni: 77 <> idx_seq: 78 <> gen pos G.h1ha.14\n",
      "Found corr. gen num! idx_uni: 78 <> idx_seq: 79 <> gen pos G.h1ha.15\n",
      "Found corr. gen num! idx_uni: 79 <> idx_seq: 80 <> gen pos G.h1ha.16\n",
      "Found corr. gen num! idx_uni: 80 <> idx_seq: 81 <> gen pos G.h1ha.17\n",
      "Found corr. gen num! idx_uni: 81 <> idx_seq: 82 <> gen pos G.h1ha.18\n",
      "Found corr. gen num! idx_uni: 82 <> idx_seq: 83 <> gen pos G.h1ha.19\n",
      "Found corr. gen num! idx_uni: 83 <> idx_seq: 84 <> gen pos G.h1ha.20\n",
      "Found corr. gen num! idx_uni: 84 <> idx_seq: 85 <> gen pos H.HA.01\n",
      "Found corr. gen num! idx_uni: 85 <> idx_seq: 86 <> gen pos H.HA.02\n",
      "Found corr. gen num! idx_uni: 86 <> idx_seq: 87 <> gen pos H.HA.03\n",
      "Found corr. gen num! idx_uni: 87 <> idx_seq: 88 <> gen pos H.HA.04\n",
      "Found corr. gen num! idx_uni: 88 <> idx_seq: 89 <> gen pos H.HA.05\n",
      "Found corr. gen num! idx_uni: 89 <> idx_seq: 90 <> gen pos H.HA.06\n",
      "Found corr. gen num! idx_uni: 90 <> idx_seq: 91 <> gen pos H.HA.07\n",
      "Found corr. gen num! idx_uni: 91 <> idx_seq: 92 <> gen pos H.HA.08\n",
      "Found corr. gen num! idx_uni: 92 <> idx_seq: 93 <> gen pos H.HA.09\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 93 <> idx_seq: 94 <> gen pos H.HA.10\n",
      "Found corr. gen num! idx_uni: 94 <> idx_seq: 95 <> gen pos H.HA.11\n",
      "Found corr. gen num! idx_uni: 95 <> idx_seq: 96 <> gen pos H.HA.12\n",
      "Found corr. gen num! idx_uni: 96 <> idx_seq: 97 <> gen pos H.HA.13\n",
      "Found corr. gen num! idx_uni: 97 <> idx_seq: 98 <> gen pos H.HA.14\n",
      "Found corr. gen num! idx_uni: 98 <> idx_seq: 99 <> gen pos H.HA.15\n",
      "Found corr. gen num! idx_uni: 99 <> idx_seq: 100 <> gen pos H.HA.16\n",
      "Found corr. gen num! idx_uni: 100 <> idx_seq: 101 <> gen pos H.HA.17\n",
      "Found corr. gen num! idx_uni: 101 <> idx_seq: 102 <> gen pos H.HA.18\n",
      "Found corr. gen num! idx_uni: 102 <> idx_seq: 103 <> gen pos H.HA.19\n",
      "Found corr. gen num! idx_uni: 103 <> idx_seq: 104 <> gen pos H.HA.20\n",
      "Found corr. gen num! idx_uni: 104 <> idx_seq: 105 <> gen pos H.HA.21\n",
      "Found corr. gen num! idx_uni: 105 <> idx_seq: 106 <> gen pos H.HA.22\n",
      "Found corr. gen num! idx_uni: 106 <> idx_seq: 107 <> gen pos H.HA.23\n",
      "Found corr. gen num! idx_uni: 107 <> idx_seq: 108 <> gen pos H.HA.24\n",
      "Found corr. gen num! idx_uni: 108 <> idx_seq: 109 <> gen pos H.HA.25\n",
      "Found corr. gen num! idx_uni: 109 <> idx_seq: 110 <> gen pos H.HA.26\n",
      "Found corr. gen num! idx_uni: 110 <> idx_seq: 111 <> gen pos H.HA.27\n",
      "Found corr. gen num! idx_uni: 111 <> idx_seq: 112 <> gen pos H.HA.28\n",
      "Found corr. gen num! idx_uni: 112 <> idx_seq: 113 <> gen pos H.HA.29\n",
      "Found corr. gen num! idx_uni: 113 <> idx_seq: 114 <> gen pos H.hahb.01\n",
      "Found corr. gen num! idx_uni: 114 <> idx_seq: 115 <> gen pos H.hahb.02\n",
      "Found corr. gen num! idx_uni: 115 <> idx_seq: 116 <> gen pos H.hahb.03\n",
      "Found corr. gen num! idx_uni: 116 <> idx_seq: 117 <> gen pos H.hahb.04\n",
      "Found corr. gen num! idx_uni: 117 <> idx_seq: 118 <> gen pos H.hahb.05\n",
      "Found corr. gen num! idx_uni: 118 <> idx_seq: 119 <> gen pos H.hahb.06\n",
      "Found corr. gen num! idx_uni: 119 <> idx_seq: 120 <> gen pos H.hahb.07\n",
      "Found corr. gen num! idx_uni: 120 <> idx_seq: 121 <> gen pos H.hahb.08\n",
      "Found corr. gen num! idx_uni: 121 <> idx_seq: 122 <> gen pos H.hahb.09\n",
      "Found corr. gen num! idx_uni: 122 <> idx_seq: 123 <> gen pos H.HB.01\n",
      "Found corr. gen num! idx_uni: 123 <> idx_seq: 124 <> gen pos H.HB.02\n",
      "Found corr. gen num! idx_uni: 124 <> idx_seq: 125 <> gen pos H.HB.03\n",
      "Found corr. gen num! idx_uni: 125 <> idx_seq: 126 <> gen pos H.HB.04\n",
      "Found corr. gen num! idx_uni: 126 <> idx_seq: 127 <> gen pos H.HB.05\n",
      "Found corr. gen num! idx_uni: 127 <> idx_seq: 128 <> gen pos H.HB.06\n",
      "Found corr. gen num! idx_uni: 128 <> idx_seq: 129 <> gen pos H.HB.07\n",
      "Found corr. gen num! idx_uni: 129 <> idx_seq: 130 <> gen pos H.HB.08\n",
      "Found corr. gen num! idx_uni: 130 <> idx_seq: 131 <> gen pos H.HB.09\n",
      "Found corr. gen num! idx_uni: 131 <> idx_seq: 132 <> gen pos H.HB.10\n",
      "Found corr. gen num! idx_uni: 132 <> idx_seq: 133 <> gen pos H.HB.11\n",
      "Found corr. gen num! idx_uni: 133 <> idx_seq: 134 <> gen pos H.HB.12\n",
      "Found corr. gen num! idx_uni: 134 <> idx_seq: 135 <> gen pos H.HB.13\n",
      "Found corr. gen num! idx_uni: 135 <> idx_seq: 136 <> gen pos H.HB.14\n",
      "Found corr. gen num! idx_uni: 136 <> idx_seq: 137 <> gen pos H.hbhc.01\n",
      "Found corr. gen num! idx_uni: 137 <> idx_seq: 138 <> gen pos H.hbhc.02\n",
      "Found corr. gen num! idx_uni: 138 <> idx_seq: 139 <> gen pos H.hbhc.03\n",
      "Found corr. gen num! idx_uni: 139 <> idx_seq: 140 <> gen pos H.hbhc.04\n",
      "Found corr. gen num! idx_uni: 140 <> idx_seq: 141 <> gen pos H.hbhc.13\n",
      "Found corr. gen num! idx_uni: 141 <> idx_seq: 142 <> gen pos H.hbhc.14\n",
      "Found corr. gen num! idx_uni: 142 <> idx_seq: 143 <> gen pos H.hbhc.15\n",
      "Found corr. gen num! idx_uni: 143 <> idx_seq: 144 <> gen pos H.HC.01\n",
      "Found corr. gen num! idx_uni: 144 <> idx_seq: 145 <> gen pos H.HC.02\n",
      "Found corr. gen num! idx_uni: 145 <> idx_seq: 146 <> gen pos H.HC.03\n",
      "Found corr. gen num! idx_uni: 146 <> idx_seq: 147 <> gen pos H.HC.04\n",
      "Found corr. gen num! idx_uni: 147 <> idx_seq: 148 <> gen pos H.HC.05\n",
      "Found corr. gen num! idx_uni: 148 <> idx_seq: 149 <> gen pos H.HC.06\n",
      "Found corr. gen num! idx_uni: 149 <> idx_seq: 150 <> gen pos H.HC.07\n",
      "Found corr. gen num! idx_uni: 150 <> idx_seq: 151 <> gen pos H.HC.08\n",
      "Found corr. gen num! idx_uni: 151 <> idx_seq: 152 <> gen pos H.HC.09\n",
      "Found corr. gen num! idx_uni: 152 <> idx_seq: 153 <> gen pos H.HC.10\n",
      "Found corr. gen num! idx_uni: 153 <> idx_seq: 154 <> gen pos H.HC.11\n",
      "Found corr. gen num! idx_uni: 154 <> idx_seq: 155 <> gen pos H.HC.12\n",
      "Found corr. gen num! idx_uni: 155 <> idx_seq: 156 <> gen pos H.hchd.01\n",
      "Found corr. gen num! idx_uni: 156 <> idx_seq: 157 <> gen pos H.HD.01\n",
      "Found corr. gen num! idx_uni: 157 <> idx_seq: 158 <> gen pos H.HD.02\n",
      "Found corr. gen num! idx_uni: 158 <> idx_seq: 159 <> gen pos H.HD.03\n",
      "Found corr. gen num! idx_uni: 159 <> idx_seq: 160 <> gen pos H.HD.04\n",
      "Found corr. gen num! idx_uni: 160 <> idx_seq: 161 <> gen pos H.HD.05\n",
      "Found corr. gen num! idx_uni: 161 <> idx_seq: 162 <> gen pos H.HD.06\n",
      "Found corr. gen num! idx_uni: 162 <> idx_seq: 163 <> gen pos H.HD.07\n",
      "Found corr. gen num! idx_uni: 163 <> idx_seq: 164 <> gen pos H.HD.08\n",
      "Found corr. gen num! idx_uni: 164 <> idx_seq: 165 <> gen pos H.HD.09\n",
      "Found corr. gen num! idx_uni: 165 <> idx_seq: 166 <> gen pos H.HD.10\n",
      "Found corr. gen num! idx_uni: 166 <> idx_seq: 167 <> gen pos H.HD.11\n",
      "Found corr. gen num! idx_uni: 167 <> idx_seq: 168 <> gen pos H.HD.12\n",
      "Found corr. gen num! idx_uni: 168 <> idx_seq: 169 <> gen pos H.hdhe.01\n",
      "Found corr. gen num! idx_uni: 169 <> idx_seq: 170 <> gen pos H.hdhe.02\n",
      "Found corr. gen num! idx_uni: 170 <> idx_seq: 171 <> gen pos H.hdhe.03\n",
      "Found corr. gen num! idx_uni: 171 <> idx_seq: 172 <> gen pos H.hdhe.04\n",
      "Found corr. gen num! idx_uni: 172 <> idx_seq: 173 <> gen pos H.hdhe.05\n",
      "Found corr. gen num! idx_uni: 173 <> idx_seq: 174 <> gen pos H.HE.01\n",
      "Found corr. gen num! idx_uni: 174 <> idx_seq: 175 <> gen pos H.HE.02\n",
      "Found corr. gen num! idx_uni: 175 <> idx_seq: 176 <> gen pos H.HE.03\n",
      "Found corr. gen num! idx_uni: 176 <> idx_seq: 177 <> gen pos H.HE.04\n",
      "Found corr. gen num! idx_uni: 177 <> idx_seq: 178 <> gen pos H.HE.05\n",
      "Found corr. gen num! idx_uni: 178 <> idx_seq: 179 <> gen pos H.HE.06\n",
      "Found corr. gen num! idx_uni: 179 <> idx_seq: 180 <> gen pos H.HE.07\n",
      "Found corr. gen num! idx_uni: 180 <> idx_seq: 181 <> gen pos H.HE.08\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos H.HE.09\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos H.HE.10\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos H.HE.11\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos H.HE.12\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos H.HE.13\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos H.hehf.01\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos H.hehf.02\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos H.hehf.03\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos H.hehf.04\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos H.hehf.05\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos H.hehf.06\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos H.hehf.07\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos H.HF.01"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 93%|███████████████████████████████████████████████████████████████████████████████████████████████████████████▎        | 62/67 [00:47<00:03,  1.52it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos H.HF.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos H.HF.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos H.HF.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos H.HF.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos H.HF.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.hfs2.01\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.hfs2.02\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.hfs2.03\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.hfs2.04\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "assinging res_nums to gprot of 7CMU\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.11\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.12\n",
      "Found corr. gen num! idx_uni: 180 <> idx_seq: 181 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 94%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████       | 63/67 [00:48<00:02,  1.49it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 6KPG\n",
      "Mapping found!\n",
      "start_uni 2 > end_uni 354\n",
      "start_pdb 1 > end_pdb 353\n",
      "Found corr. gen num! idx_uni: 2 <> idx_seq: 2 <> gen pos G.HN.09\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 3 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 4 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 5 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 6 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 7 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 8 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 9 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 10 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 11 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 12 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 13 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 14 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 15 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 16 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 17 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 18 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 19 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 20 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 21 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 22 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 23 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 24 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 25 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 26 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 27 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 28 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 29 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 30 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 31 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 32 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 33 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 34 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 35 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 36 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 37 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 38 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 39 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 40 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 41 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 42 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 43 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 44 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 45 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 46 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 47 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 48 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 49 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 50 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 51 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 52 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 53 <> gen pos G.H1.09\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 181 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 182 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 183 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 184 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 185 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 186 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 187 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 188 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 189 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 190 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 191 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 192 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 193 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 194 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 195 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 196 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 197 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 198 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 199 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 200 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 201 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 202 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 203 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 204 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 205 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 206 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 207 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 208 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 209 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 210 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 211 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 212 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 213 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 214 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 215 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 216 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 217 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 218 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 219 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 220 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 221 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 222 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 223 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 224 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 225 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 226 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 227 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 228 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 229 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 230 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 231 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 232 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 233 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 234 <> gen pos G.s4h3.09\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 235 <> gen pos G.s4h3.10\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 236 <> gen pos G.s4h3.11\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 237 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 238 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 239 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 240 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 241 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 242 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 243 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 244 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 245 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 246 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 247 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 248 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 249 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 250 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 251 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 252 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 253 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 254 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 255 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 256 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 257 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 258 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 259 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 260 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 261 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 262 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 263 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 264 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 265 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 266 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 267 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 268 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 269 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 270 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 271 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 272 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 273 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 274 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 275 <> gen pos G.HG.06"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 96%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████▊     | 64/67 [00:48<00:02,  1.47it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 276 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 277 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 278 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 279 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 280 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 281 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 282 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 283 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 284 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 285 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 286 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 287 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 288 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 289 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 290 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 291 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 292 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 293 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 294 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 295 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 296 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 297 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 298 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 299 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 300 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 301 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 302 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 303 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 304 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 305 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 306 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 307 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 308 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 309 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 310 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 311 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 312 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 313 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 314 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 315 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 316 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 317 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 318 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 319 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 320 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 321 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 322 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 323 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 324 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 325 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 326 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 327 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 328 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 329 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 330 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 331 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 332 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 333 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 334 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 335 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 336 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 337 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 338 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 339 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 340 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 341 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 342 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 343 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 344 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 345 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 346 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 347 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 348 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 349 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 350 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 351 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 352 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 353 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7CX2\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 394\n",
      "start_pdb 1 > end_pdb 394\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 56 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 57 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 58 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 59 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 60 <> gen pos G.H1.08\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 235 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 236 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 237 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 238 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 239 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 240 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.HG.11"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 97%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████▌   | 65/67 [00:49<00:01,  1.45it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 355 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 356 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 357 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 358 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 359 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 360 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 361 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 362 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 363 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 364 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 365 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 366 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 367 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 368 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 369 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 370 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 371 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 372 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 373 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 374 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 375 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 376 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 377 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 378 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 379 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 380 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 381 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 382 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 383 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 384 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 385 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 386 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 387 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 388 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 389 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 390 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 391 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 392 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 393 <> gen pos G.H5.25\n",
      "assinging res_nums to gprot of 7E2X\n",
      "Mapping found!\n",
      "start_uni 1 > end_uni 354\n",
      "start_pdb 1 > end_pdb 354\n",
      "Found corr. gen num! idx_uni: 3 <> idx_seq: 4 <> gen pos G.HN.10\n",
      "Found corr. gen num! idx_uni: 4 <> idx_seq: 5 <> gen pos G.HN.11\n",
      "Found corr. gen num! idx_uni: 5 <> idx_seq: 6 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 6 <> idx_seq: 7 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 7 <> idx_seq: 8 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 9 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 10 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 11 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 12 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 13 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 14 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 15 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 16 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 17 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 18 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 19 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 20 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 21 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 22 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 23 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 24 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 25 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 26 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 27 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 28 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 29 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 30 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 31 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 32 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 33 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 34 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 35 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 36 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 37 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 38 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 39 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 40 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 41 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 42 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 43 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 44 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 45 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 46 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 47 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 48 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 49 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 50 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 51 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 52 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 53 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 54 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 55 <> gen pos G.H1.10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 181 <> idx_seq: 182 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 182 <> idx_seq: 183 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 183 <> idx_seq: 184 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 184 <> idx_seq: 185 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 185 <> idx_seq: 186 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 186 <> idx_seq: 187 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 187 <> idx_seq: 188 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 188 <> idx_seq: 189 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 189 <> idx_seq: 190 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 190 <> idx_seq: 191 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 191 <> idx_seq: 192 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 192 <> idx_seq: 193 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 193 <> idx_seq: 194 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 194 <> idx_seq: 195 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 195 <> idx_seq: 196 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 196 <> idx_seq: 197 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 197 <> idx_seq: 198 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 198 <> idx_seq: 199 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 199 <> idx_seq: 200 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 200 <> idx_seq: 201 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 201 <> idx_seq: 202 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 202 <> idx_seq: 203 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 204 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 205 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 206 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 207 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 208 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 209 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 210 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 211 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 212 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 213 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 214 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 215 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 216 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 217 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 218 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 219 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 220 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 221 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 222 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 223 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 224 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 225 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 226 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 227 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 228 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 229 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 230 <> gen pos G.s4h3.04\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 231 <> gen pos G.s4h3.05\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 232 <> gen pos G.s4h3.06\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 233 <> gen pos G.s4h3.07\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 234 <> gen pos G.s4h3.08\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 241 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 242 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 243 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 244 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 245 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 246 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 247 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 248 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 249 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 250 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 251 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 252 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 252 <> idx_seq: 253 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 253 <> idx_seq: 254 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 254 <> idx_seq: 255 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 255 <> idx_seq: 256 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 256 <> idx_seq: 257 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 257 <> idx_seq: 258 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 258 <> idx_seq: 259 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 259 <> idx_seq: 260 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 261 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 262 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 263 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 264 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 265 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 266 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 267 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 268 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 269 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 270 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 271 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 272 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 273 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 274 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 275 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 276 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 277 <> gen pos G.HG.07\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 278 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 279 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 280 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 281 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 282 <> gen pos G.HG.14"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\r",
      " 99%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████▎ | 66/67 [00:50<00:00,  1.49it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 283 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 284 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 285 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 286 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 287 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 288 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 289 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 290 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 291 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 292 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 293 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 294 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 295 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 296 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 297 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 298 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 299 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 300 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 301 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 302 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 303 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 304 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 305 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 306 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 307 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 308 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 309 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 310 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 311 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 312 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 313 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 314 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 315 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 316 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 317 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 318 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 319 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 320 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 321 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 322 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 323 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 324 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 325 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 326 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 327 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 328 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 329 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 330 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 331 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 332 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 333 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 334 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 335 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 336 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 337 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 338 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 339 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 340 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 341 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 342 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 343 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 344 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 345 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 346 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 347 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 348 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 349 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 350 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 351 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 352 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 353 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 354 <> gen pos G.H5.26\n",
      "assinging res_nums to gprot of 7JVP\n",
      "Mapping found!\n",
      "start_uni 2 > end_uni 394\n",
      "start_pdb 1 > end_pdb 393\n",
      "Found corr. gen num! idx_uni: 8 <> idx_seq: 8 <> gen pos G.HN.26\n",
      "Found corr. gen num! idx_uni: 9 <> idx_seq: 9 <> gen pos G.HN.27\n",
      "Found corr. gen num! idx_uni: 10 <> idx_seq: 10 <> gen pos G.HN.28\n",
      "Found corr. gen num! idx_uni: 11 <> idx_seq: 11 <> gen pos G.HN.29\n",
      "Found corr. gen num! idx_uni: 12 <> idx_seq: 12 <> gen pos G.HN.30\n",
      "Found corr. gen num! idx_uni: 13 <> idx_seq: 13 <> gen pos G.HN.31\n",
      "Found corr. gen num! idx_uni: 14 <> idx_seq: 14 <> gen pos G.HN.32\n",
      "Found corr. gen num! idx_uni: 15 <> idx_seq: 15 <> gen pos G.HN.33\n",
      "Found corr. gen num! idx_uni: 16 <> idx_seq: 16 <> gen pos G.HN.34\n",
      "Found corr. gen num! idx_uni: 17 <> idx_seq: 17 <> gen pos G.HN.35\n",
      "Found corr. gen num! idx_uni: 18 <> idx_seq: 18 <> gen pos G.HN.36\n",
      "Found corr. gen num! idx_uni: 19 <> idx_seq: 19 <> gen pos G.HN.37\n",
      "Found corr. gen num! idx_uni: 20 <> idx_seq: 20 <> gen pos G.HN.38\n",
      "Found corr. gen num! idx_uni: 21 <> idx_seq: 21 <> gen pos G.HN.39\n",
      "Found corr. gen num! idx_uni: 22 <> idx_seq: 22 <> gen pos G.HN.40\n",
      "Found corr. gen num! idx_uni: 23 <> idx_seq: 23 <> gen pos G.HN.41\n",
      "Found corr. gen num! idx_uni: 24 <> idx_seq: 24 <> gen pos G.HN.42\n",
      "Found corr. gen num! idx_uni: 25 <> idx_seq: 25 <> gen pos G.HN.43\n",
      "Found corr. gen num! idx_uni: 26 <> idx_seq: 26 <> gen pos G.HN.44\n",
      "Found corr. gen num! idx_uni: 27 <> idx_seq: 27 <> gen pos G.HN.45\n",
      "Found corr. gen num! idx_uni: 28 <> idx_seq: 28 <> gen pos G.HN.46\n",
      "Found corr. gen num! idx_uni: 29 <> idx_seq: 29 <> gen pos G.HN.47\n",
      "Found corr. gen num! idx_uni: 30 <> idx_seq: 30 <> gen pos G.HN.48\n",
      "Found corr. gen num! idx_uni: 31 <> idx_seq: 31 <> gen pos G.HN.49\n",
      "Found corr. gen num! idx_uni: 32 <> idx_seq: 32 <> gen pos G.HN.50\n",
      "Found corr. gen num! idx_uni: 33 <> idx_seq: 33 <> gen pos G.HN.51\n",
      "Found corr. gen num! idx_uni: 34 <> idx_seq: 34 <> gen pos G.HN.52\n",
      "Found corr. gen num! idx_uni: 35 <> idx_seq: 35 <> gen pos G.HN.53\n",
      "Found corr. gen num! idx_uni: 36 <> idx_seq: 36 <> gen pos G.hns1.01\n",
      "Found corr. gen num! idx_uni: 37 <> idx_seq: 37 <> gen pos G.hns1.02\n",
      "Found corr. gen num! idx_uni: 38 <> idx_seq: 38 <> gen pos G.hns1.03\n",
      "Found corr. gen num! idx_uni: 39 <> idx_seq: 39 <> gen pos G.S1.01\n",
      "Found corr. gen num! idx_uni: 40 <> idx_seq: 40 <> gen pos G.S1.02\n",
      "Found corr. gen num! idx_uni: 41 <> idx_seq: 41 <> gen pos G.S1.03\n",
      "Found corr. gen num! idx_uni: 42 <> idx_seq: 42 <> gen pos G.S1.04\n",
      "Found corr. gen num! idx_uni: 43 <> idx_seq: 43 <> gen pos G.S1.05\n",
      "Found corr. gen num! idx_uni: 44 <> idx_seq: 44 <> gen pos G.S1.06\n",
      "Found corr. gen num! idx_uni: 45 <> idx_seq: 45 <> gen pos G.S1.07\n",
      "Found corr. gen num! idx_uni: 46 <> idx_seq: 46 <> gen pos G.s1h1.01\n",
      "Found corr. gen num! idx_uni: 47 <> idx_seq: 47 <> gen pos G.s1h1.02\n",
      "Found corr. gen num! idx_uni: 48 <> idx_seq: 48 <> gen pos G.s1h1.03\n",
      "Found corr. gen num! idx_uni: 49 <> idx_seq: 49 <> gen pos G.s1h1.04\n",
      "Found corr. gen num! idx_uni: 50 <> idx_seq: 50 <> gen pos G.s1h1.05\n",
      "Found corr. gen num! idx_uni: 51 <> idx_seq: 51 <> gen pos G.s1h1.06\n",
      "Found corr. gen num! idx_uni: 52 <> idx_seq: 52 <> gen pos G.H1.01\n",
      "Found corr. gen num! idx_uni: 53 <> idx_seq: 53 <> gen pos G.H1.02\n",
      "Found corr. gen num! idx_uni: 54 <> idx_seq: 54 <> gen pos G.H1.03\n",
      "Found corr. gen num! idx_uni: 55 <> idx_seq: 55 <> gen pos G.H1.04\n",
      "Found corr. gen num! idx_uni: 56 <> idx_seq: 56 <> gen pos G.H1.05\n",
      "Found corr. gen num! idx_uni: 57 <> idx_seq: 57 <> gen pos G.H1.06\n",
      "Found corr. gen num! idx_uni: 58 <> idx_seq: 58 <> gen pos G.H1.07\n",
      "Found corr. gen num! idx_uni: 59 <> idx_seq: 59 <> gen pos G.H1.08\n",
      "Found corr. gen num! idx_uni: 60 <> idx_seq: 60 <> gen pos G.H1.09\n",
      "Found corr. gen num! idx_uni: 61 <> idx_seq: 61 <> gen pos G.H1.10\n"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Found corr. gen num! idx_uni: 203 <> idx_seq: 203 <> gen pos G.hfs2.05\n",
      "Found corr. gen num! idx_uni: 204 <> idx_seq: 204 <> gen pos G.hfs2.06\n",
      "Found corr. gen num! idx_uni: 205 <> idx_seq: 205 <> gen pos G.hfs2.07\n",
      "Found corr. gen num! idx_uni: 206 <> idx_seq: 206 <> gen pos G.S2.01\n",
      "Found corr. gen num! idx_uni: 207 <> idx_seq: 207 <> gen pos G.S2.02\n",
      "Found corr. gen num! idx_uni: 208 <> idx_seq: 208 <> gen pos G.S2.03\n",
      "Found corr. gen num! idx_uni: 209 <> idx_seq: 209 <> gen pos G.S2.04\n",
      "Found corr. gen num! idx_uni: 210 <> idx_seq: 210 <> gen pos G.S2.05\n",
      "Found corr. gen num! idx_uni: 211 <> idx_seq: 211 <> gen pos G.S2.06\n",
      "Found corr. gen num! idx_uni: 212 <> idx_seq: 212 <> gen pos G.S2.07\n",
      "Found corr. gen num! idx_uni: 213 <> idx_seq: 213 <> gen pos G.S2.08\n",
      "Found corr. gen num! idx_uni: 214 <> idx_seq: 214 <> gen pos G.s2s3.01\n",
      "Found corr. gen num! idx_uni: 215 <> idx_seq: 215 <> gen pos G.s2s3.02\n",
      "Found corr. gen num! idx_uni: 216 <> idx_seq: 216 <> gen pos G.S3.01\n",
      "Found corr. gen num! idx_uni: 217 <> idx_seq: 217 <> gen pos G.S3.02\n",
      "Found corr. gen num! idx_uni: 218 <> idx_seq: 218 <> gen pos G.S3.03\n",
      "Found corr. gen num! idx_uni: 219 <> idx_seq: 219 <> gen pos G.S3.04\n",
      "Found corr. gen num! idx_uni: 220 <> idx_seq: 220 <> gen pos G.S3.05\n",
      "Found corr. gen num! idx_uni: 221 <> idx_seq: 221 <> gen pos G.S3.06\n",
      "Found corr. gen num! idx_uni: 222 <> idx_seq: 222 <> gen pos G.S3.07\n",
      "Found corr. gen num! idx_uni: 223 <> idx_seq: 223 <> gen pos G.S3.08\n",
      "Found corr. gen num! idx_uni: 224 <> idx_seq: 224 <> gen pos G.s3h2.01\n",
      "Found corr. gen num! idx_uni: 225 <> idx_seq: 225 <> gen pos G.s3h2.02\n",
      "Found corr. gen num! idx_uni: 226 <> idx_seq: 226 <> gen pos G.s3h2.03\n",
      "Found corr. gen num! idx_uni: 227 <> idx_seq: 227 <> gen pos G.H2.01\n",
      "Found corr. gen num! idx_uni: 228 <> idx_seq: 228 <> gen pos G.H2.02\n",
      "Found corr. gen num! idx_uni: 229 <> idx_seq: 229 <> gen pos G.H2.03\n",
      "Found corr. gen num! idx_uni: 230 <> idx_seq: 230 <> gen pos G.H2.04\n",
      "Found corr. gen num! idx_uni: 231 <> idx_seq: 231 <> gen pos G.H2.05\n",
      "Found corr. gen num! idx_uni: 232 <> idx_seq: 232 <> gen pos G.H2.06\n",
      "Found corr. gen num! idx_uni: 233 <> idx_seq: 233 <> gen pos G.H2.07\n",
      "Found corr. gen num! idx_uni: 234 <> idx_seq: 234 <> gen pos G.H2.08\n",
      "Found corr. gen num! idx_uni: 235 <> idx_seq: 235 <> gen pos G.H2.09\n",
      "Found corr. gen num! idx_uni: 236 <> idx_seq: 236 <> gen pos G.H2.10\n",
      "Found corr. gen num! idx_uni: 237 <> idx_seq: 237 <> gen pos G.h2s4.01\n",
      "Found corr. gen num! idx_uni: 238 <> idx_seq: 238 <> gen pos G.h2s4.02\n",
      "Found corr. gen num! idx_uni: 239 <> idx_seq: 239 <> gen pos G.h2s4.03\n",
      "Found corr. gen num! idx_uni: 240 <> idx_seq: 240 <> gen pos G.h2s4.04\n",
      "Found corr. gen num! idx_uni: 241 <> idx_seq: 241 <> gen pos G.h2s4.05\n",
      "Found corr. gen num! idx_uni: 242 <> idx_seq: 242 <> gen pos G.S4.01\n",
      "Found corr. gen num! idx_uni: 243 <> idx_seq: 243 <> gen pos G.S4.02\n",
      "Found corr. gen num! idx_uni: 244 <> idx_seq: 244 <> gen pos G.S4.03\n",
      "Found corr. gen num! idx_uni: 245 <> idx_seq: 245 <> gen pos G.S4.04\n",
      "Found corr. gen num! idx_uni: 246 <> idx_seq: 246 <> gen pos G.S4.05\n",
      "Found corr. gen num! idx_uni: 247 <> idx_seq: 247 <> gen pos G.S4.06\n",
      "Found corr. gen num! idx_uni: 248 <> idx_seq: 248 <> gen pos G.S4.07\n",
      "Found corr. gen num! idx_uni: 249 <> idx_seq: 249 <> gen pos G.s4h3.01\n",
      "Found corr. gen num! idx_uni: 250 <> idx_seq: 250 <> gen pos G.s4h3.02\n",
      "Found corr. gen num! idx_uni: 251 <> idx_seq: 251 <> gen pos G.s4h3.03\n",
      "Found corr. gen num! idx_uni: 260 <> idx_seq: 260 <> gen pos G.s4h3.12\n",
      "Found corr. gen num! idx_uni: 261 <> idx_seq: 261 <> gen pos G.s4h3.13\n",
      "Found corr. gen num! idx_uni: 262 <> idx_seq: 262 <> gen pos G.s4h3.14\n",
      "Found corr. gen num! idx_uni: 263 <> idx_seq: 263 <> gen pos G.s4h3.15\n",
      "Found corr. gen num! idx_uni: 264 <> idx_seq: 264 <> gen pos G.H3.01\n",
      "Found corr. gen num! idx_uni: 265 <> idx_seq: 265 <> gen pos G.H3.02\n",
      "Found corr. gen num! idx_uni: 266 <> idx_seq: 266 <> gen pos G.H3.03\n",
      "Found corr. gen num! idx_uni: 267 <> idx_seq: 267 <> gen pos G.H3.04\n",
      "Found corr. gen num! idx_uni: 268 <> idx_seq: 268 <> gen pos G.H3.05\n",
      "Found corr. gen num! idx_uni: 269 <> idx_seq: 269 <> gen pos G.H3.06\n",
      "Found corr. gen num! idx_uni: 270 <> idx_seq: 270 <> gen pos G.H3.07\n",
      "Found corr. gen num! idx_uni: 271 <> idx_seq: 271 <> gen pos G.H3.08\n",
      "Found corr. gen num! idx_uni: 272 <> idx_seq: 272 <> gen pos G.H3.09\n",
      "Found corr. gen num! idx_uni: 273 <> idx_seq: 273 <> gen pos G.H3.10\n",
      "Found corr. gen num! idx_uni: 274 <> idx_seq: 274 <> gen pos G.H3.11\n",
      "Found corr. gen num! idx_uni: 275 <> idx_seq: 275 <> gen pos G.H3.12\n",
      "Found corr. gen num! idx_uni: 276 <> idx_seq: 276 <> gen pos G.H3.13\n",
      "Found corr. gen num! idx_uni: 277 <> idx_seq: 277 <> gen pos G.H3.14\n",
      "Found corr. gen num! idx_uni: 278 <> idx_seq: 278 <> gen pos G.H3.15\n",
      "Found corr. gen num! idx_uni: 279 <> idx_seq: 279 <> gen pos G.H3.16\n",
      "Found corr. gen num! idx_uni: 280 <> idx_seq: 280 <> gen pos G.H3.17\n",
      "Found corr. gen num! idx_uni: 281 <> idx_seq: 281 <> gen pos G.H3.18\n",
      "Found corr. gen num! idx_uni: 282 <> idx_seq: 282 <> gen pos G.h3s5.01\n",
      "Found corr. gen num! idx_uni: 283 <> idx_seq: 283 <> gen pos G.h3s5.02\n",
      "Found corr. gen num! idx_uni: 284 <> idx_seq: 284 <> gen pos G.h3s5.03\n",
      "Found corr. gen num! idx_uni: 285 <> idx_seq: 285 <> gen pos G.S5.01\n",
      "Found corr. gen num! idx_uni: 286 <> idx_seq: 286 <> gen pos G.S5.02\n",
      "Found corr. gen num! idx_uni: 287 <> idx_seq: 287 <> gen pos G.S5.03\n",
      "Found corr. gen num! idx_uni: 288 <> idx_seq: 288 <> gen pos G.S5.04\n",
      "Found corr. gen num! idx_uni: 289 <> idx_seq: 289 <> gen pos G.S5.05\n",
      "Found corr. gen num! idx_uni: 290 <> idx_seq: 290 <> gen pos G.S5.06\n",
      "Found corr. gen num! idx_uni: 291 <> idx_seq: 291 <> gen pos G.S5.07\n",
      "Found corr. gen num! idx_uni: 292 <> idx_seq: 292 <> gen pos G.s5hg.01\n",
      "Found corr. gen num! idx_uni: 293 <> idx_seq: 293 <> gen pos G.HG.01\n",
      "Found corr. gen num! idx_uni: 294 <> idx_seq: 294 <> gen pos G.HG.02\n",
      "Found corr. gen num! idx_uni: 295 <> idx_seq: 295 <> gen pos G.HG.03\n",
      "Found corr. gen num! idx_uni: 296 <> idx_seq: 296 <> gen pos G.HG.04\n",
      "Found corr. gen num! idx_uni: 297 <> idx_seq: 297 <> gen pos G.HG.05\n",
      "Found corr. gen num! idx_uni: 298 <> idx_seq: 298 <> gen pos G.HG.06\n",
      "Found corr. gen num! idx_uni: 299 <> idx_seq: 299 <> gen pos G.HG.07"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "100%|████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 67/67 [00:50<00:00,  1.32it/s]"
     ]
    },
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "\n",
      "Found corr. gen num! idx_uni: 300 <> idx_seq: 300 <> gen pos G.HG.08\n",
      "Found corr. gen num! idx_uni: 301 <> idx_seq: 301 <> gen pos G.HG.09\n",
      "Found corr. gen num! idx_uni: 302 <> idx_seq: 302 <> gen pos G.HG.10\n",
      "Found corr. gen num! idx_uni: 303 <> idx_seq: 303 <> gen pos G.HG.11\n",
      "Found corr. gen num! idx_uni: 304 <> idx_seq: 304 <> gen pos G.HG.12\n",
      "Found corr. gen num! idx_uni: 305 <> idx_seq: 305 <> gen pos G.HG.13\n",
      "Found corr. gen num! idx_uni: 306 <> idx_seq: 306 <> gen pos G.HG.14\n",
      "Found corr. gen num! idx_uni: 307 <> idx_seq: 307 <> gen pos G.HG.15\n",
      "Found corr. gen num! idx_uni: 308 <> idx_seq: 308 <> gen pos G.HG.16\n",
      "Found corr. gen num! idx_uni: 309 <> idx_seq: 309 <> gen pos G.HG.17\n",
      "Found corr. gen num! idx_uni: 310 <> idx_seq: 310 <> gen pos G.hgh4.01\n",
      "Found corr. gen num! idx_uni: 311 <> idx_seq: 311 <> gen pos G.hgh4.02\n",
      "Found corr. gen num! idx_uni: 312 <> idx_seq: 312 <> gen pos G.hgh4.03\n",
      "Found corr. gen num! idx_uni: 313 <> idx_seq: 313 <> gen pos G.hgh4.04\n",
      "Found corr. gen num! idx_uni: 314 <> idx_seq: 314 <> gen pos G.hgh4.05\n",
      "Found corr. gen num! idx_uni: 315 <> idx_seq: 315 <> gen pos G.hgh4.06\n",
      "Found corr. gen num! idx_uni: 316 <> idx_seq: 316 <> gen pos G.hgh4.07\n",
      "Found corr. gen num! idx_uni: 317 <> idx_seq: 317 <> gen pos G.hgh4.08\n",
      "Found corr. gen num! idx_uni: 318 <> idx_seq: 318 <> gen pos G.hgh4.09\n",
      "Found corr. gen num! idx_uni: 319 <> idx_seq: 319 <> gen pos G.hgh4.10\n",
      "Found corr. gen num! idx_uni: 320 <> idx_seq: 320 <> gen pos G.hgh4.11\n",
      "Found corr. gen num! idx_uni: 321 <> idx_seq: 321 <> gen pos G.hgh4.12\n",
      "Found corr. gen num! idx_uni: 322 <> idx_seq: 322 <> gen pos G.hgh4.13\n",
      "Found corr. gen num! idx_uni: 323 <> idx_seq: 323 <> gen pos G.hgh4.14\n",
      "Found corr. gen num! idx_uni: 324 <> idx_seq: 324 <> gen pos G.hgh4.15\n",
      "Found corr. gen num! idx_uni: 325 <> idx_seq: 325 <> gen pos G.hgh4.16\n",
      "Found corr. gen num! idx_uni: 326 <> idx_seq: 326 <> gen pos G.hgh4.17\n",
      "Found corr. gen num! idx_uni: 327 <> idx_seq: 327 <> gen pos G.hgh4.18\n",
      "Found corr. gen num! idx_uni: 328 <> idx_seq: 328 <> gen pos G.hgh4.19\n",
      "Found corr. gen num! idx_uni: 329 <> idx_seq: 329 <> gen pos G.hgh4.20\n",
      "Found corr. gen num! idx_uni: 330 <> idx_seq: 330 <> gen pos G.hgh4.21\n",
      "Found corr. gen num! idx_uni: 331 <> idx_seq: 331 <> gen pos G.H4.01\n",
      "Found corr. gen num! idx_uni: 332 <> idx_seq: 332 <> gen pos G.H4.02\n",
      "Found corr. gen num! idx_uni: 333 <> idx_seq: 333 <> gen pos G.H4.03\n",
      "Found corr. gen num! idx_uni: 334 <> idx_seq: 334 <> gen pos G.H4.04\n",
      "Found corr. gen num! idx_uni: 335 <> idx_seq: 335 <> gen pos G.H4.05\n",
      "Found corr. gen num! idx_uni: 336 <> idx_seq: 336 <> gen pos G.H4.06\n",
      "Found corr. gen num! idx_uni: 337 <> idx_seq: 337 <> gen pos G.H4.07\n",
      "Found corr. gen num! idx_uni: 338 <> idx_seq: 338 <> gen pos G.H4.08\n",
      "Found corr. gen num! idx_uni: 339 <> idx_seq: 339 <> gen pos G.H4.10\n",
      "Found corr. gen num! idx_uni: 340 <> idx_seq: 340 <> gen pos G.H4.11\n",
      "Found corr. gen num! idx_uni: 341 <> idx_seq: 341 <> gen pos G.H4.12\n",
      "Found corr. gen num! idx_uni: 342 <> idx_seq: 342 <> gen pos G.H4.13\n",
      "Found corr. gen num! idx_uni: 343 <> idx_seq: 343 <> gen pos G.H4.14\n",
      "Found corr. gen num! idx_uni: 344 <> idx_seq: 344 <> gen pos G.H4.15\n",
      "Found corr. gen num! idx_uni: 345 <> idx_seq: 345 <> gen pos G.H4.16\n",
      "Found corr. gen num! idx_uni: 346 <> idx_seq: 346 <> gen pos G.H4.17\n",
      "Found corr. gen num! idx_uni: 347 <> idx_seq: 347 <> gen pos G.h4s6.01\n",
      "Found corr. gen num! idx_uni: 348 <> idx_seq: 348 <> gen pos G.h4s6.02\n",
      "Found corr. gen num! idx_uni: 349 <> idx_seq: 349 <> gen pos G.h4s6.03\n",
      "Found corr. gen num! idx_uni: 350 <> idx_seq: 350 <> gen pos G.h4s6.04\n",
      "Found corr. gen num! idx_uni: 351 <> idx_seq: 351 <> gen pos G.h4s6.08\n",
      "Found corr. gen num! idx_uni: 352 <> idx_seq: 352 <> gen pos G.h4s6.09\n",
      "Found corr. gen num! idx_uni: 353 <> idx_seq: 353 <> gen pos G.h4s6.10\n",
      "Found corr. gen num! idx_uni: 354 <> idx_seq: 354 <> gen pos G.h4s6.11\n",
      "Found corr. gen num! idx_uni: 355 <> idx_seq: 355 <> gen pos G.h4s6.12\n",
      "Found corr. gen num! idx_uni: 356 <> idx_seq: 356 <> gen pos G.h4s6.13\n",
      "Found corr. gen num! idx_uni: 357 <> idx_seq: 357 <> gen pos G.h4s6.20\n",
      "Found corr. gen num! idx_uni: 358 <> idx_seq: 358 <> gen pos G.S6.01\n",
      "Found corr. gen num! idx_uni: 359 <> idx_seq: 359 <> gen pos G.S6.02\n",
      "Found corr. gen num! idx_uni: 360 <> idx_seq: 360 <> gen pos G.S6.03\n",
      "Found corr. gen num! idx_uni: 361 <> idx_seq: 361 <> gen pos G.S6.04\n",
      "Found corr. gen num! idx_uni: 362 <> idx_seq: 362 <> gen pos G.S6.05\n",
      "Found corr. gen num! idx_uni: 363 <> idx_seq: 363 <> gen pos G.s6h5.01\n",
      "Found corr. gen num! idx_uni: 364 <> idx_seq: 364 <> gen pos G.s6h5.02\n",
      "Found corr. gen num! idx_uni: 365 <> idx_seq: 365 <> gen pos G.s6h5.03\n",
      "Found corr. gen num! idx_uni: 366 <> idx_seq: 366 <> gen pos G.s6h5.04\n",
      "Found corr. gen num! idx_uni: 367 <> idx_seq: 367 <> gen pos G.s6h5.05\n",
      "Found corr. gen num! idx_uni: 368 <> idx_seq: 368 <> gen pos G.H5.01\n",
      "Found corr. gen num! idx_uni: 369 <> idx_seq: 369 <> gen pos G.H5.02\n",
      "Found corr. gen num! idx_uni: 370 <> idx_seq: 370 <> gen pos G.H5.03\n",
      "Found corr. gen num! idx_uni: 371 <> idx_seq: 371 <> gen pos G.H5.04\n",
      "Found corr. gen num! idx_uni: 372 <> idx_seq: 372 <> gen pos G.H5.05\n",
      "Found corr. gen num! idx_uni: 373 <> idx_seq: 373 <> gen pos G.H5.06\n",
      "Found corr. gen num! idx_uni: 374 <> idx_seq: 374 <> gen pos G.H5.07\n",
      "Found corr. gen num! idx_uni: 375 <> idx_seq: 375 <> gen pos G.H5.08\n",
      "Found corr. gen num! idx_uni: 376 <> idx_seq: 376 <> gen pos G.H5.09\n",
      "Found corr. gen num! idx_uni: 377 <> idx_seq: 377 <> gen pos G.H5.10\n",
      "Found corr. gen num! idx_uni: 378 <> idx_seq: 378 <> gen pos G.H5.11\n",
      "Found corr. gen num! idx_uni: 379 <> idx_seq: 379 <> gen pos G.H5.12\n",
      "Found corr. gen num! idx_uni: 380 <> idx_seq: 380 <> gen pos G.H5.13\n",
      "Found corr. gen num! idx_uni: 381 <> idx_seq: 381 <> gen pos G.H5.14\n",
      "Found corr. gen num! idx_uni: 382 <> idx_seq: 382 <> gen pos G.H5.15\n",
      "Found corr. gen num! idx_uni: 383 <> idx_seq: 383 <> gen pos G.H5.16\n",
      "Found corr. gen num! idx_uni: 384 <> idx_seq: 384 <> gen pos G.H5.17\n",
      "Found corr. gen num! idx_uni: 385 <> idx_seq: 385 <> gen pos G.H5.18\n",
      "Found corr. gen num! idx_uni: 386 <> idx_seq: 386 <> gen pos G.H5.19\n",
      "Found corr. gen num! idx_uni: 387 <> idx_seq: 387 <> gen pos G.H5.20\n",
      "Found corr. gen num! idx_uni: 388 <> idx_seq: 388 <> gen pos G.H5.21\n",
      "Found corr. gen num! idx_uni: 389 <> idx_seq: 389 <> gen pos G.H5.22\n",
      "Found corr. gen num! idx_uni: 390 <> idx_seq: 390 <> gen pos G.H5.23\n",
      "Found corr. gen num! idx_uni: 391 <> idx_seq: 391 <> gen pos G.H5.24\n",
      "Found corr. gen num! idx_uni: 392 <> idx_seq: 392 <> gen pos G.H5.25\n",
      "Found corr. gen num! idx_uni: 393 <> idx_seq: 393 <> gen pos G.H5.26\n",
      "Writing files with generic numbers on receptors and gproteins.\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "\n"
     ]
    }
   ],
   "source": [
    "p.assign_generic_numbers_g(f=f_act, pdb_ids=[], overwrite=False, folder='data/processed/', fill_H5=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "09fc6787",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "557"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(p.dfl)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "60cc79e8",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "p.apply_filter(f_act)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 22,
   "id": "01d2365b",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "67"
      ]
     },
     "execution_count": 22,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(p.dfl)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "2da17cdf",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "df = p.dfl[10]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 28,
   "id": "4100b0d8",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>PDB</th>\n",
       "      <th>group_PDB</th>\n",
       "      <th>auth_asym_id</th>\n",
       "      <th>label_asym_id</th>\n",
       "      <th>label_seq_id</th>\n",
       "      <th>auth_seq_id</th>\n",
       "      <th>label_comp_id</th>\n",
       "      <th>id</th>\n",
       "      <th>label_atom_id</th>\n",
       "      <th>type_symbol</th>\n",
       "      <th>...</th>\n",
       "      <th>identifier</th>\n",
       "      <th>label_comp_sid</th>\n",
       "      <th>label_2_uni</th>\n",
       "      <th>gen_pos</th>\n",
       "      <th>gen_pos1</th>\n",
       "      <th>gen_pos2</th>\n",
       "      <th>uniprot_comp_sid</th>\n",
       "      <th>gprot_pos</th>\n",
       "      <th>uniprot_comp_id</th>\n",
       "      <th>fam_comp_id</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>6LFM</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>2</td>\n",
       "      <td>3</td>\n",
       "      <td>CYS</td>\n",
       "      <td>2</td>\n",
       "      <td>CA</td>\n",
       "      <td>C</td>\n",
       "      <td>...</td>\n",
       "      <td>CXCR2_HUMAN</td>\n",
       "      <td>C</td>\n",
       "      <td>2</td>\n",
       "      <td></td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td></td>\n",
       "      <td>G.HN.09</td>\n",
       "      <td>C</td>\n",
       "      <td>C</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>6LFM</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>3</td>\n",
       "      <td>4</td>\n",
       "      <td>THR</td>\n",
       "      <td>8</td>\n",
       "      <td>CA</td>\n",
       "      <td>C</td>\n",
       "      <td>...</td>\n",
       "      <td>CXCR2_HUMAN</td>\n",
       "      <td>T</td>\n",
       "      <td>3</td>\n",
       "      <td></td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td></td>\n",
       "      <td>G.HN.10</td>\n",
       "      <td>T</td>\n",
       "      <td>T</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>6LFM</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>4</td>\n",
       "      <td>5</td>\n",
       "      <td>LEU</td>\n",
       "      <td>15</td>\n",
       "      <td>CA</td>\n",
       "      <td>C</td>\n",
       "      <td>...</td>\n",
       "      <td>CXCR2_HUMAN</td>\n",
       "      <td>L</td>\n",
       "      <td>4</td>\n",
       "      <td></td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td></td>\n",
       "      <td>G.HN.11</td>\n",
       "      <td>L</td>\n",
       "      <td>L</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>6LFM</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>5</td>\n",
       "      <td>6</td>\n",
       "      <td>SER</td>\n",
       "      <td>23</td>\n",
       "      <td>CA</td>\n",
       "      <td>C</td>\n",
       "      <td>...</td>\n",
       "      <td>CXCR2_HUMAN</td>\n",
       "      <td>S</td>\n",
       "      <td>5</td>\n",
       "      <td></td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td></td>\n",
       "      <td>G.HN.30</td>\n",
       "      <td>S</td>\n",
       "      <td>S</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>6LFM</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>6</td>\n",
       "      <td>7</td>\n",
       "      <td>ALA</td>\n",
       "      <td>29</td>\n",
       "      <td>CA</td>\n",
       "      <td>C</td>\n",
       "      <td>...</td>\n",
       "      <td>CXCR2_HUMAN</td>\n",
       "      <td>A</td>\n",
       "      <td>6</td>\n",
       "      <td></td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td></td>\n",
       "      <td>G.HN.31</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>6LFM</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>7</td>\n",
       "      <td>8</td>\n",
       "      <td>GLU</td>\n",
       "      <td>34</td>\n",
       "      <td>CA</td>\n",
       "      <td>C</td>\n",
       "      <td>...</td>\n",
       "      <td>CXCR2_HUMAN</td>\n",
       "      <td>E</td>\n",
       "      <td>7</td>\n",
       "      <td></td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td></td>\n",
       "      <td>G.HN.32</td>\n",
       "      <td>E</td>\n",
       "      <td>E</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>6LFM</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>8</td>\n",
       "      <td>9</td>\n",
       "      <td>ASP</td>\n",
       "      <td>43</td>\n",
       "      <td>CA</td>\n",
       "      <td>C</td>\n",
       "      <td>...</td>\n",
       "      <td>CXCR2_HUMAN</td>\n",
       "      <td>D</td>\n",
       "      <td>8</td>\n",
       "      <td></td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td></td>\n",
       "      <td>G.HN.33</td>\n",
       "      <td>D</td>\n",
       "      <td>D</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>6LFM</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>9</td>\n",
       "      <td>10</td>\n",
       "      <td>LYS</td>\n",
       "      <td>51</td>\n",
       "      <td>CA</td>\n",
       "      <td>C</td>\n",
       "      <td>...</td>\n",
       "      <td>CXCR2_HUMAN</td>\n",
       "      <td>K</td>\n",
       "      <td>9</td>\n",
       "      <td></td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td></td>\n",
       "      <td>G.HN.34</td>\n",
       "      <td>K</td>\n",
       "      <td>K</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>6LFM</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>10</td>\n",
       "      <td>11</td>\n",
       "      <td>ALA</td>\n",
       "      <td>60</td>\n",
       "      <td>CA</td>\n",
       "      <td>C</td>\n",
       "      <td>...</td>\n",
       "      <td>CXCR2_HUMAN</td>\n",
       "      <td>A</td>\n",
       "      <td>10</td>\n",
       "      <td></td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td></td>\n",
       "      <td>G.HN.35</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>6LFM</td>\n",
       "      <td>ATOM</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "      <td>11</td>\n",
       "      <td>12</td>\n",
       "      <td>ALA</td>\n",
       "      <td>65</td>\n",
       "      <td>CA</td>\n",
       "      <td>C</td>\n",
       "      <td>...</td>\n",
       "      <td>CXCR2_HUMAN</td>\n",
       "      <td>A</td>\n",
       "      <td>11</td>\n",
       "      <td></td>\n",
       "      <td>0.0</td>\n",
       "      <td>0.0</td>\n",
       "      <td></td>\n",
       "      <td>G.HN.36</td>\n",
       "      <td>A</td>\n",
       "      <td>A</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>10 rows × 26 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "     PDB group_PDB auth_asym_id label_asym_id  label_seq_id auth_seq_id  \\\n",
       "1   6LFM      ATOM            A             A             2           3   \n",
       "7   6LFM      ATOM            A             A             3           4   \n",
       "14  6LFM      ATOM            A             A             4           5   \n",
       "22  6LFM      ATOM            A             A             5           6   \n",
       "28  6LFM      ATOM            A             A             6           7   \n",
       "33  6LFM      ATOM            A             A             7           8   \n",
       "42  6LFM      ATOM            A             A             8           9   \n",
       "50  6LFM      ATOM            A             A             9          10   \n",
       "59  6LFM      ATOM            A             A            10          11   \n",
       "64  6LFM      ATOM            A             A            11          12   \n",
       "\n",
       "   label_comp_id  id label_atom_id type_symbol  ...   identifier  \\\n",
       "1            CYS   2            CA           C  ...  CXCR2_HUMAN   \n",
       "7            THR   8            CA           C  ...  CXCR2_HUMAN   \n",
       "14           LEU  15            CA           C  ...  CXCR2_HUMAN   \n",
       "22           SER  23            CA           C  ...  CXCR2_HUMAN   \n",
       "28           ALA  29            CA           C  ...  CXCR2_HUMAN   \n",
       "33           GLU  34            CA           C  ...  CXCR2_HUMAN   \n",
       "42           ASP  43            CA           C  ...  CXCR2_HUMAN   \n",
       "50           LYS  51            CA           C  ...  CXCR2_HUMAN   \n",
       "59           ALA  60            CA           C  ...  CXCR2_HUMAN   \n",
       "64           ALA  65            CA           C  ...  CXCR2_HUMAN   \n",
       "\n",
       "   label_comp_sid label_2_uni  gen_pos  gen_pos1  gen_pos2 uniprot_comp_sid  \\\n",
       "1               C           2                0.0       0.0                    \n",
       "7               T           3                0.0       0.0                    \n",
       "14              L           4                0.0       0.0                    \n",
       "22              S           5                0.0       0.0                    \n",
       "28              A           6                0.0       0.0                    \n",
       "33              E           7                0.0       0.0                    \n",
       "42              D           8                0.0       0.0                    \n",
       "50              K           9                0.0       0.0                    \n",
       "59              A          10                0.0       0.0                    \n",
       "64              A          11                0.0       0.0                    \n",
       "\n",
       "   gprot_pos  uniprot_comp_id fam_comp_id  \n",
       "1    G.HN.09                C           C  \n",
       "7    G.HN.10                T           T  \n",
       "14   G.HN.11                L           L  \n",
       "22   G.HN.30                S           S  \n",
       "28   G.HN.31                A           A  \n",
       "33   G.HN.32                E           E  \n",
       "42   G.HN.33                D           D  \n",
       "50   G.HN.34                K           K  \n",
       "59   G.HN.35                A           A  \n",
       "64   G.HN.36                A           A  \n",
       "\n",
       "[10 rows x 26 columns]"
      ]
     },
     "execution_count": 28,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "df[df['gprot_pos']!=''].head(10)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2bbb2294",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Assigning generic residue numbers to anything but **active complexes** is a waste of computational time and potentially errenous, since looking for the *BEST* alignment of any gprotein to *anything* in your structure **may result in things getting labelled that are NOT the gprotein** (there is a threshold on the alignment score, but I've not checked what happens... so I take no responsibility ^^). "
   ]
  },
  {
   "cell_type": "markdown",
   "id": "e0cd9a8c",
   "metadata": {
    "hidden": true
   },
   "source": [
    "The fill_H5 flag was used since many of the H5-helices did not get labelled - especially in weird active complexes where not the full H5 is present."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "821e374e",
   "metadata": {
    "heading_collapsed": true,
    "hidden": true
   },
   "source": [
    "## Modifying *.cif* files"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 32,
   "id": "5b1626b3",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "  0%|                                                                                                                              | 0/7 [00:00<?, ?it/s]\n",
      "  0%|                                                                                                                            | 0/352 [00:00<?, ?it/s]\u001b[A\n",
      " 15%|████████████████▊                                                                                                 | 52/352 [00:00<00:00, 509.74it/s]\u001b[A\n",
      " 29%|█████████████████████████████████                                                                                | 103/352 [00:00<00:00, 301.94it/s]\u001b[A\n",
      " 39%|████████████████████████████████████████████▎                                                                    | 138/352 [00:00<00:00, 274.29it/s]\u001b[A\n",
      " 48%|█████████████████████████████████████████████████████▉                                                           | 168/352 [00:00<00:00, 257.81it/s]\u001b[A\n",
      " 57%|████████████████████████████████████████████████████████████████▌                                                | 201/352 [00:00<00:00, 276.85it/s]\u001b[A\n",
      " 65%|█████████████████████████████████████████████████████████████████████████▊                                       | 230/352 [00:00<00:00, 260.74it/s]\u001b[A\n",
      " 73%|██████████████████████████████████████████████████████████████████████████████████▌                              | 257/352 [00:00<00:00, 256.95it/s]\u001b[A\n",
      " 81%|███████████████████████████████████████████████████████████████████████████████████████████▏                     | 284/352 [00:01<00:00, 255.60it/s]\u001b[A\n",
      " 88%|███████████████████████████████████████████████████████████████████████████████████████████████████▌             | 310/352 [00:01<00:00, 249.19it/s]\u001b[A\n",
      "100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 352/352 [00:01<00:00, 274.94it/s]\u001b[A\n",
      " 57%|███████████████████████████████████████████████████████████████████▍                                                  | 4/7 [00:01<00:00,  3.11it/s]\n",
      "  0%|                                                                                                                            | 0/352 [00:00<?, ?it/s]\u001b[A\n",
      "  7%|███████▊                                                                                                          | 24/352 [00:00<00:01, 235.24it/s]\u001b[A\n",
      " 14%|███████████████▊                                                                                                  | 49/352 [00:00<00:01, 239.62it/s]\u001b[A\n",
      " 30%|██████████████████████████████████▎                                                                              | 107/352 [00:00<00:00, 392.83it/s]\u001b[A\n",
      " 50%|████████████████████████████████████████████████████████▏                                                        | 175/352 [00:00<00:00, 505.19it/s]\u001b[A\n",
      " 64%|████████████████████████████████████████████████████████████████████████▌                                        | 226/352 [00:00<00:00, 359.77it/s]\u001b[A\n",
      " 76%|██████████████████████████████████████████████████████████████████████████████████████                           | 268/352 [00:00<00:00, 308.37it/s]\u001b[A\n",
      " 86%|█████████████████████████████████████████████████████████████████████████████████████████████████▎               | 303/352 [00:00<00:00, 287.46it/s]\u001b[A\n",
      "100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 352/352 [00:01<00:00, 304.16it/s]\u001b[A\n",
      "100%|██████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 7/7 [00:02<00:00,  2.86it/s]\n"
     ]
    }
   ],
   "source": [
    "structure = p.dfl[10]\n",
    "df = spread_gen_pos_to_residue_atoms(structure)\n",
    "doc = write_cif_with_gen_seq_numbers(df, replace_idx = -7)\n",
    "doc.write_file('data/modified/'+df['PDB'].iloc[0]+'_modified.cif')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "2d49d2b0",
   "metadata": {
    "hidden": true
   },
   "source": [
    "The only thing to mention here is that cif (or pdb files in general) and their \"visualization\" in pymol are fixed, and we merely choose a column to overwrite (e.g. 'b-factors') with the generic position numbers which can be labelled in pymol.\n",
    "This works reasonably well for gpcr generic position numbers (since they can be represented as a float, however for gproteins the type of *string* makes this crude method fail (assigns a 0.0 instead of 'G.H5.26'), another column could be the 'auth_seq_id'..."
   ]
  },
  {
   "cell_type": "markdown",
   "id": "713a3327",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "# ANALSYSIS"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "66a3ae2c",
   "metadata": {
    "hidden": true
   },
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "a34a1754",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "b5e86574",
   "metadata": {
    "hidden": true
   },
   "source": [
    "## DISTANCE ANALYSIS"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "03ee3dac",
   "metadata": {
    "hidden": true
   },
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "0319f36f",
   "metadata": {
    "hidden": true
   },
   "source": [
    "## HELICES - ANGLE ANALYSIS"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "6c93fd5f",
   "metadata": {
    "hidden": true
   },
   "source": [
    "To do helices analysis (both helices in the gprotein and gpcrs) we need to initialize a processor in 'rg' mode:"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 36,
   "id": "6be9146a",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "del p"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 37,
   "id": "d5b71709",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Reading files with generic numbers on receptors and gproteins.\n"
     ]
    }
   ],
   "source": [
    "p = CifProcessor()\n",
    "p.read_pkl_metainfo()\n",
    "p.read_pkl(mode='rg', folder='data/processed/')"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "1764123b",
   "metadata": {
    "hidden": true
   },
   "source": [
    "Filter the data..."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 39,
   "id": "cc1096d7",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": [
    "f_act = p.make_filter(State='Active', Cl='Rhodopsin', gprotein=True)\n",
    "f_act = f_act[f_act['Method']=='cryo-EM']\n",
    "p.apply_filter(f_act)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 40,
   "id": "4562f18d",
   "metadata": {
    "hidden": true
   },
   "outputs": [
    {
     "data": {
      "text/plain": [
       "67"
      ]
     },
     "execution_count": 40,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "len(p.dfl)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c794892b",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "59d354df",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "0cd41550",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "4dfaa598",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "6a078504",
   "metadata": {
    "heading_collapsed": true
   },
   "source": [
    "# AFFINITY/SELECTIVITY PREDCITION"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "a419bff4",
   "metadata": {
    "hidden": true
   },
   "source": [
    "## AFFINITIES"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "17ed4769",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "f20d7d24",
   "metadata": {
    "hidden": true
   },
   "source": [
    "## DATALOADER / DATASET"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "3980dc02",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "69b0967e",
   "metadata": {
    "hidden": true
   },
   "source": [
    "## NEURAL NETWORK"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "fbcce6ed",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "markdown",
   "id": "05b2531c",
   "metadata": {
    "hidden": true
   },
   "source": [
    "## REVERSE NN ANALYSIS"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "c6a7feb2",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "6174530a",
   "metadata": {
    "hidden": true
   },
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
